**Cancer is an epigenetic disease. Discuss.**

# Intro 

* Epigenetics refers to heritable changes in phenotype that do not alter the genotype 
* What does it mean when we say cancer is an epigenetic disease 

# Chromatin modifications 
* Chromatin modifications 
    * DNA methylation 
    * Histone modification 
    * Nucleosome remodelling 
    * Superenhancers? 

# miRNAs
* Non-coding RNAs – act as epigenetic modulators, as they modify gene expression levels without changing genetic sequence 
    * microRNA mode of action 
    * Alteration of miRNA expression in cancer 
        * Tumour suppressor 
            * Mutations in pri-miRNA sequence e.g. SNP in pri-miR-16 decreased its rate of processing, and this was associated with CLL development 
            * Copy number alteration. E.g. miR-16-1 and miR-15a deletion is found in over 15% of CLL patients. These miRs suppress Bcl-2, which is an oncogene. 
            * SNP in target gene 3’ UTR. e.g. KRAS 3’ UTR can have SNPs; this can prevent Let-7 binding, regulation 
            * Genetic abnormalities – e.g. 61 miRNAs have been identified in breakpoint regions and fragile sites (e.g. let-7 miRNAs) 
            * Altered levels of miRNA biogenesis machinery, e.g. Dicer decreased in level by 60% in 39% of ovarian cancer patients 
                * In fact, hemizygous genetic deletion of Dicer occurs in 27% of tumours across tissue types (does not explain why miRNA is depleted in other tumours without Dicer haploinsufficiency though) 
            * Loss of tumor suppressor genes that activate them e.g. p53 
                * E.g. miR-34 is involved in inducing apoptosis, senescence, loss of migration. Loss of p53 means decrease in miR-34 expression 
                * >46% of miRNA promoters contain potential p53 binding sites 
* Acquire epigenetic repressive marks e.g. increase in methylation
	* E.g. miR-127 is epigenetically repressed in 75% of primary prostate and bladder tumours; miR-127 normally inhibits expression of proto-oncogene _BCL6_ 
	* Oncogenic 
		* Oncogenic miRNA could lose epigenetic repressive marks
		* Genetic amplification 
		* Activation by TFs e.g. MYC upregulates miR-17/92 cluster 
    * miRNAs involved in diverse signalling pathways, and so it is unsurprising that they can also be dysregulated in cancer 
    * Most miRNAs are repressed in cancer tissue, which is associated with de-differentiation 
    * microRNAs affect all hallmarks of cancer 

## Oncogenic miRNAs
* microRNAs that act as oncogenes (how?) 
    * Self-sufficiency in growth signals, e.g. let-7 
    * Insensitivity to anti-growth signals, e.g. miR-17/19
    * Evade apoptosis, e.g. miR-34a 
    * Angiogenesis e.g. miR-210 
    * Invasion and metastasis e.g. miR-10b 
    * E.g. miR-21 targets PTEN, and is upregulated in human glioblastoma 
    * E.g. miRNA-155 is upregulated in breast, lung, and hematopoietic cancers; mechanism of action is unknown 
    * Suppression of intrinsic tumour suppression. E.g. Myc induces cell apoptosis when it is hyperactive via an apoptotic inducer, Bim. miR-17/92 cluster targets pro-apoptotic gene Bim, is overexpressed in several cancer types. This allows myc to be overexpressed without repercussions on the cancer cell survival 
        * Ectopic expression of miR-17 in colon cancer also was associated with chemoresistance 
    * Repression of tumour suppression genes. E.g. miRNAs modulate pRb pathway at almost all levels. 3’ UTR of pRB gene contains binding site for miR-106a, which is overexpressed in a range of human cancers. miR-17-92 downregulates E2F. 
        * On the other hand, there is a loss of p53-induced miRNAs that interact with the pRb pathway. E.g. miR-34a suppresses expression of cyclin D1 and CDK6, while miR-192 and miR-215 enhance accumulation of p21 and induce cell cycle arrest. With loss of p53, this suppression cannot occur, thus promoting cell cycle progression 
        * miR124 and miR137 are also lost because of inactivation by epigenetic changes in tumour cells, which lifts suppression of CDK6
        * miR-17/20 is also often downregulated in breast cancers, and it targets D1
        * Can see that miRNAs regulate signalling pathways at multiple layers, and each layer can be affected by miRNA
    * Angiogenesis. miR126 increased angiogenic effects of VEGF, FGF by targeting Spred-1, an inhibitor of the VEGF and FGF signalling pathways 
    * Metastasis. 
        * Targeting transcription factors. SNAI and ZEBfamily  transcription factors are involved in EMT; binds to E-box-like promoter in genes. 3’ UTR of ZEB contains miR-200 binding sites, and MiR-204 directly inhibits Snai1 mRNA transcription 
        * Epigenetic modification of metastatic genes. E.g. miR-101 inhibits EZH2 expression, and EZH2 is a histone methyltransferase that silences genes involved metastasis 
        * Cadherin switch. During EMT, E-cadherin is lost, while N-cadherin is upregulated. miR-9 inhibits translation of E-cadherin. miR-199 and 218 are potential regulators of N-cadherin, but this has not yet been confirmed. 
        * Loss of apical-basal polarity. Polarity is needed to organise epithelial tissues. miRNAs regulate Par-3, which is involved in formation of tight junctions in epithelial tissue. TGFbeta signalling activates miR-491-5p, which inhibits Par-3 translation. Par-4 is inhibited by miR-451. 
        * Altered integrin expression. This alters the adhesion of cells to the surrounding ECM. Focal adhesion kinase (FAK)-Src complex conveys signals from integrins and receptor tyrosine kinases to cytoskeleton. Some miRNAs are known to regulate FAK signalling, e.g. miR-151, which indirectly suppresses FAK signalling 
        * PTEN suppression. E.g. miR-17-92 
    * E.g. miR-221/222 → TRAIL resistance in lung cancer, Tamoxifen resistance in breast cancer
        * Also reported that they decrease expression of epithelium-specific genes while increasing expression of mesenchymal genes → Promote EMT? 
            * FOSL1 transcription factor stimulates transcription of miR-222/221, and the miRNA targets TRPS1, a Zeb2 inhibitor. This leads to increased Zeb2 expression. 

## Tumour suppressor miRNAs
* microRNAs that act as tumour suppressors 
    * E.g. miR-15 and 16 target BCL2, an antiapoptotic gene. These miRNAs are downregulated in chronic lymphocyte leukemia
    * E.g. let-7 family targets RAS, and is downregulated in lung cancer 
    * E.g. miR-127, targets BCL6, downregulated in prostate and bladder tumours 
        * Saito et al reported that miR-127 is silenced by DNA methylation, as it is embedded in a CpG island 
    * E.g. mir-101, targets polycomb group protein EZH2, downregulated in bladder transitional cell carcinoma 
    * Upregulation of tumour suppressor genes. E.g. miR-29a-c suppress p85, the regulatory subunit of PI3K, and CDC42, a Rho GTPase. Both inhibit p53 activity. So miR-29  upregulates p53 activity 
    * Mediate signalling of tumour suppressor genes. E.g. mir-34 family – are direct p53 targets, induce apoptosis and cell cycle arrest. BCL2 and MYCN are miR-34a targets.
        * Low expression in CLL is associated with p53 inactivation 
    * Downregulate oncogenes directly. E.g. miR-145 directly downregulates c-myc. miR-145 is induced by p53, linking p53 activity to downregulation of oncogenes 
    * miR-200 – miR-200 family suppress EMT, thus reducing invasiveness 
        * Gregory et al – constitutive expression of miR-200 family miRNAs in mesenchymal cells promotes MET, while inhibiting miR_200 stimulates mesenchymal state, leading to increased expression of ZEB1, and increased migration 
        * ZEB1 and SIP1 bind to E-box proximal promoter element of miR-200 to repress transcription → negative feedback lop 

# piRNAs
* piRNAs 
    * What are they? 
        * Single-stranded ncRNAs of 26-31 nucleotides, interact with PIWI proteins (part of Aroganaute family) 
        * Mode of action 
    * Involvement in cancer 
        * E.g. piR-L-163 downregulated in NSCLC cancer cell line compared to epithelial cell line
        * E.g. in breast, a number of piRNAs identified to be differentially expressed between tumour and normal tissue 

# tRNA-derived fragments
* tRNA-derived fragments 
    * What are they? 
        * tRNA-derived fragments are cleavage products of anticodon loop in mature tRNAs. Cleavage is done by the ribonuclease, angiogenin 
        * Can also be derived from ends of mature tRNA or primary tRNA transcript. Cleavage enzyme is unknown 
        * Or from anticodon stem loop of tRNAs 
    * Involvement in cancer 
        * E.g. Several tRNA-derived fragments are highly expressed in estrogen receptor-positive breast and androgen receptor-positive prostate cancer cell lines 
        * E.g. tRF-3027 downregulated during malignant transformation of B cells 
            * tRF-3027 represses expression of RPA1, which is involved in DNA repair and cell proliferation 
        * E.g. Goodarzi et al (2015) – a set of tRF-2s can act as tumor suppressors by sequestering YBX1, an RNA binding protein that stabilises oncogenic transcripts 
        * Found to be involved in cell proliferation – e.g. tRF-1001 found to be positively-correlated with cell proliferation by promoting G2-M transition in prostate cancer cell lines  

# Small nucleolar RNAs
* Small nucleolar RNAs (snoRNAs) 
    * What are they? 
        * Small nc RNA implicated in chemical modification of rRNA 
        * Form complexes with protein partners to form snoRNPs
        * snoRNAs recognise and bind complementary sequences on target RNA&lt; then signal to partner proteins which base to modify 
    * Involvement in cancer 
        * E.g. ACA42 (a snoRNA) found to downregulate CDC2L6 in an miRNA-like manner 
        * E.g. SNORA42 is frequently upregulated in NSCLC; in clinical samples, high expression is correlated to poor survival 

# lncRNAs
* lncRNAs 
    * What are they? 
        * ncRNAs of >200 nt 
        * Regulate transcription via many modes of action: signal, decoy, scaffold, guide, enhancer RNAs
            * Decoy – present decoy binding sites to TFs and other regulatory factors to reduce their availability
            * Scaffold – Provide platforms for complexes, e.g. RNPs
            * Guide – interact with RNPs to direct them to target genes 
            * Enhancer – Produced from enhancer regions, influence 3D organisation of DNA, or tether interacting proteins to enhancer regions 
    * Involvement in cancer 
        * Dysregulation of lncRNAs identified in glioblastoma, breast cancer, colorectal cancer, and more 
        * Tumour suppessors 
            * E.g. some eRNAs can bind p53; removal of these eRNAs reduces p53-dependent cell cycle arrest 
            * P53 activates a number of lncRNA-coding genes e.g. lincRNA-p21, which is a co-activator of p21 expression. lincRNA-p21 promotes apoptosis by targeting the Wnt-β-catenin pathway 
        * Oncogenes 
            * E.g. HOTAIR – molecular scaffold for PRC2 complex and LSD1-containing CoREST/REST complex 
                * HOTAIR functions with PRC2 in repressing HOXD locus by spreading H3K27me3 marks 
                * Overexpressed in advanced breast, colorectal cancer 
                * Manipulating HOTAIR levels in cancer cells can change their invasive ability by changing PRC2 occupancy (on what?) (Wang and Chang, 2011)
* E.g.  HOTTIP – transcriptional activation of 5’ HOXA genes; HOTTIP is brought into proximity of 5’ HOXA genes via chromatin looping and recruits MLL1 to add H3K4me3 which activates transcription 
    * 5’ HOXA cluster has been found to play a role in many leukemias, so possible that HOTTIP also could be overexpressed 
* Inducing EMT – e.g. In breast cancer, cancer-associated fibroblasts activate HOTAIR transcriptionally by binding SMAD2/3/4 to its promoter. HOTAIR represses CDK5RAP1, which stimulates CDK5 and thus EMT. 
* Acting as competing endogenous RNAs (ceRNAs) – e.g. lncRNA HULC is upregulated in hepatocellular carcinoma. HULC contains miR-372 binding sites, and its overexpression leads to reduced mIR-372 activity. miR-372 targets PRKACB, so repression of miR-372 activity upregulates PRKACB activity, which is to phosphorylate cAMP responsive element (CRE)-binding protein, a known oncogene. 
* p53 is regulated by lncRNAs, e.g. MALAT1; MALAT1 is upregulated in several cancers. Its depletion leads to activation of TP53 
* MALAT1 also induces EMT via PI3K/AKT pathway 
* How much does epigenetic contribute to cancer? 
    * Other non-epigenetic factors 
    * Cancer is highly heterogenous – not just an epigenetic disease alone, but epigenetics can play a large factor in altering the normal patterns of gene expression and signalling to promote uncontrolled cell proliferation. 

# Conclusion 
* Information can be used for clinical applications, e.g. epigenetic therapies 	
    * E.g. Saito et al reported reactivation of downregulated miR-127 in T24 bladder cancer cells by treating with chromatin modifying drugs, e.g. DNA methylation, HDAC inhibitors 
    * Or use ncRNAs as biomarkers for cancer detection 
        * Diagnostic signature of small non-coding RNAs in cancer
        * E.g. miR-21 expressed at high levels in glioblastoma compared to normal brain tissue → diagnostic marker 
        * Microarray analysis in breast cancer – miR-21 is overexpressed and can be used as prognostic marker 
        * As miRNAs are often found in circulation and in exosomes, could use miRNAs as a non-invasive biomarker, since only need blood sampling to detect tumour 
    * miRNA antagonists of oncogenic miRNAs, miRNA replacement of tumour suppressor miRNAs 
* More information can be obtained via deep sequencing 
* Mechanisms of lncRNAs not fully understood → necessary to get a better understanding of how they act in cancer
* miRNAs regulate protein expression in a variety of signalling pathways 
* Unlike other proto-oncogenes and tumour suppressors proteins, miRNAs all use the same mode of actions, to their variability is really dependent on nucleotide sequence information 