**What part do small linear motifs (SLiMs) play in the control of the cell cycle?**

**Introduction**
- SLiMs 3-4 amino acids long motifs found in an unstructured region 
- Very short, so can be found across the genome
- Largely regulate the timing of different protein-protein interactions throughout the cell cycle 
- Earliest SLiMs were found to function in cell cycle regulation 
	- As Tim Hunt defined it: "The motifs are linear...three-dimensional organisation is not required to bring distant segments of the molecule together to make the recognisable unit". In any case, SLiM motifs are small enough that unlikely to fold to form 3D structure 
- SLiMs generally bind with lower affinity to binding  partner, between 1-150 µM

# Main 
**SLiMs regulates sequence of phosphorylation events, ensuring cell cycle proceeds in correct order**

**SLiMs regulate timing of degradation, ensuring cell cycle is irreversible**

**Widespread nature of SLiMs allows cell cycle regulators to integrate signals** ? ?? 

# Conclusion 
- Difficult to detect because of their short length and because they are often found within disordered regions. Also poorly conserved 
- Short motifs more likely to arise/disappear via spontaneous mutations 
	- Evolutionarily labile -> fine-tune cell cycle regulation by changing sequence of SLiMs 

**Regulation of phosphorylation state**
- Cdk substrates have SLiMs, which are recognised by cyclins
	- E.g. CLN recognises L-P, CLB5 recognises RxL, CLB2 recognises LxF (mitotic cyclin in budding yeast)
- Cyclin CDKs have an extra kinase subunit called Cks that binds to the C-terminus of kinase and it recognises
- pRb binds cyclin D via SliMs; one is a classical cyclin RxL docking motif that binds the hydrophobic pocket of the cyclin, and another LxCxE motif in the cyclin that binds a groove on Rb 
- Polo has a kinase domain with C-terminal polo boxes; these polo boxes bind to S-pT/pS-P/Xaa which are generated by Cdk. Cdk phosphorylation thus generates SliMs for Polo kinase binding 
- Accessory proteins to phosphatase regulate substrate specificity of the phosphatase by recognising different SLiMs 
- Phosphorylated ENSA has a SLiM motif that are recognised by B55 phosphatase 
- PIN1 WW domain – pS-P recognition motif overlaps with sequences modified by CDKs 
	- Phosphorylation of CDC25C recognised by PIN1 
	- E.g. the transcription factor MYBL2 is phosphorylated by CDKs, creating a PIN1 WW domain binding motif. PIN1 binding causes conformational changes in MYBL2 that makes more sites available for phosphorylation, e.g. T476 and T487. This enables Polo kinase 1 docking, and Polo kinase mediated phosphorylation activates MYBL2 to transcribe cell cycle genes 
	- Sequential phosphorylation and binding to SLiMs ensures correct timing of MYBL2 signalling 
- RVxF motif most common PP1 docking motif, binding to hydrophobic groove in catalytic subunit 
	- Binding of RV(S/T)F to PP1 is subject to phosphorylation by aurora B. Phosphorylation blocks PP1 binding 
- RKVxxVIR motif described for PP2A B55 regulatory subunit 
- pRB – "docking sites for PP1 and Cdk-Cyclins (e.g. Cyclin-A2 (CCNA2)) on pRB1 overlap, which makes their binding to RB1 mutually exclusive. Hypophosphorylated RB1 blocks E2F-dependent transcription, while hyperphosphorylation inactivates RB1 as a repressor, thereby promoting cell cycle progression."

**Regulation of degradation**
- APC/C can recognise different substrates at different times via SLiMs e.g. destruction box (D-box), KEN box, ABBA motif
	- destruction motifs are recognised by two proteins Cdh1 and Cdc20, which are co-activators of APC/C 
	- D-box recognised by both Cdc20 and Cdh1 while KEN box recognised preferentially by Cdh1 
	- since cdc20 itself contains a KEN box that is recognised by Cdh1, this ensures temporal degradation of Cdc20 and replacement by Cdh1 as the APC/C co-activator 
	- Some APC/C target proteins contain only D-box, or only KEN-box, or both 
	- ABBA motif binds CDC20/CDH1, and is found in a number of cell cycle proteins 
		- In yeast ACM1, ABBA acts cooperatively with KEN, D-box motifs to inhibit APC 
			- ACM1 may function similarly to BubR1 
- However statistical occurrence of destruction motifs is high, yet not all proteins with destruction motifs are degraded 
- Small variations in SLiM sequence and combinations of sequences can alter affinity of interactions 
- SLiMs can also be post-translationally modified to change their recognition by APC/C; this enables competition between substrates and regulators 

Proteins in cell cycle must be degraded as soon as they have completed their function 

**Regulation of localisation**
SLiMs regulate the timing of protein-protein interactions, involved in phosphatase recruitment, kinase recruitment, and ubiquitin-ligase recognition. .

**SLiM motif cooperation**
- In some cases just 1 linear motif interaction is sufficient for the function 
- Cooperativity between several motifs can also increase overall affinity between two interacting partners 
	- Independent moderate affinity interactions
	- Or binding to one increases affinity for another 

**slim competition** 
- regulatory enzymes to compete to modify same residue within the SliM
- Different PTM states of motif could then bind to different interacting domains 

**14-3-3 proteins**
- optimal binding motifs Rxx(pS/pT)xP 
- control G2-M checkpoint 
- Entry into M phase triggered by increase in activity of CDK1-cyclin B, and this requires removal of inhibitory Y15 and T14 phosphorylations generated by Wee1 and MYT1 respectively 
	- Phosphorylations removed by CDC25 phosphatases 
	- CDC25 activates by phosphorylation by PLK1, MAPKs, CDK1 via positive feedback loop 
	- Inhibitory phosphorylations added in response to DNA damage by CHK1, CHK2, MK2 
	- 14-3-3 mediates CHK1, CHK2 and MK2 dependent suppression of CDC25 
	- 14-3-3 binds to CDC25C that has been phosphorylated by these kinases to induce reduction in CDC25 phosphatase activity, and sequesters it away from phospho-inhibited CDK1-cyclin B 
	- 14-3-3 also binds to phosphorylated CDC25B 
- Phosphatases e.g. PP1 and PP2A can dephosphorylate the CDC25s, among other mechanisms to dissociate 14-3-3 from CDC25 to allow cell cycle to continue 




control timing
ensure unidirectionality 

how to ensure specificity of binding? 
- SLiM cooperation – combinatorial binding 
- Phosphorylation, most common PTM  

Why use SLiMs for cell cycle regulation? 
- Motifs are widespread -> many substrates can be regulated in a pleiotropic manner by cell cycle regulators, resulting in global changes in the cell required for proliferation 
- Modular nature, gives flexibility to integrate different signals