# Module A 
> [!summary]- [[A1 Mass Spectrometry and Proteomics]]
> 1. Principle
> 2. Process
> 3. Single protein applications
> 	1. Structural determination
> 	2. Protein identification
> 4. Proteomics applications 
> 	1. Protein complexes
> 	2. Quantification
> 		1. Label-free methods
> 		2. Stable isotope labelling 
> 		3. Isobaric labelling
> 	3. Protein interactions
> 		1. Cross-link MS
> 		2. H-D exchange 
> 		3. AP-MS
> 	4. Spatial localisation of proteins 
> 		1. LOPIT 
> 		2. Protein correlation profiling 
> 		3. Proximity tagging 
> 		4. Organelle enrichment/subtractive enrichment 
> 	5. Clinical proteomics 
> 5. Limitations

**Essay**
- [[Describe how mass spectrometry can be used to gain insight into multi-protein complex components.]]

**Detailed outline**
- [[To what extent is it possible to use mass spectrometry to systematically identify and study the protein complement within a cell? How might one best go about this?]]
- [[Using mass spectrometry to investigate PTMs and PPIs|How can Mass Spectrometry-based approaches be used to investigate changes in a cellular response that involve stable protein modifications and protein-protein interactions? Briefly outline complementary methods which could be used to verify the initial findings.]] 

**Outline**
- [[Protein identification-what are the strengths and weaknesses of using mass spectrometry over an ELISA?]]

> [!summary]- [[A2 X-ray crystallography]]
> 1. Principle
> 2. Process
> 	1. Expression and purification
> 	2. Crystallisation 
> 	3. Acquire diffraction pattern 
> 	4. Solve phase problem 
> 	5. Fitting of electron density map to atomic model 
> 	6. Analysis of model 
> 3. Applications 
> 	1. Protein dynamics 
> 		1. Time-resolved XRC
> 		2. X-ray free electron laser 
> 4. Membrane XRC 
> 5. Advantages and disadvantages

> [!summary]- [[A3 Cryo-EM]]
> 1. Principle
> 2. Process
> 3. Applications
> 	1. Studying difficult-to-crystallise proteins
> 	2. Studying large proteins 
> 	3. Protein dynamics (time-resolved cryo-EM)
> 	4. Studying rare proteins
> 	5. Studying structures in-situ (cryo-ET)
> 4. Membrane cryo-EM 
> 5. Advantages and disadvantages

**Essay**
- [[Cryo-electron microscopy is the method of choice for determining protein structure and function. Discuss and give pros and cons of the method.]]

> [!summary]- [[A4 NMR]]
> 1. Principle 
> 2. Process 
> 3. Applications
> 	1. Structural determination 
> 	2. Protein folding
> 	3. Protein interactions
> 		1. Ligand-observed
> 			1. Saturation transfer difference spectroscopy 
> 			2. Cross-transfer spectroscopy
> 			3. WaterLOGSY
> 		2. Protein-observed
> 			1. 2D NMR 
> 			2. H-D exchange 
> 			3. NOE
> 	4. Protein dynamics 
> 		1. Fast (ns-ps)
> 			1. 15N spin relaxation analysis
> 			2. Paramagnetic relaxation enhancement
> 			3. Molecular dynamics simulations / chemical shift predictions
> 		2. Slow (µs-ms)
> 			1. Real time 2D NMR 
> 			2. Relaxation dispersion 
> 			3. T2 measurement 
> 		3. Very slow (>ms)
> 			1. ZZ 
> 			2. HDX 
> 			3. 2D EXSY 
> 4. Membrane protein NMR 
> 5. Limitations

**Detailed outline**
- [[How has NMR spectroscopy advanced the understanding of G-protein coupled receptor signalling?]]

> [!summary]- [[A5 Preparing membrane proteins for structural study]]
> 1. Interactions of membrane proteins with membrane lipids
> 2. Challenges in structural determination of membrane proteins 
> 3. Expression of membrane proteins
> 4. Membrane mimetic systems 
> 5. Comparing methods for structural determination of membrane protein

**Outline**
- [[Explain why the outcomes of X-ray crystallography and NMR structural studies of the same membrane protein might differ and describe the possible implications of such a scenario.]]

> [!summary]- [[A6 Comparing XRC, CryoEM, NMR]]
> 1. NMR vs XRC
> 2. XRC vs Cryo-EM 
> 3. Combining methods
> 4. Other structural methods
> 	1. Atomic force microscopy 
> 	2. Electron paramagnetic resonance
> 	3. Machine learning

> [!summary]- [[A7 G proteins]]
> Ras, Rho, Arf, Ran, Rab 
> 1. Small G protein signalling pathway
> 	1. G proteins 
> 	2. GDIs
> 	3. GEFs
> 	4. GAPs
> 	5. Receptors 
> 2. Small G protein structure and function 
> 	1. Nucleotide binding site
> 	2. Switch mechanism 
> 	3. Allosteric lobe 
> 	4. Post-translational modifications
> 		1. Prenylation 
> 		2. Ubiquitination 
> 		3. Phosphorylation 
> 3. Heterotrimeric G protein signalling pathway 
> 	1. Signalling cycle
> 	2. Activators: GPCRs and GBA motif proteins 
> 	3. Effectors of Gα
> 		1. Gt and cGMP PDE
> 		2. Gs, Gi and adenylyl cyclase 
> 		3. Gq and PLC 
> 	4. Effectors of Gβ/γ
> 		1. Ion channels 
> 	5. Terminators: Gα, Effectors, RGS proteins
> 4. Heterotrimeric G protein structure and function 
> 	1. Gα
> 	2. Gβ/γ
> 	3. Lipid modifications 
> 5. Structural techniques for studying G proteins
> 	1. NMR 
> 6. Structure-based drug design for Ras 
> 	1. Direct targeting 
> 	2. Fragment-based drug discovery 
> 	3. Covalent inhibitors
> 	4. Protacs 
> 	5. Prenylation inhibition 
> 	6. GDI inhibitors
> 	7. Disruption of membrane interactions 
> 	8. Effector inhibition (RAF, PI3K, Ral)

**Essay**
- [[Discuss how G proteins are post-translationally modified. How do these modifications contribute to their functional interactions with other proteins?]]

**Detailed outline**
- [[Based on your knowledge of how G proteins function, design strategies to interfere with G protein signalling.]]
- [[Describe the unique features of the small G protein ___ compared to the canonical small G protein, Ras. How do these features of ___ allow it to function?]]
- [[Compare and contrast the small G proteins with heterotrimeric G proteins. Briefly speculate why these different mechanisms might have evolved.]]

> [!summary]- [[A8 Machinery of Translation]]
> 1. Structural components of translation 
> 	1. Ribosome 
> 	2. EF-Tu 
> 2. Conformational changes during translation 
> 	1. Initiation 
> 	2. Elongation
> 		1. Recruitment of tRNA to ribosome
> 		2. Decoding 
> 		3. Post-GTP hydrolysis 
> 		4. Accommodation 
> 		5. Peptidyl transfer 
> 		6. Translocation 
> 		7. Recycling of EF-Tu 
> 	3. Structural techniques to study elongation 
> 	4. Termination
> 	5. Ribosome recycling 
> 3. Fidelity of translation
> 	1. Errors in protein biosynthesis 
> 	2. tRNA charging 
> 	3. Decoding 
> 		1. Watson-crick pairing 
> 		2. Interaction between codon:anticodon helix and decoding center nucleotides 
> 		3. Distortions in tRNA body beyond anticodon 
> 		4. Accommodation 
> 		5. Kinetic proofreading 
> 4. Chemistry of peptide bond formation 
> 5. Protein biogenesis factors 
> 	1. Chaperones
> 	2. Localisation  

**Essay**
- [[Explain why energy is needed in translation.]]

**Detailed outline**
- [[Discuss how the ribosome might be considered an allosteric assembly.]]
- [[What are the outstanding mechanistic questions in understanding the translation of mRNA into protein, and how would you go about solving them?]]
- [[How have structural data of various conformations of elongating ribosomes advanced our understanding of translation?]]

> [!summary]- [[A9 Protein Engineering (NEW)]]
> 1. Enzyme catalysis 
> 2. Rational design 
> 3. Directed evolution of RNA 
> 	1. Ribozymes 
> 	2. SELEX
> 	3. Tagged substrates 
> 4. Fitness landscapes 
> 	1. Directed evolution is navigation of a fitness landscape 
> 	2. Features of fitness landscapes
> 	3. Effect of epistasis on landscape 
> 5. Strategies for directed evolution
> 	1. Selecting a starting sequence 
> 	2. Generate a DNA library 
> 		1. Random methods 
> 		2. Directed methods 
> 		3. Recombination methods 
> 	3. Stepwise vs multiple mutations
> 	4. Screening methods 
> 		1. Functional assays
> 		2. Phage display 
> 		3. Colony screening 
> 		4. Ribosome display
> 		5. mRNA-puromycin fusion/mRNA display 
> 		6. Droplet screening 
> 			1. In-vitro compartmentalisation 
> 			2. Compartmentalised self-replication (CSR)
> 			3. Microfluidics 
> 	5. Solving local optima problem 
> 6. Strategies for specific goals
> 	1. Novel enzymatic activity 
> 	2. Increasing specificity 
> 	3. Improving stability 
> 7. Natural selection vs directed evolution 

**Outline**
- [[Outline different approaches to generate functional proteins with novel properties and discuss any difficulties likely to be encountered]]

## Broad questions
- Comparing different methods – mostly, is NMR and X-ray crystallography redundant? 
- Understanding both structure and dynamics, not just either 
- Combining techniques to get a holistic understanding of protein **structure** AND **function** (and possibly other single molecules)

**Detailed outline**
- [[Discuss how the structures of dynamic proteins and assemblies can be studied. What functional insights can be gained from such studies?]]
- [[What methods can be used to study single molecules in biology? Using examples, outline some of the advantages of such approaches.]]
- [[How would you identify and characterise the components of an extracellular-ligand-dependent signalling pathway that activates transcription factors?]]

**Outline**
- [[What is the experimental evidence for internal motions in proteins? How do these motions contribute to an enzyme’s function?]]

# Module B 
**Be careful about differentiating pro- and eukaryotic systems! 

> [!summary]- [[B1 Transcription]]
> Contents

**Essay**
- [[At which locations along the gene length does RNA polymerase pause? Why and how?]]
- [[Discuss the roles of the carboxyl-terminal domain (CTD) of the large subunit of RNA Pol II in gene expression.]]
- [[How do transcription factors find their target sites in the genome?]]

**Detailed outline**
- [[Describe the components of the transcription pre-initiation complex and their roles.]]
- [[How does the “promoter escape process” limit the rate of transcription by RNA polymerase II?]]
- [[Bacteria, yeast and higher eukaryotes regulate transcription in different ways. Discuss the similarities and differences in these regulatory mechanisms and their functional significance.]]
- [[Investigate possible roles of a protein with an RNA binding domain|You have identified a mouse protein that you suspect to have a role in gene expression. The sequence of the protein contains a predicted RNA binding domain. Describe how you would set about investigating the possible roles of the protein in gene expression]]
- [[Characterise a transcript and its regulation|You discover an uncharacterized transcript originating from a new transcriptional start site. How would you initially characterize the new RNA and the regulation of its expression? Which experimental techniques and computational analyses would you employ in your study?]]

-> for transcription essays, come up with an experiment that could be used to answer unanswered issues

> [!summary]- [[B2 RNA localisation]]
> Contents

**Essay**
- [[What methods can be used to detect and quantify mRNA? What would govern your choice?]]

> [!summary]- [[B3 Chromatin structure and gene expression]]
> Contents

**Outline**
- [[Discuss the factors that can affect expression of transgenes in transgenic organisms.]]

> [!summary]- [[B4 RNA splicing]]

**Essay**
- [[Which mechanisms determine the assembly of splicing complexes at regulated splice sites?]]
- [[How have in silico (computational) approaches contributed to our understanding of pre-mRNA splicing and its regulation]] (from AZ)

**Detailed outline**
- [[Discuss insights into regulation of tissue-specific AS enabled by global profiling techniques]]
- [[What arguments support or oppose the biological importance of alternative splicing?]]
- [[Discuss the functional importance of properly regulated alternative pre-mRNA splicing.]]
- [[Discuss the evidence that helped to address the problem of the “information deficit” in mammalian consensus splice site sequences]]

**Outline**
- [[How is neuron-specific inclusion of alternative exons in mRNA achieved?]]

> [!summary]- [[B5 Translation and translation regulation in eukaryotes]]
> 

**Essays**
- [[The 3’UTR of mRNAs provides diverse opportunities for regulation of eukaryotic translation. Discuss, using experimental examples]]

**Detailed outlines**
- [[Discuss the evidence for the “closed loop model” of an efficiently translated mRNA, and briefly comment on the implications this model poses for the regulation of translation.]]
- [[Discuss how the formation of RNA-RNA duplexes can be used to regulate gene expression in eukaryotic cells.]]
- [[Discuss how structural biology has advanced our understanding of translation initiation]] (from CS)

**Outlines**
- [[Which features of the 5’ region of an eukaryotic mRNA determine its translational efficiency and, given an mRNA sequence, how might you test that efficiency?]]

> [!summary]- [[B6 Cytoplasmic RNA decay]]
> Contents

**Detailed outlines**
- [[Discuss the current model of deadenylation in eukaryotic organisms, focusing on the experimental evidence underpinning the model]]

> [!summary]- [[B7 Small RNAs]]
> Contents

> [!summary]- [[B8 Nuclear-organelle coordination]]

**Detailed outlines**
- [[How is gene expression in the mitochondria regulated?]]

**Notes**
- [[Discuss the mechanisms that coordinate gene expression between mitochondria or chloroplast and the nuclear genes encoding organelle genes|Discuss mechanisms for coordinating gene expression between mitochondria/chloroplasts and the nucleus/nuclear genes encoding chloroplast/mitochondrial genes]]

> [!summary]- Phase separation
> Contents

**Outline**
- [[How do phase separation and biomolecular condensates contribute to translational regulation?]]

## Experimental techniques
The topics also cover different experimental techniques for investigating
- DNA-DNA interactions
	- [[B3 Chromatin structure and gene expression#Techniques for determining long-distance interactions between non-contiguous regions of DNA]]
- DNA-protein interactions 
	- [[B1 Transcription#Studying transcription]]
	- [[B3 Chromatin structure and gene expression#Using ChIP-seq to study histone modifications]]
- Protein-RNA interactions 
	- [[B3 Chromatin structure and gene expression#ChIRP for identifying ncRNA interactions]]
- Detecting and profiling mRNA 
	- [[B2 RNA localisation]]
- RNA splicing 
- Protein-RNA interactions 
- With invention of highthroughput sequencing, global profiling methods and omics methods are now important 

# Module C (core) 
> [!summary] [[C1 ER Targeting and Integration]]
> Targeting proteins to the ER 

**Essay**
- [[Discuss the mechanisms that ensure the correct selection of proteins to be targeted to the endoplasmic reticulum for membrane insertion or secretion.]]

**Detailed outline**
- [[What mechanisms in the secretory pathway facilitate efficient quality control in protein folding and protein targeting?]]

**Outlines**
- [[Many different types and topologies of integral membrane proteins are inserted into the endoplasmic reticulum. How well do we understand these processes of membrane protein targeting and insertion?
- [[Which are the most important aspects of protein glycosylation pathways in the endoplasmic reticulum that remain incompletely understood?  How would you go about characterising these aspects of the pathway?]]

> [!summary]- [[C2 PTMs and Protein trafficking]]
> 1. Protein folding and quality control in the ER
> 2. Anterograde transport from ER to Golgi involving COP II
> 3. Retrograde transport from late to early Golgi and from Golgi to ER involving COP I
> 4. SNARE target vesicles to destination membranes 
> 5. Clathrin-mediated endocytosis
> 6. Endosomal pathway 

*Diagrams would be good for this 

**Outlines**
- [[What mechanisms control the selectivity of protein targeting within the endocytic and secretory pathway?]]
- [[To what extent can traffic through the secretory and endocytic pathways be described in terms of a stable compartment model or a continuous maturation model?]]

> [!summary] [[C3 Cytoskeleton and Organelle Movement]]
>

**Essay**
- [[How do cells signal to the actin cytoskeleton?]]

# Module C (Branch 1)

> [!summary]- [[C5 Lipid metabolism in plants]]
> Contents

**Outline**
- [[What factors are likely to determine the chemical diversity found in the fatty acids present in plant seeds? How might these factors impact on biotechnological efforts to make non-native fatty acids in transgenic plants?]]
- [[Discuss the metabolic pathways of fatty acid biosynthesis in plants and how they might be manipulated for bioengineering]]

> [!summary]- [[C6 Cell walls, energy, materials]]
> Contents

**Essay**
- [[Discuss how the structures and conformations of plant cell wall polysaccharides influence their function and breakdown by microbes.]]

**Detailed outline**
- [[To what extent do we understand the biosynthesis and function of hemicelluloses?]]

## Broad questions
- How does the cell establish and maintain the identify of different cellular organelles and compartments? 
- Experimental techniques for understanding complex systems
	- [[How does reconstitution from purified components help us understand how cells work?]]
	- Methods and approaches to identifying and characterising protein complexes in cells
	- Methods to understand actin cytoskeleton regulation 
	- How can we characterise less understood aspects of protein glycosylation pathways in ER? 
	- Contributions of light microscopy to understanding the cell 
- Comparative biology 
	- What can eukaryotes do that prokaryotes can't, and why? 
	- Value of comparative biology in biochemistry 
- Protein folding and sorting from ER have high error rate 
- Discuss molecular variety present on external face of plasma membrane of eukaryotic cells 
- If evolution stopped in the microbial world, would that be good or bad? 

# Module D 
## Mutations and genomic instability

> [!summary]- [[D2 DNA damage repair]]
> - Types of DNA damage
> - DNA damage repair pathways
> - Avoidance of replicative senescence

> [!summary]- [[D3 Oncogenes and Tumour Suppressors]]
>  - p53 
>  - Myc 
>  - Oncogene cooperation 
>  - Why have proto-oncogenes not been eliminated via evolution?
>  - How many mutations are needed for cancer? 

**Detailed outline**
- [[How many mutations are needed in cancer?]]
- [[Cell-cell communication pathways are the targets of somatic changes driving cancer development and not individual tumour suppressor- or onco-genes. Discuss.]]
- [[Given that proto-oncogenes carry liabilities for cancer, why have they not been eliminated from the genome of multicellular organisms?]]

> [!summary]- [[D7 Small RNAs and cancer]]
> - Is cancer an epigenetic disease 
> - How can small RNAs at as oncogenes/tumour suppressor genes

**Essay**
- [[How might miRNAs act as oncogenes?]]

**Detailed outline**
- [[Cancer is an epigenetic disease. Discuss.]]

## Intrinsic hallmarks 
> [!summary]- [[D1 DNA Replication]]
> - Licensing 
> - Mcm2-7 replicative helicase 
> - Separation of licensing and initiation 
> - Role of CDK 

**Detailed outline**
- [[Describe in molecular detail how DNA replication licensing and initiation are completely separated within the cell cycle]]

> [!summary]- [[D4 Apoptosis]]
> - What determines whether a cell lives or dies 
> - How do viruses evade apoptosis 
> - Importance of apoptosis to cancer development
> - Avoidance of apoptosis 
> - Inducing apoptosis in other cells that might resist the cancer (e.g. immune cells)

> [!summary]- [[D5 Cell Cycle]]
> - SLiMs
> - Ensuring unidirectionality, all or nothing
> - Role of SCFs
> - CDKs 

**Essays**
- [[How do we know that the cell cycle results from alternating high and low Cdk activities?]]

**Detailed outline**
- [[What mechanisms ensure that the cell cycle goes in only one direction?]] (source: MZ)

**Outline**
- [[What part do small linear motifs (SLiMs) play in the control of the cell cycle?]]

> [!summary]- [[D6 Vertebrate somatic cell proliferation]]
> Contents

**Outline**
- [[What determines whether or not mammalian cells proliferate and whether they live or die]]

> [!summary]- [[D11 Cancer Stem Cells]]
> - Self renewal 
> - CSC hypothesis

- Changes in cell metabolism 
- Loss of attenuation of intracellular signalling 

## Extrinsic processes  
- Altered cell-cell communication pathways 

> [!summary]- [[D10 Tumour Viruses]]
> - How to establish that a viral infection causes cancer? 
> - Mechanism of action 

> [!summary]- [[D12 Invasion and Metastasis]]
> - Factors determining metastatic potential 
> - Signalling

**Essay**
- [[There are still no effective preventions or treatments for metastasis of most solid tumour cancers.  Discuss why it is so challenging to develop these.]]

**Detailed outline**
- [[How would you determine the metastatic potential of a primary tumour?]]

> [!summary]- [[D13 Tumour microenvironment]]
> - Cell types

**Detailed outline**
- [[Discuss whether cancers are aberrant tissues rather than aberrant cells.]]

## Techniques to understand cancer 

> [!summary]- [[D8 Cancer Genomics]]

**Essay**
- [[Whole genome sequencing (WGS) of cancers has been implemented into the NHS for cancer management. Explain how next-generation sequencing has permitted the revolution of WGS. What are the new insights that have been gained through WGS?]]


> [!summary]- [[D9 Experimental Models]]
> - Advantages/disadvantages of each system 
> - Cancer hallmarks that can be modelled in vivo vs in vitro 
> - Applications
> - Examples

**Essay**
- [[How can the study of laboratory organisms inform us about cancer in humans?]]

**Outline**
- [[The only valid experimental model for understanding cancer is the human patient. Discuss.]]

- Investigate how chromosomal instability drives cancer development 

## Cancer therapies 
### Broad questions 
**Detailed outline**
- [[What is the best cancer target and why, the tumour cells or their microenvironment?]]

**Outlines**
- [[Why is cancer so difficult to cure?]]
- [[How do you think cancer will be treated in 10 years' time?]]

### Specific therapies 

> [!summary]- [[D14.1 Drug-based cancer therapies]]
> Contents

**Detailed outline**
- [[Activation of the Ras pathway is implicated in most cancers. Why has Ras been difficult to target pharmacologically?]]

> [!summary]- [[D14.2 Antibody therapies]]
> Contents

> [!summary]- [[D14.3 Immune therapies]]
> Contents

- How can properties of cancer be exploited in therapy? How do they limit therapy? 
	- Metabolism 
	- Small RNAs 
	- Microenvironment 
	- Self-renewal 

## Points of view 
**Detailed outline**
- [[Discuss the notion that cancer is simply aberrant regulation of normal processes.]]
- Is genomic instability a cause or consequence of cancer? 
- Discuss the proposition that pathways rather than genes are the target of mutation in cancer.