# Base excision repair: A critical player in many games

Author: Wallace, S.S
Note type: Source
Project: Base editing
Source type: Journal

# Base repair enzymes

## **Types of glycosylases**

First enzymes in BER pathway 

- 4 for removal of mispaired uracil, thymine — all monofunctional
- 6 for repair of oxidative damage — bifunctional
- 1 for removal of alkylated bases — monfunctional

## **Monofunctional DNA glycosylases**

- Mammalian DNA glycosylases that recognize base damage have homologs in bacteria; human enzymes can complement repair defecets in bacteria
- Alkyl guanine glycosylases/methyl purine glycosylase (AGG/MPG) recognizes alkylated purines, ethenopurines
- Uracil DNA glycosylase removes uracil, 5-hydroxyuracil, other uracil derivatives in ss and dsDNA
- Single-strand-selective monofunctional uracil glycosylase 1 (SMUG1) recognizes same substrates as UNG in ss and dsDNA, including 5-hydroxymethyluracil (5-hmU)
- Thymine DNA glycosylase (TDG) removes uracil, thymine, 5-hydroxymethyluracil **when paired with guanine** in dsDNA
- Methyl CpG binding domain protein 4 (MBD4) recognizes uracil, thymine, 5-hydroxymethyluracil **when paired with guanines in CpG dinucleotides**
- MUTYH removes adenine when misincorporated opposite 8-oxoG or FapyG

## Bifunctional glycosylases

- OGG1 repairs 8-oxoG, FapyG paired with cytosine
- NTH1 repairs oxidized pyrimidines and formamidopyrimidines
- Nei-like proteins NEIL1 and NEIl2 remove oxidized pyrimidines and formamidopyrimidines in both ssDNA and dsDNA, with NEIL2 preferring ssDNA
    - NEIL1 is associated with the replication fork — acts as a "cow catcher" to remove oxidized DNA lesions
    - NEIL2 prefers lesions in ssDNA and bubble structures
- NEIL3 recognizes a wide spectrum of oxidized pyrmidines and formamidopyrimidines, but has weak glycosylase activity on dsDNA, and weak lyase activity on all substrates
- All NEIL proteins best recognize further oxidation products of 8-oxoG, i.e. spiroiminodihydantoin (Sp) and guanidinohydantoin (Gh)

## APEs

### **APE1**

- Has 3→5 exonuclease, 3' phosphodiesterase activity, and AP endonuclease activity
- N-terminal contains a redox regulatory region

### **APE2**

- Weak AP endonuclease activity
- Strong 3' phosphodiesterase and 3→5 exonuclease activity

### Pol β

- Main repair polymerase that mediates nick sealing in humans

# Handoff mechanism

- Repair occurs by a handoff mechanism
- Glycosylases remain bound to AP site product until displaced by APE1
    - This step is rate limiting for most mammalian glycosylases
- Subsequent reactions are coordinated by PARP1 and/or XRCC1 with Pol β and DNA ligase
    - Immunoprecipitates of XRCC1 can completely repair AP sites
- Glycosylases and APE1 may also interact with downstream enzymes — immunoprecipitates of NEIL1 and 2 also contain PNK, Pol β, LigIIIα, XRCC1, PCNA, FEN1
- UNG2 shown to associate with APE1, Pol β, XRCC1, PNK, replicative polymerases, ligase 1, protein cyclin A

# Mechanism of base excision

- All glycosylases that recognize damaged bases evert damaged base out of DNA into a substrate binding pocket, then cleave the N-glycosyl bond
- APE1 also everts abasic site out of DNA helix for catalysis
- Despite this manipulation of the DNA, the overall B-form of the DNA structure is maintained by insertion of "void filling" amino acid residues that stand in for the extruded base and interact with the opposite base

# Search for lesions

- Glycosylases appear to use sliding and distributive interactions to search for lesions
- Do DNA glycosylases dissociate from DNA after finding a lesion, or does it continue on the same molecule to find a second lesion?
    - At low salt concentrations, bacterial Udg, MutY, and Fpg glycosylases seem to slide several hundred bp before dissociating from DNA
    - Human AGG hops over a protein bound to DNA to find second lesion
    - Human Udg can hop over nicks and gaps
- OGG1 moves in bidirectional and random manner; no chance observed as salt concentration varied ⇒ OGG1 does not hop on DNA
- Bacterial Fpg, Nei, Nth had bidirectional, random, and highly redundant movement
    - These enzymes use a wedge residue to look for damage
    - Enzymes observed to stop at a frequency proportional to number of damages in DNA molecule
- DNA glycosylases may rotationally scan DNA and randomly spot check for damage, slowing down while doing so; once it finds a lesion, it stops to remove it

# BER in chromatin

- Processing damaged bases by glycosylases or abasic sites by AP endonucleases depends on the helical orientation of the lesion — what is the lesion's position relative to the nucleosome dyad axis?
- Lesions that are more outward facing are more readily processed by glycosylases, APE1
- Lesions that are inward facing into the histone octamer are processed much less efficiently
    - Spontaneous, reversible, partial unwinding of nucleosomal DNA allows glycosylase or AP endonuclease to access the lesion
- Pol β is also less efficient with inward facing lesions
- However, ligase IIIα/XRCC1 efficiency is the same for outward and inward nicks
- Are there nucleosome remodelling complexes that are recruited during BER?
    - SW1/SNF reportedly increases efficiency of initial BER steps
    - ISW1, ISW2 reportedly enhance uracil excision from nucleosomes

# Regulation of BER in eukaryotic cells

- BER enzymes are regulated at multiple levels by different signalling pathways
- Complex regulation is required as there are external stimuli and variations in internal stimuli
    - E.g. ROS causes DNA lesions that are repaired by BER, but they are also important for different signalling pathways
- BER enzyme transcriptional regulation is currently unclear
- BER glycosylases have tissue-specific expression
    - e.g. NEIL3 is only expressed in stem cells, blood forming organs, and during embryonic development
- BER glycosylase expression may change throughout the cell cycle
    - E.g. UNG2 is upregulated in the S phase
    - TDG is expressed the most in G1
    - MUTYH upregulated in S phase
    - NEIL1 upregulated in S phase
    - NEIL3 only found in proliferating cells, cancer cells & during embryogenesis and is upregulated in S phase with levels maintained through G2/M
    - NEIL2 expression does not change during cell cycle
- BER glycosylases are regulated by post-translational modifications
    - OGG1, MUTYH, UNG2, MBD4 upregulated by phosphorylation
    - TDG, AGG/MPG, NEIL2, OGG1 can be acetylated
    - TDG can be SUMO-lated to alter its subcellcular localization
    - NEIL2 inactivated by acetylation
    - OGG1 upregulated by acetylation
    - APE1 can be phosphorylated at different sites
    - APE1 can be acetylated to affect its binding to Y-box binding protein (YB-1)
        - Y-box is a sequence motif found in certain promoters, enhancers
        - Acetylation level changes in response to cellular stress
    - MULE (E3 ubiquitin ligase) monoubiquitylates BER proteins when BER proteins are in excess
    - CHIP polyuibiquitylates BER proteins to mark them for degradation
    - Any Pol β, LigIII/XRCC1 not part of a BER complex is ubiquitylated by CHIP for degradation
    - UBR3 (E3 ubiquitin ligase) polyubiquitylates APE1 for egradation
- Many BER proteins may have nuclear localization sequences as they can be found in both the mitochondria and nucleus
    - NTH1, NEIL1, NEIl3, MUTYH, OGG1 have identified NLSs
    - NEIL2 does not have a classic NLS but it mainly localizes to nucleus
    - OGG1 has 7 splice variants — some localize to mitochondria and nucleus and one (OGG1β) localizes only to mitochondria
    - MUTYH has different isoforms, some which go to nucleus and other to mitochondria
    - UNG1 is in mitochondria
    - AAG/MPG localizes to mitochondria
    - Both AAG/MPG and UNG1 interact with mitochondrial ss binding protein, This prevents them from removing lesions from ssDNA, which could result in the formation of lethal AP sites and single strand breaks
    - APE2 is more often in nucleus but sometimes in mitochondria

# Base excision repair subpathways, crosspathways

- BER interacts with nucleotide incision repair (NIR), nucleotide excision repair (NER), ribonucleotide excision repair (RER), and crosslink repair
- Nucleotide incision repair: APE1 cleaves damaged DNA upstream from various oxidative damage, resulting in a 3'OH and 5'P next to a damaged nucleotide
    - In cell-fre extracts, NIR activity is predominant for 5-hydroxycytosine
    - 5' DNA strand containing damage is cleaved by FEN1
    - Subsequent repair by long patch BER
- Nucleotide excision repair (NER) linked to oxidative damage repair
    - NER distorts the oxidized base, thymine glycol
    - In the absence of CSB or XPC/XPA, 8-oxoG damage levels in DNA increases
    - XPC stimulates OGG1
    - in human fibroblasts, 8-oxoG can be removed by transcription-coupled repair which uses OGG1 with CSB, XPA, UVSSA
    - CSB stimulates a number of glycosylases
    - APE1, yeast NER can repair AP sites
- Ribonucleotide excision repair (RER) removes ribonucleotides misincorporated into DNA
    - Ribonuclease H2 (RNase H2) initiates RER and recognizes DNA/RNA hybrids
    - RNase H2 produces an incision with a 5' sugar and 3' OH, which are substrates for long patch BER (involving pCNA, replicative polymerase, FEN1 or Exo1, and Ligase I
- BER may participate in interstrand crosslink repair
    - Interstrand crosslinks may be formed endogenously by malonialdehyde, which is a product of lipid peroxidation
    - Crosslinks may also be caused by exogenous agents e.g. psoralens (forms monoadducts, interstrand crosslinks)
    - Crosslinks are repaired by NER or homology directed repair
    - AGG/MPG may participate in repair of psoralen interstrand crosslinks
    - NEIL1 can excise psoralen-induced monoadducts in dsDNA → initiates BER pathway
    - NEIL1 can also excise an unhooked interstrand crosslink remnant to initiate BER
- However, BER might actually inhibit repair of crosslinks

# BER and telomeres

- Telomere DNA is G-rich (TTAGGG repeats) → susceptible to oxidation
    - Oxidation products may inhibit DNA replication and binding by shelterin complexes that protect telomere ends from degradation
- TRF2 interacts with Pol β and induces telomere dysfunction
- TRF2 also interacts with other BER proteins
- Oxidative damage is repaired more slowly in telomeres than in non-telomeric DNA
- In mice with *Nth* -/-, telomeres had more damage foci and fragility in 20% oxygen
- In *Ogg1* -/- mice tissue and primary embryonic fibroblasts, telomeres were longer under 3% oxygen
- But in 20% oxygen, *Ogg1* -/- had shorter telomeres
- Loss of APE1 alters TRF2 binding to telomeres
- Telomeres form G quadruplex structures that may be involved in maintaining telomere structure and altering telomerase activity
    - NTH1 and OGG1 cannot remove thymine glycol or 8-oxoG lesions from telomeric quadruplex DNA
    - NEIL3 can remove both thymine glycol and the further oxidation products of 8-oxoG, Sp and Gh, from quadruplex structures
    - NEIL1 can remove 8-oxoG from quadruplex

# BER and transcriptional regulation

- Oxidative damage and recognition by BER enzymes may regulate transcription of certain genes
- In the *VGEF* gene, the hypoxic response element is enriched with oxidative modifications
    - When AP sites were introduced into the hypoxic region, reporter gene expression increased
- Recruitment of estrogen receptors in response to estrogen in MCF7 cells also involves oxidative reactions
    - Receptor recruitment is driven by a methylase LSD1 which demethylates H3-lysine 9 at both enhancer and promoter sites
    - LSD1 also initiates an FAD-dependent oxidative reaction which forms hydrogen peroxide
    - This oxidizes nearby guanines to 8-oxoG
    - 8-oxoG recruits OGG1, TOPOIIβ to the promoter that creates transient nicks → conformational change → transcriptional activation
- LSD1/BER also mediates transcriptional regulation of MYC-activated genes
    - In MYC-responsive promoters of *Ncl* and *Cod* genes, both OGG and APE1 are involved in LSD1/BER transcriptional regulation

# BER and active demethylation

- BER may alter transcription by demethylating genomic cytosine
    - TET oxidases oxidize 5-methylcytosine to 5-hydroxymethylcytosine
    - then, AID/APOBEC deaminates 5-methylcytosine to thymine, or 5-hydroxymethylcytosine to 5-hydroxymethyluracil
    - 5-hydroxymethylcytosine may be further oxidized by TET oxidase to 5-formylcytosine and 5-carboxylcytosine
    - Cytosine oxidation products are substrates for TDG, MBD4, SMUG1 which then initiate BER
    - TDG associates with a number of transcriptional co-regulators e.g. CBP/p300 (acetylates histone tails)
    - MDB4 is involved in active DNA methylation
        - removes Ts paired with Gs
        - also binds to CpG and interacts with AID/APOBEC
        - Removes 5-hydroxymethyluracil paired with G
            - Phosphorylation of MDB4allows it to act on 5-methylcytosine paired with G
    - TDG also removes Ts in T-G mispairs
    - As this results in a demethylated C, transcription is activated '
    - Oxidized products of 5-methylcytosine may be importanat for genome-wide methylation
        - During embryogenesis, genome wide demethylation occurs before programming is re-established by *de novo* DNA methylases. These methylation marks are later maintained by maintenance methylases
        - TDG removes genome-wide oxidized bases, which is followed by BER processing to restore original C
        - TDG also interacts with DNMT3a, a *de novo* DNA methylase ⇒ prevents remethylation until genome wide demethylation is complete
        - TDG is ubiquitously exprssed during embryonic development

# BER in immune response

- Somatic hypermutation is initiated by AID
    - AID deaminates cytosine to uracil in a particular trinucleotide sequence context in ssDNA, which are then fixed as C→T transitions
    - The resultant U may also be removed by UNG2 — resulting AP site is bypassed by translesion polymerase → base substitution at AP site
    - Any uracils that still remain are removed by BER
    - Mismatch repair may also result in mutagenesis as it is error prone
- Class switch recombination (CSR) is also initiated by AID
    - AID is recruited to actively transcribing switch regions to deaminate cytosines
    - Uracil is excised in either ss or dsDNA by UNG
    - The closely opposed AP sites are cleaved by APE1 → creates a DSB substrate for class switch

# BER enzyme interactions

- UNG interacts with PCNA
- MUTYH colocalized with PCNA
- NEIL1 binds to PCNA, RPA, FEN-1, WRN, etc.
- NEIL2 associates with RNA Pol II and other transcription proteins — transcription-coupled BER

# Reference

Wallace, S. S. (2014). Base excision repair: A critical player in many games. DNA Repair, 19, 14. [https://doi.org/10.1016/j.dnarep.2014.03.030](https://doi.org/10.1016/j.dnarep.2014.03.030)

