# The Jigsaw Puzzle of mRNA Translation Initiation in Eukaryotes: A Decade of Structures Unraveling the Mechanics of the Process
[Open in Zotero](zotero://select/items/@HashemFrank_2018)

**Authors:** Yaser Hashem, Joachim Frank

**Year:** 2018

**URL:** https://doi.org/10.1146/annurev-biophys-070816-034034

**Tags:** #structural_evidence 

**Abstract:** Translation initiation in eukaryotes is a highly regulated and rate-limiting process. It results in the assembly and disassembly of numerous transient and intermediate complexes involving over a dozen eukaryotic initiation factors (eIFs). This process culminates in the accommodation of a start codon marking the beginning of an open reading frame at the appropriate ribosomal site. Although this process has been extensively studied by hundreds of groups for nearly half a century, it has been only recently, especially during the last decade, that we have gained deeper insight into the mechanics of the eukaryotic translation initiation process. This advance in knowledge is due in part to the contributions of structural biology, which have shed light on the molecular mechanics underlying the different functions of various eukaryotic initiation factors. In this review, we focus exclusively on the contribution of structural biology to the understanding of the eukaryotic initiation process, a long-standing jigsaw puzzle that is just starting to yield the bigger picture.

---

- Translation initiation is highly regulated and is the rate limiting step for translation 
- Many initiation factors are assembled
- Eventually ribosome assembled at start codon on a ribosomal binding site (Kozak sequence)
- Translation is very energy intense – mobilises nearly **half** of energy production of cell 
- Eukaryotic translation is more highly regulated as there are more components of the translation machinery, larger rRNA chains 

# Ribosome
- Eukaryotic ribosomes contain RNA insertions, known as expansion segments, which can also be accompanied by extensions of ribosomal proteins 
	- Function of RNA ESs not known, but likely to have some eukaryote-specific function 
	- ESs have high level of flexibility as seen from cryo-EM studies 
	- Cryo-EM structure of mammalian 43S PIC shows that ESs 6 and 7 interact with eIF3 
		- Helix A of ES6 interact with peripheral module b-i-g of eIF3 and ES7 interacts with subunits a and c of eIF3 core 
	- We need structures of complexes formed by ribosome with different protein factors at different stages of translation 

# eIF3 
- Involved in initiation process from start to finish 
- cryo-EM, XRC and NMR have been used to obtain structures of eIF3 and its subunits 

### Structure 
- First 3D structure of eIF3 in a native pre-initiation complex obtained from EM via random conical tilt technique
- Early cryo-EM studies show how the octameric core of mammalian eIF3 is organised
	- Topology similar to proteasome lid, COP9 signalosome 
	- But too low resolution (~12Å) to reveal the exact architecture of the
- Early XRC and NMR determined structures of fragments, subunits of eIF3
	- But these fragment structure were too sparse to be used to interpret the cryo-EM maps 
- A later cryo-EM study had improved resolution due to development of direct electron detector cameras 
	- des Georges et al (2015) solved structure of mammalian 43S PIC at ~6Å, allowing for fine structure of eIF3 to be seen 
	- In this study, octameric core contacts the 40S ribosomal subunit via its left arm and head with ribosomal proteins eS1/eS26 and uS15/eS27 respectively 
	- Peripheral domains of eIF3 also made contact with rRNA helix 16 and RACK1 (a ribosomal protein) 
	- Interpretation of cryo-EM map was possible due to high resolution structures of eIF3 fragments and homologous subunits from proteasome lid, COP9 signalosome 
	- Octameric core subunits were assigned using immunolabeled cryo-EM 
- Exact composition varies between species 

### Function 
- A cryo-EM study by Hashem et al (2013) showed how eIF3 promotes dissociation of 80S into LSU and SSU and prevent their association 
	- Found that N-terminal end of eIF3c interferes with intersubunit bridge B4 between rpL30e and rpS13e 
- The cryo-EM study by des Georges et al (2015) showed that eIF3 stabilises binding of eIF2 TC to 40S subunit, by interaction between eIF3d and eIF2α-D1
- eIF3 also promotes recruitment of mRNA to SSU together with eIFG, but interaction between eIF3 and eIFG has not been observed structurally, so mechanism is not understood well 
- Cryo-EM studies show how eIF3 participates in start codon recognition (see below)

# eIF2-TC
- eIF2γ subunit is the GTPase subunit and has an XRC structure
	- XRC structure shows it is structurally homologous to EF-Tu (or eEF1A in eukaryotes)
	- Structural studies also showed eIF2γ without nucleotide or in complex with GDP or GTP 
- NMR and X-ray used to study structure other subunits, eIF2α, β 
- XRC structure of an archaeal eIF2-TC showed how initiator tRNA interacts with eIF2 
	- Initiator tRNA interacts from CCA end with eIF2γ domain II, and from its elbow with eIF2α domains D1 and D2 
	- GTP interacts with G domain of eIF2γ (unsurprisingly)
	- This study showed that eIF2 interacts with tRNA quite differently from EF-Tu despite their structural homology 
- Interaction between TC and 40S not clear 
	- Hashem et al (2013) cryo-EM structure of TC with 40S showed that TC interacts mostly with 40S via eIF2α domain D1 and initiator tRNA anticodon stem-loop (which inserts at P site)
	- eIF2γ domain III does not directly contact ribosomal helix 44 but is brought into h44 proximity 

# eIF1 and eIF1A 
- Important for start codon selection 
- eIF1 structure first solved by solution state NMR 
	- Small globular domain with two unstructured N, C-terminal tails 
- eIF1A similar in topology with globular domain and unstructured tails
	- Homolog of bacterial IF1, so assumed to bind in a similar way to SSU near A site 
- X-ray structure of eIF1 in yeast SSU solved, showing that it binds near P site on SSU 
- More recent XRC (Weisser et al, 2013) found that presence of eIF1, eIF1A on SSU causes conformational changes in SSU, mainly in A site region and upper portion of h44 
	- Could be important for function as this upper portion of h44 faces eIF2 TC – maybe the conformational change helps in recruitment of eIF2-TC? 
	- This interaction also explains why eIF1 and eIF1A exhibit binding synergy, since they both interact with h44 

# Other eIFs 
- Full length eIF3j has no available structures
	- There is 1 NMR structure containing N-terminal tail of eIF3j interacting with eIF3b-RRM domain 
	- Cryo-EM structure of 40S-eIF1-eIF1A-eIF3j complex from yeast was solved to show that eIF3j globular domain is located at intersubunit face of 40S subunit, contacting eIF1A near A site 
		- Possibly the N-terminal is projected towards eIF3b-RRM 
- eIF3g is part of eIF3 but function not well understood 
	- binds to eIF3i WD40 domain, forms part of b-i-g module, and positioned at mRNA channel entrance 
- eIF5 NTD acts as GAP for eIF2, CTD stimulates interactions between some translation factors 
	- Only has some partial structure, no full structures 
	- XRC of eIF5 CTD shows it contains a HEAT domain 
	- NMR studies by the Asano group showed how eIF5 interacts with initiation complex  
		- During scanning, eIF5 and eIF3c-NTD help to anchor eIF1 in P site, so that eIF1 maintains PIC in open conformation 
		- eIF1A-NTT binds eIF5 to prevent it from binding to eIF2β-NTT
		- Start codon recognition arrests scanning, causing eIF1 dissociation 
		- eIF5 binds to eIF2β-NTT 
		- eIF1 release promotes ejection of eIF2 bound to eIF5 

# Cap binding complex
- Structure of entire CBC has not been determined 
- CBC = eIF4F, eIF4A, eIF4B 
	- eIF4F = eIF4E cap binding protein, eIF4G scaffold, eIF4A DEAD-box RNA helicase
- CBC recruits capped mRNA to 43S PIC and unwinds 5' UTR 
	- Recruitment – eIF4G scaffold interacts with cap-eIF4E complex and eIF3-40S complex 
- XRC of eIF4E showed how it interacts with the modified guanine base in the 5' cap 
	- eIF4E stacks modified guanine base between two tryptophans 
- XRC showed how eIF4E and eIFG interact 
	- Interaction occurs via Tyr-XXXX-Leu-φ peptide segment 
- XRC structure by Gross et al (2003) showed that conformational coupling also occurs between eIF4E and eIF4G, and this drives ribosome loading onto the cap 
- XRC structure by Schutz et al (2008) of full-length eIF4A interacting with eIF4G shows how they interact 
- eIF4G has not been as structurally well-determined; only fragments have been solved in interaction with eIF4E and eIF4A 
	- XRC study showed structure of conserved HEAT domain from eIF4GII -> eIF4A/IRES binding domain 
	- XRC of C-terminal domain shows two atypical HEAT domain -> second binding site for eIF4A and Mnk1 binding site (kinase for eIF4E)
	- XRC of PABP and eIF4G PABP fragment shows how CBC and PABP interact 
	- XRC of an eIFG5 homolog (DAP5) was solved
- eIF4B promotes association of MRNA to 40S subunit and stimulates eIF4A helicase activity 
	- Full structure not known 
	- NMR study found structure of RRM and that RRM has low affinity for RNA, needing N- and C-terminus regions to bind to RNA with high affinity 

# DHX29 helicase
- Required for scanning mRNAs with highly structures 5' UTRs 
- Partial homology model of DHX29 has been proposed 
- Hashem et al (2013) cryo-EM study of 43S PIC showed general shape of DHX29 bound to SSU on h16, and that DHX29 extends to intersubunit side of SSU via a helical linker
- DHX29 and eIF3j appear to be mutually exclusive due to a later cryo-EM study by Aylett et al (2015) which showed that eIF3j and DHX29 binding sites overlap 
- RecA1 domain appears to interact with eIF3 b-i-g module mainly via eIF3i 
- but molecular mechanism of DHX29 still unclear 

# ABCE1 NTPase 
- Ribosomal recycling factor 
	- Cryo-EM solved structure of ABCE1 in context of ribosomal recycling complex 
- But also involved in initiation as an antiribosomal association factor that disrupts B3 intersubunit bridge 
	- In complex with SSU, ABCE1 has a different conformation than in recycling complex. 
		- Heuer et al (2017) (cryo-EM), Kiosze-Becker et al (2016) (XL-MS) 
		- These studies show that in complex with SSU, NBDs are fully closed, FeS domain is rotated by ~180° for binding to h44 in SSU 
		- Could function in formation of initiation complex 

# eIFs not part of pre-initiation complex 
- eIF2B – GEF for eIF2γ 
	- Activity inhibited by stress-induced phosphorylation of eIF2α
	- XRC structures of eIF2B showed molecular mechanism for nucleotide exchange, and that eIF2α phosphorylation inhibits it due to formation of a nonproductive eIF2-eIF2B complex, which blocks GDP exchange on eIF2γ subunit 
- eIF6 is an antiribosomal association factor that binds to 60S subunit 
	- cryo-EM structure of 60S-eIF6 showed it binds to sarcin-rich loop, and is posed to disrupt B6 intersubunit bridge, preventing SSU and LSU from associating 
- eIF5B – stimulates association of 40S SSU (carrying initiator tRNA in P site) with LSU 
	- Eukaryotic homolog of bacterial IF2 
	- XRC of IF2 without nucleotide, with GDP or with GTP show that it undergoes structural rearrangements upon GTP hydrolysis, involving domain IV rotation 
	- Cryo-EM of 80S-Met-tRNA<sub>i</sub><sup>Met</sup>-mRNA-eIF5B show the molecular mechanism for eIF5B action 
		- Interaction between eIF5B domain IV and initiator tRNA stabilise large-scale conformational changes 
		- tRNA recognition by doman IV, and recognition of SRL by G domain, stimulate GTPase activity of eIF5B
		- In other words, eIF5B GTPase activity is stimulated only when LSU and SSU are associated, locking in the complex 

# IRES translation 
- Structural studies have also been helpful to understand how cap-independent, IRES-based translation occurs 
- cryo-EM has been particularly important, providing first structures of IRES, and interaction between HCV IRES and SSU 
- Type 3 and 4 have had detailed structural studies, type 1 and 2 not clear 