# Methods for enzyme library creation

Author: Alejaldre, L., Pelletier, J.N., Quaglia, D.
Note type: Source
Source type: Journal

Three strategies: random mutagenesis, recombination/gene shuffling, targeted approaches

# Overview

## **Random mutagenesis**

- Use error prone PCR to introduce mutations in random locations
- Useful if structure or functional regions of protein are unknown, but unlikely to find beneficial mutation

## **Recombination**

- Fragment mutated copies of the gene of interest and reassemble them to create new combinations of mutations
- E.g. SCHEMA — guided recombination of fragments of homologoues genes

## **Targeted approaches**

- Site directed mutagenesis
- Site saturation mutagenesis (SSM) — introduces degenerate codons at a specific position
    - Single site saturation libraries — higher quality, small size, greater access to seuqence diversity, but curtails discovery of epistatic effects
- Combinatorial SSM
    - Combinatorial saturation libraries – more than one site is mutated at once and the combination of possible mutations are assessed e.g. iterative saturation mutation (ISM)
    - In ISM, sites of interests are chosen (as many as necessary) that generally
    contain 1 to 3 amino acids (or more). Each site is then randomized via saturation mutagenesis. Best hits from first round of randomization are subject to further iterations of randomization via saturation mutagenesis at other sites
        - Chocie of sites of interest is informed by prior structural knowledge
    - Combinatorial active-site saturation (CAST) is used to increase substrate specificity
    - FRISM – uses rational prediction to restrict mutants to be screened