# Optimized base editors enable efficient editing in cells, organoids and mice

Author: Zafra, M.P., et al
Note type: Source
Project: Base editing
Source type: Journal

- Cas9n DNA sequence in BE3 was not optimized for expression in mammalian cells, and it contained a large number of nonfavored codons
- Cas9n in BE3 also contained 6 polyadenylation sites (why is this bad) → may result in premature termination of transcription
- Codon optimization of Cas9n in mammalian cells increases expression of BE3 and hence base editing efficiency
- Cas9 variants with high fidelity or with alternative PAMs also had improved expression following codon optimization
- Adding a nuclear localization signal (NLS) to the N terminus of Cas9 can improve the efficiency of gene targeting
    - In particular when NLS has a FLAG epitope tag, localization to nucleus increases, which increased base editing efficiency
- The reengineered enzymes were active in multiple cell types and could also edit somatic cells *in vivo*

Zafra, M. P., Schatoff, E. M., Katti, A., Foronda, M., Breinig, M., Schweitzer, A. Y., Simon, A., Han, T., Goswami, S., Montgomery, E., Thibado, J., Kastenhuber, E. R., Sánchez-Rivera, F. J., Shi, J., Vakoc, C. R., Lowe, S. W., Tschaharganeh, D. F., & Dow, L. E. (2018). Optimized base editors enable efficient editing in cells, organoids and mice. Nature Biotechnology, 36(9), 888. [https://doi.org/10.1038/nbt.4194](https://doi.org/10.1038/nbt.4194)