**Describe the components of the transcription pre-initiation complex and their roles.**

Assumption – eukaryotes 
Describe – structural 

# Roles
* TFIID (discuss the role of TBP-associated factors in initiation) 
    * Promoter recognition 
    * TATA binding protein 
        * Recognise TATA box
    * TAFs (their role is 1 whole essay…) 
    * TRFs
* TFIIA
    * Required for activated transcription 
* TFIIB
    * Pol II
        * Specifies TSS 
            * B reader helps to select TSS 
        * Binds dock and wall domains of polymerase to recruit Pol II-TFIIF complex
        * Bridges RNA Pol II and promoter 
        * B linker contributes to DNA melting in archaea 
        * Stimulate RNA synthesis allosterically (Cramer 65) 
    * Pol III – TFIIB-related factor called BRF1 
        * Forms PIC of similar architecture 
    * Pol I – TFIIB-related factor TAF1B 
        * Binds polymerase protrusion domain 
        * Architecture of PIC with DNA is different – DNA is bent in different direction and loaded deeper into cleft 
* TFIIF
    * Binds at promoter with Pol II and B, D, E
    * Reduces non-specific DNA binding by Pol II 
    * Subunit roles 
    * Stabilises TFIIB and PIC 
    * Stimulates promoter escape 
    * It is temporarily lost from the transcription complex after initiation, but can re-associate with Pol II through transcription to stimulate elongation e.g. at paused elongation complexes 
* RNA Pol (what about other Pols? Pol I, Pol III?) 
    * RNA Pol I – large ribosomal RNA precursor
    * RNA Pol II – mRNAs and various ncRNAs
        * CTD 
            * Refer to other essay on CTD role 
            * Ser5p on CTD disrupts its interaction with mediator, facilitating promoter escape 
    * RNA Pol III – tRNA, small ribosomal RNA 
    * Plants have two additional Pol II-like polymerases, IV and V, involved in transcription of siRNA for RNA-directed DNA methylation, which causes gene silencing  
* TFII E, H 
    * Promoter escape (essay: how does promoter escape limit rate of transcription by RNAP II?) 
    * TFIIE binds non-phosphorylated CTD, recruits and stimulates TFIIH ATPase activity, stabilises open promoter complex 
        * Open promoter complex stabilised by binding DNA bubble above active center cleft 
    * TFIIE also regulates DSIF binding as it binds to site on clamp domain where DSIF binds; so loss of TFIIE may be important for DSIF (which causes proximal promoter pausing) 
    * ATP hydrolysis by TFIIH subunit, XPB 
        * Hydrolysis unwinds DNA and propels DNA into the polymerase active center 
        * There is less extensive protein-DNA interactions in Pol II PIC compared to Pol III, may be why it needs XPB to open the DNA 
        * Requirement for XPB provides another point for regulation of transcription by RNAP II
    * In RNAP I and III PICs, DNA opens spontaneously upon binding to the promoter; reason is unclear. Speculated that the promoter contains DNA sequence features that increase meltability. So TFIIE, H not necessary. 
    * TFIIH also involved in DNA nucleotide excision repair pathway, thus coordinating DNA repair to transcription 
        * TFIIH contains XPB and XPD which are helicases involved in DNA repair. It opens the DNA for excision of damaged nucleotides. 
            * Coin et al (2007) – ATPase activity of XPB and helicase activity of XPD drive opening. XPB is not used as a helicase. TFIIH in patients with mutated XPB could not induce DNA opening. 
                * Mutation in helicase motifs of XPB did not affect TFIIH ability to participate in NER 
        * CAK subcomplex regulates cell cycle transition from G2 to M phase by activating phosphorylation of several kinases e.g. CDK1, 2, 4, 6 
        * Mutations in XPB, XPD, p8 subunits lead to disease e.g. xeroderma pigmentosum (sensitivity to UV due to inability to repair UV-induced DNA damage) 
* Mediator complex – regulates Pol II transcription initiation and elongation 
    * Mediator is a bridge between TFs and PIC, so could be a regulatory hub to coordinate signals to transcription 
        * Tail binds activating TFs 
        * Contains IDRs, allowing it to interact with many proteins and phase separate
        * Undergoes conformational changes
    * Structure
        * Contains a conserved core that contact Pol II, TFIIB, and TFIIH
        * Peripheral components can differ between species 
        * Mediator complex may be bound to a Mediator kinase module to form a larger CDK-Mediator complex 
    * Role in elongation
        * Stimulates TFIIH kinase subdomain to phosphorylate CTD of Pol II to facilitate transition to elongation 
        * E.g. Depletion of Mediator CDK8 slows Pol II elongation rate at gene bodies 
        * E.g. MED14 depletion (by degron) reduced Pol II transcription globally 
    * Role in initiation 
        * Dynamic model 
        * Mediator and TFIID work together to position TFIIH in the PIC of Pol II 
            * TAF2 contacts p8, p52 subunits of TFIIH to position XPB for promoter opening 
            * Mediator contacts XPB 
            * Mediator positions CDK-activating kinase module TFIIH (including CDK7, cyclin H, MAT1) so that it can phosphorylate Pol II CTD. Mediator also relieves CDK7 autoinhibition. 
        * Mediator and TFIID orient Pol II in PIC
    * Role in proximal promoter pausing in some contexts
        * E.g. Inhibiting Mediator CDK8 inhibits pause releasing during IFN-gamma activation 
    * Role at enhancers (metazoans only) 
        * Mediator can bind eRNA and has high occupancy at enhancers (ChIP) 
        * Clustering of TFs and Mediator via IDRs around enhancers (particularly superenhancers), promoters can promote LLPS 
            * Mediator and TFIID have limited diffusion in yeast nuclei -> restricted to specific regions? 
        * Mediator connects TFs bound to upstream activating sequences with adjacent promoters in yeast
            * This interaction may be promoted by Mediator-CTD interaction 
* RNAP I PIC only
    * TIF-IA – regulates PIC assembly. Phosphorylation prevents binding to Pol I, thus repressing transcription 
* RNAP III PIC only

**Include diagram** 

# Comparison of Pol PICs


<table>
  <tr>
   <td>Pol I 
   </td>
   <td>Pol II
   </td>
   <td>Pol III 
   </td>
  </tr>
  <tr>
   <td colspan="3" >PIC core: promoter DNA, RNA Pol, TBP, TFIIB-like factor, TFIIF-like protein, TFIIE or proteins with TFIIE-related domains 
<ul>

<li>Core stabilises open complex and initiates translation 
</li>
</ul>
   </td>
  </tr>
  <tr>
   <td>TAF1B (human)/Rrn7 (yeast) 
<p>
Rrn7 associates with Rrn6, Rrn11 to form core factor for Pol I transcription 
<p>
Rrn7/TAF1b have N-terminal B ribbon domain, C-terminal B core domain, interconnecting region containing B reader and B linker 
<p>
Rrn7/TAF1b required for polymerase recruitment to promoter. Contact polymerase dock domain via B ribbon, and may bind TBP and DNA via B core domains
   </td>
   <td>TFIIB
<p>
Similar architecture 
<p>
Required for polymerase recruitment to promoter 
<ul>

<li>B linker contributes to DNA melting in archaea 

<li>B reader helps to select TSS 
</li>
</ul>
   </td>
   <td>TFIIB-related factor 1 (Brf1) complexes with TBP and the protein B’’ to form TFIIIB 
<p>
Similar architecture
<p>
Brf1 required for polymerase recruitment to promoter; contacts same as Rrn7/TAF1b 
   </td>
  </tr>
  <tr>
   <td>A49/34.5 has similar dimerisation module as TFIIF that binds polymerase lobe 
<p>
A49/34.5 involved in Pol I PIC stabilisation, and increases rate of Pol I loading on genes 
<p>
A49/34.5 enhances RNA elongation, cleavage
   </td>
   <td>TFIIF 
   </td>
   <td>C37/53 has TFIIF-like dimerisation module that binds polymerase lobe 
<p>
Facilitates formation of open promoter complex, Pol III termination, reinitiation 
   </td>
  </tr>
  <tr>
   <td>A49 C-terminal structurally homologous to TFIIE WH domains 
<p>
WH domains also located on polymerase clamp over the cleft
<p>
WH domain  binds single stranded DNA, so it may also stabilise open promoter complex 
   </td>
   <td>TFIIE
   </td>
   <td>C82/34/31 also contains WH domains, which are also located on clamp over the cleft
<p>
C34 may also have a function in DNA opening  
   </td>
  </tr>
  <tr>
   <td colspan="3" >Peripheral components 
   </td>
  </tr>
  <tr>
   <td>Opens DNA via binding energy, no ATP required
   </td>
   <td>TFIIH – XPB subunit opens DNA, ATP hydrolysis required 
<p>
Core initiation complex may not be able to open NDA on its own due to changes in TFIIB reader/linker region compared to TFIIB-like factors in Pol I, III 
   </td>
   <td>Opens DNA via binding energy, no ATP required
<p>
C-terminal extension in Brf1 binds TBP and C34, that are involved in DNA opening 
   </td>
  </tr>
  <tr>
   <td>In yeast, TBP only associates with core factor subunits Rrn6, Rrn7, the upstream activating sequence-binding factor 
<p>
Rrn3 binds Pol I in a position that resembles Mediator on Pol II -> topological conservation of interaction sites? 
   </td>
   <td>TBP forms part of TFIID complex, and sometimes SAGA coactivator complex for some genes.
<p>
TBP binds Mediator 
<p>
TBP has a conserved role in binding TFIIB-like factors but also binds factors unique to each PIC 
   </td>
   <td>TBP forms part of TFIIIB complex, interacts with Pol III-specific factors TFIIIC and SNAPc. TFIIIC recruits polymerase and TFIIIB at promoters with gene-internal binding sites. SNAPc does the same role for promoters with gene-external binding sites 
   </td>
  </tr>
</table>

# Conclusion

* Therapeutic targets 
    * E.g. Mediator – Many oncoproteins are TFs, and TFs activate Pol II via Mediator, so if TF-mediator binding is disrupted, could block TF signalling 
* Evolutionary relationship between Pol PICs, PICs in different domains (archaea vs eukaryotes) 
    * Why need three different ones? Why plants need 2 extra Pols? 