**Discuss how the formation of RNA:RNA duplexes can be used to regulate gene expression in eukaryotic cells.**

# Processes 
- mRNA degradation 
	- Deadenylation – IRE in 3' UTR shields, miRNA promotes
	- Slicing – siRNA 
- Translation initiation control
	- miRNA – different models
	- IRES 
	- IRE in 5' uTR
	- Kozak sequence – 18S rRNA pairing and leaky scanning
	- Decoding centre
- mRNA localisation – regulation in space 
	- Bicoid – stem loop to which protein binds
- Splicing 

# Mechanism 
## Small RNAs 
- Biogenesis of small RNAs requires formation and recognition of duplexes 
- Small RNA mode of action relies on RNA:RNA duplex formation between guide and target
	- miRNAs typically bind 3' UTR with imperfect complementarity 
	- siRNAs typically bind coding sequences with perfect complementarity  

## Kozak sequence recognition by 18S rRNA
- Kozak sequence = optimal context of start AUG codon 
- Suboptimal context results in leaky scanning, where first AUG is skipped and initiation occurs at next AUG 
- Context nucleotides stabilise interactions with components of 48S initiation complex

## Long non-coding RNAs
- Secondary structures within lncRNAs important for function 
- e.g. HOTAIR secondary structure allows it to interact with binding pocket in Co-Rest and with PRC2, which are chromatin modifiers -> affect gene expression
- Form duplexes with mRNAs to influence their stability e.g. recruitment of degradation machinery 
- Base pairing with mRNA -> block access to translational machinery? 

## 5' UTR secondary structures
- Generally inhibit translation initiation 
- Unwinding is required 
- Downstream secondary structure at appropriate distance improves recognition of context-poor AUG 
- Iron response elements (IREs)

## IRES 
- Secondary structure promotes 40S binding? 

## Decoding centre
- Correct anticodon-codon binding forms RNA:RNA duplex -> structure is recognised, leading to conformational changes and EF-Tu GTP hydrolysis, thus ensuring fidelity 

## Splicing 
- Base pairing between snRNA and pre-mRNA allows for recognition of splice sites and branch point 

## RNA editing 
- mRNA editing by ADARs 