**Discuss the <span style="text-decoration:underline;">insights</span> into the <span style="text-decoration:underline;">regulation of tissue-specific alternative splicing</span> that have been enabled by <span style="text-decoration:underline;">techniques that allow global profiling of alternative splicing events</span>.**

Discuss – insights, remaining questions (What insights cannot be gained from these methods)

Explain principle behind technique at first mention** 

# Intro 
* Importance of AS 
* What is global profiling 
* Advantages of global profiling – determine which mechanisms are common, and which are unique to a particular model system, thus allowing experimentalists to work on splicing systems that are generalisable 
    * High throughput methods allow for creation of large datasets of coregulated splicing events, essential for deciphering the splicing code 
* 3 main insights: Identification of tissue-specific ASEs, what cis elements are involved in regulation of tissue-specific ASE, and what protein factors are involved 

# Body 
## Global profiling methods for identifying tissue-specific ASEs  

* Splice-sensitive microarrays
    * Examples 
        * Clark et al (2007) – discovered tissue-specific exons using exon microarrays that probe for annotated and predicted exons. Tissue-specific exons were identified from 16 different normal adult tissues, with 73% of detected genes being differentially alternatively spliced. 
        * Johnson et al (2009) – Used whole genome exon microarrays for transcriptomic analysis of 13 regions of the mid-fetal human brain found that 76% of all genes are expressed, and 2% of genes exhibited differential exon usage that was region-dependent
    * Limitations
        * Cross-hybridisation
        * Limited dynamic range 
        * Low signal to noise raio due to background
        * Not ideal for discovering new ASEs, e.g. inclusion of pseudo exons. Can identify novel ASEs to a limited extent if probes for computationally predicted exons are included on the microarray 
        * Only used for organisms w sequenced genomes 
        * Largely made obsolete by RNA seq methods 
* RNA Seq – deep sequencing. Reads are mapped back to genome, number of reads = quantity 
    * Advantages over microarrays 
        * Identify novel splicing events in a tissue as sequence reads can span exon-exon junctions 
        * Broader dynamic range than microarrays
        * Will become cheaper with advancements in sequencing technology 
    * Examples 
        * Wang et al (2008) – found 22k tissue-specific ASEs identified using 15 human tissue/cell lines. Did RNAseq and mapped reads to exon-exon junctions to find that 92-94% of human genes undergo AS. Most AS events vary between tissues 
        * Long read RNA sequencing allows for complete transcripts to be sequenced, e.g. was used to identify ~8k complete transcript isoforms of _Dscam1_
        * Quantify ASEs
            * In principle, changes in ASEs between two tissues can be quantified by comparing the number of reads mapping to exon inclusion vs skipping, or by normalising number of reads mapped to particular splice junction/exon by number of reads across gene 
                * However, for genes with lower expression, they have fewer reads, so differences in ASE frequency will result in more subtle differences in reads that are not readily detected; it has been estimated that ~200 million reads are needed to accurately quantify splicing levels in 80% of genes (in human?) 
* Focused sequencing (on splice junctions) 
    * RNA-mediated annealing, selection, extension, and ligation (50)
    * Preselection by customised capture arrays (51) e.g. Hodges et al (2007) – developed a microarray that can capture >200k exons for sequencing 
            * Developmentally-regulated splicing 20, 31 
* Mass spectrometry – for identification of tissue-specific isoforms
    * Abascal et al 2017 

## Global profiling methods can be used to identify elements of the splicing code in particular tissues 

* Splicing code = cis elements involved in regulation of splicing 
* Bioinformatic analysis of co-regulated ASEs to identify splicing code
    * Data generated by RNA-Seq and microarray experiments can be analysed bioinformatically to identify motifs and binding regulators that frequently affect splicing 
    * E.g. Castle et al (2008) used microarrays to profile 48 human tissues and screen for 4-mer to 7-mer motifs associated with ~24.5k AS exons; 143 motifs were enriched near tissue-specific exons 
        * Most frequent motifs (UCUCU, UGCAUG) are binding consensus sequences for PTB/nPTB and FOX 
* Machine learning of splicing code 
    * Barash et al (2010) – distinct combinations of features are predicting of five different tissue categories of AS: CNS, muscle, embryo, digestive organs, and tissue-independent 
    * The limitation of machine learning is that ML models are a “black box” – although it can predict splicing outcomes from features, it doesn’t tell us why they arise. This leaves the mechanism a mystery. 
    * The machine learning predictions are useful in designing new experiments but its predictions should still be validated experimentally; could particularly be a problem for splice sites that use a unique regulatory mechanism not found in the training data set 

## Global profiling methods can be used to generate factor-centric “splicing maps” of where splicing regulatory factors bind to RNA in particular tissues 

* KO/factor depletion + splice sensitive arrays (18, 25, 32-24) / RNA-seq 
    * E.g. Analysis of alterations in AS in neocortex of wild type and Nova2(-/-) mice using Affymetrix junction array identified 50 ASEs regulated by NOVA 
        * Genes affected by NOVA were mostly involved in synaptic function 
* CLIP, HITS-CLIP/CLIP-Seq 
    * HITS-CLIP = CLIP + high throughput sequencing of reverse transcribed tags 
    * E.g. Licatalosi et al (2008) used HITS-CLIP to map NOVA protein binding sites 
        * This study also revealed position-dependence of Nova effect on splicing 
    * Limitations of CLIP – 
* SELEX 
    * Define optimal binding site for an RBP in different tissues 
    * E.g. Buckanovich and Darnell (1997) – used SELEX to enrich for RNA ligands of NOVA-1. Identified long stem-loop RNA ligands, the loop region of which contains a 15bp pyrimidine-rich element essential for Nova-1 binding 
    * Advantage – effectively can screen transcriptome for optimal binding sites
    * Limitation – may be low throughput for identifying binding sites of multiple RBPs
* Quantitative proteomics 
    * Spellman et al (2007) – To identify PTB-regulated ASEs in HeLa cells, knocked down PTB and employed quantitative proteomics (LC-MS/MS) to identify the targets of PTB and nPTB. These targets were found to be upregulated or downregulated in PTB+nPTB knockdown cells via 2D DiGE. For some targets, the peptides were diagnostic of specific protein isoforms (e.g. muscle-specific TPM2 isoform was inferred to be upregulated because of peptides diagnostic of muscle-specific exon 7 and 10; this was confirmed by RT-PCR analysis) 
    * Limitation – may not capture all isoforms. Sometimes diagnostic peptides are not detected, even if RT-PCR analysis of transcriptome suggests that the corresponding exon has been included in the mRNA 
        * Spellman et al (2007) – Did LC-MS/MS of proteome from PTB/nPTB knockdown cells to study PTB-dependent alternative splicing events in HeLa cells. Peptides diagnostic of TPM2 exon 6 skipping or inclusion were not found, yet RT-PCR showed a shift towards exon 6 inclusion in the mRNA
* Bioinformatics 
    * NOVA map indicates position-dependent activity E.g. Identify YCAY motifs within 200 nt of AS exons or flanking constitutive exons and predict whether the motifs act as enhancers or silencers of splicing → create NOVA RNA map of silencers and enhancers
    * NOVA clusters within downstream intron are enhancers, while clusters in exon and upstream intron are silencers 
* Experiments to investigate the splicing code and splicing factor binding sites are complementary and feed into each other. For example, motifs identified in splicing code experiments are often also binding sites for factors that could be investigated in splicing map investigations. Conversely, identifying binding sites for splicing factors allows the splicing code to be annotated for function 

# Conclusion 

* Global profiling is a powerful tool for understanding tissue-specific splicing. Since splicing often differs between tissues, it is important that these methods continue to differentiate tissues. 
* Given the widespread nature of splice sites, “big data” methods are likely to be a mainstay of the field 
* However, there are a number of questions that appear yet to be fully answered by these global profiling techniques 
    * What are the underlying mechanisms that define the logic of the splicing code? 
        * E.g. why do FOX, NOVA proteins inhibit from upstream but activate from downstream 
    * How does phylogenetics and evolution inform these ASEs? 
        * Application of global profiling on tissue lines from different species could elucidate how AS has evolved 
    * Relevance to humans – experiments have been performed on cell lines because acquiring organ samples can be somewhat invasive depending on the organ? Unlike the genome, the transcriptome has been shown to be tissue-specific.  
* Applications in the clinic 
    * There has been a push towards personalised medicine involving acquisition of the patient’s genomic sequence to predict their likelihood of developing a disease. Perhaps the transcriptome could also be acquired? Findings on splicing code, splicing factors could be used to develop algorithms that detect pathological aberrations in the transcriptome  
    * In particular, splicing appears to be very important in generating protein diversity in neurons, with important implications for neuroscience (e.g. aberrant AS has been implicated in autism). 