**How is gene expression in the mitochondria regulated? **

# Introduction 

* The mitochondria are key organelles that perform the critical role of producing energy and metabolites for the cell. As such, gene expression is under tight regulation 
* Like regulation of genes from the nucleus, gene expression in the mitochondria is regulated at multiple levels 
* As some genes are located in the mitochondria and some are in the nucleus, there also has to be careful coordination in gene expression between the two compartments to ensure stoichiometric amounts of each subunit is synthesised 
    * Imbalanced subunit production can cause defects in oxidative phosphorylation pathway and oxidative stress

# By the nucleus 

* Transcription 
    * Nucleus contains transcription factors that regulate factors required for mt gene expression 
        * E.g. Nuclear respiratory factor (NRF)-1 is a transcription factor for many nuclear genes involved with mt respiratory function (e.g. polymerase, TFAM, TFB2M, cyt c)
        * E.g. NRF-2 is a TF for other nuclear genes involved in mt function, e.g. TFAM, TFB2M, cytochrome oxidase IV
        * E.g. Peroxisome proliferator-activated receptor gamma coactivator (PGC)-1 family alpha enhances expression of NRF-1 and NRF-2
        * E.g. PPARβ also enhances nuclear-encoded mt gene expression by stabilising PGC-1α and directly activating expression of NRF-1
            * ChIP shows that PPARβ binds to NRF-1 promoter, and overexpression increasses promoter activity in reporter assays 
* Translation 
    * Nuclear-encoded translational activators
        * E.g. Pet494, a translational activator for _COX3_ mRNA 
        * E.g. cyt c subunit translation is controlled by at least 7 different factors encoded in the nucleus. They have various functions, including stimulating mRNA binding to the ribosome, activating initiation or elongation 
        * Currently mechanism of activation is unclear, but in yeast, it appears that the activators require 5’ UTR of mRNA 
        * E.g. TACO1 (only activator in humans identified) – regulates COX1 translation, but unknown how 
    * Nuclear-encoded translational repressors 
        * ATP6/8 synthesis 
        * E.g. nuclear-encoded miR-1 (in muscle tissue) – during muscle differentiation, miR-1 localisation to mitochondria increases rate of translation 
    * In response to
        * Availability of cofactors 
            * E.g. _Bc1_ complex contains 10 subunits; only heme-containing cyt b is encoded by mitochondrial genome 
                * When cyt b is being synthesised, it interacts with Cbp3-Cbp6 complex which protects it from degradation, thus increasing synthesis rate. When cyt b acquires its first heme, it interacts with Cbp4 to form an assembly intermediate that is ready to engage in further assembly. When cyt b acquires second heme, Cbp3-Cbp6 detaches from Cyt b, and it assembles with the nuclear-encoded subunits into the _bc1_ complex. Cbp3-Cbp6 in turn continue to activate Cyt b synthesis. Hence, heme availability and nuclear-encoded subunit availability limits the rate of Cyt b synthesis; Cyt b synthesis is lowered if no additional bc1 complexes can be assembled 
                    * Nuclear encoded subunits can be adjusted, which in turn regulates mitochondrial translation of cyt b 
            * E.g. cytochrome oxidase complex 
                * E.g. Cox1 and Mss51 – Mss51 activates translation of _COX1 _mRNA, binding to 5’UTR. Cox1 assembly requires heme _a_, which is synthesised from heme _b_. Heme _b_ is required to activate Mss51. As such, in a low level of heme _b_, Mss51 is not activated, and _COX1_ mRNA translation falls, thus avoiding an accumulation of unassembled Cox1. 
        * Availability of other subunits in the complex (epistasy of synthesis) 
            * In yeast, mitochondrial translation is stimulated by the presence of nuclear-encoded subunits, this is mediated by translational activators 
                * E.g. mt-encoded Atp9 translation increases in presence of nuclear-encoded Atp1
                    * Mechanism – cytosolic multisynthase complex (AME) consists of glutamyl-tRNA synthetase and methionyl-RS, During metabolic shift from fermentation to respiration, GluRS localises to mitochondria to stimulate mitochondrial translation, while MetRS localises to nucleus to stimulate nuclear transcription by associating with RNA Pol II 
* E.g. Mss51 acts as a chaperone, interacting with Cox1 when it is synthesised. Mss51 gets trapped in an assembly intermediate via tight binding to Cox1, and this is dependent on other assembly factors. Mss51 only gets released if assembly continues. So when there is a high level of unassembled Cox1, Cox1 translation decreases. 
    * Mutation of COX14, an assembly factor, liberates Mss51 to activate translation of _COX1_ mRNA. This partially rescues Cox1 synthesis in cytochrome oxidase mutants with inhibited assembly 
    * Also has a protective function here – unassembled COX1 sensitises yeast cells to oxidative stress 
* E.g. in humans, nuclear-encoded OXPHOS assembly factor regulate mitochondrial translation rate in response to availability of nuclear-encoded subunits 
    * E.g. C12ORF62 regulates translation of CO1 
    * In the absence of nuclear subunits, synthesis of the mitochondrial subunits is stalled 
    * But mechanism is not known 
* Post-translational regulation 
    * Protein degradation 
        * If a subunit is produced in excess, it is degraded by mitochondrial proteases 
            * E.g. in _S. cerevisiae_, membrane-bound i-AAA protease degrades unassembled mt-encoded subunits of complex IV in strains lacking another nuclear-encoded subunit, Cox4 or lacking nuclear-encoed cytochrome c 

# Within mitochondria 

* Post-transcriptional regulation of mRNA stability (humans) 
    * Each gene has a different copy number of mRNA transcripts in the mitochondria, and this copy number correlates with the half-life of the mRNA,  suggesting that RNA degradation is important to regulation 
    * Leucine-rich PPR motif-containing protein (LRPPRC)-SRA stem loop-interacting RNA-binding protein (SLIRP) complex  is a general mRNA chaperone that prevents secondary structure formation in a transcript-dependent manner 
        * Complex also stimulates mt poly(A) polymerase to control poly(A) length 
        * Also inhibits mRNA turnover by degradosome 
* Post-transcriptional regulation 
    * During mtUPR, MRPP3 is degraded. MRPP3 is a subunit of mitochondrial RNAse P which cleaves 5’ ends of tRNA to release tRNA and adjoining mRNA during punctuated processing. Without RNase P, mRNAs accumulate with unprocessed 3’ ends, decreasing their rate of translation 
* Translation regulation
    * Decoding capacity of mt tRNAs
        * tRNAs in mito can be modified at nucleotide bases, which affects their folding and stability. 
    * mRNA stability
        * FASTKD2 binds coding region of ND6 transcript to promote processing, while FASTK binds 3’ UTR of ND6 to regulate stability 
* In mammals, localisation of mtRNAs to nucleoprotein structures, called mitochondrial RNA granules (MRGs)  
    * Maturation of mtRNAs comprises several step: synthesis, cleavage of 5’ and 3’ ends, modification of nucleotides, 3’ end editing, translation, degradation 
    * MRGs contain many different kinds of proteins, many involved in mtRNA processing ⇒ MRGs are RNA maturation centres   
        * E.g. G-rich RNA sequence binding factor 1 (GRSF1) binds to RNase P complex. Loss of GSFR1 disrupts mtRNA processing, as GSFR1 dissolves stable G quadruplex structures in antisense transcripts (from light-strand transcripts) that can interfere with translation of heavy chain mRNA. This exposes them to degradation by RNA degradosome 
    * Degradation of mtRNAs may occur in D-foci which contain degradosome complex 
    * Immature mRNAs could be sequestered to MRGs to avoid erroneous translation 

# Conclusion

* Mitochondrial genome heavily relies on nuclear genes for expression and regulation 
* Mitochondria-nuclear coordination contribute to many biological processes e.g. development, disease, ageing 
* However, understanding of mitochondria gene expression regulation seems rather neglected compared to that of the nucleus 
* Future questions in the field of mt gene expression regulation
    * Mechanism of action of different translational activators  
    * Function of mtRNA modifications
    * Biogenesis of mt ribosomes
    * Regulation of localisation to RNA granules 
    * Applying global profiling methods to better understand regulation 
        * Ribosome profiling 
        * Deep sequencing
            * RNA-seq 
    * Regulation is variable across species, we don’t have a good understanding of how it varies, and what the function of these variations are
        * A lot of the research has been done in budding yeast. 
    * Regulation under different conditions e.g. stress, ageing, disease 