**Which features of the 5’ region of an <span style="text-decoration:underline;">eukaryotic</span> mRNA determine its translational efficiency and, given an mRNA sequence, how might you test that efficiency?**

Translational efficiency under what contexts? 

Features of 5’ region 
* Presence of cap  
    * Closed loop model 
    * Protection from degradation 
    * Presence of cap can decrease translation efficiency during infection of certain viruses 
* 5’ UTR 
    * IRES 
        * Rare cellular IRES-like structures allow for translation when cap-dependent translation compromised 
* TOP mRNAs
    * mTORC1 activates La-related protein 1 (LARP1) that binds to TOP sequence 
    * Mechanism is poorly defined but these mRNA have a TOP sequence in 5’ UTR 
* Regulatory protein binding sites – binding prevents recruitment of 43S and scanning 
    * PABP autoregulation 
    * Iron response elements
* Length of 5’ UTR 
    * In yeast, if too short, can cause leaky scanning 
        * Genome-wide mapping of yeast 5’ UTRs showed that mRNAs with short 5’ UTRs have lower translation efficiency
    * In mammals, 5’ UTR element translation initiator of short 5’ UTRs (TISU) allows cap-dependent but scanning-independent initiation on mRNAs with very short 5’ UTRs; this is common for mRNA encoding mitochondrial proteins 
* Secondary structure in 5’ UTR 
    * Needs eIF4A to unwind secondary structure, otherwise scanning-dependent initiation is inhibited 
    * Kudla et al (2009) 
* Start codon 
    * Near-cognate start codons are less efficiently used 
        * Mismatch with tRNAi anticodon → destabilisation of 48S PIC 
    * Context of codon – Kozak sequence 
* uORFs 
    * AUG-initiated uORFs may translation initiation and thus translation efficiency for downstream ORFs of some genes, via 2 mechanisms
        * Encoded peptide stalls elongating 80S ribosome at or near uORF stop codon 
        * Or, stall ribosome from scanning to the downstream ORF stop codon 
            * Genome-wide data suggests that uORFs with AUG codons with strong Kozak consensus are more inhibitory

Test efficiency 
* Independent variable 
    * Cap
        * Decapping 
    * Sequence
        * Mutation analysis 
        * Transcriptome analysis – do certain sequences in mRNA affect translation efficiency? 
* Dependent variable – translation efficiency 
    * Translation reporting system 
    * Polysome analysis 
    * mRNA seq + ribosome profiling – ratio of ribosome footprint density to mRNA density is a measure of translation efficiency 
        * Can also get additional information about ribosome binding preference, as well as higher resolution information about ribosome behaviour along all nucleotides on the mRNA 
    * Pulse labelling of protein, mRNA 

Conclusion 
* Regulation in 5’ UTR generally rarer than in 3’ UTR 
* Algorithms to predict efficiency of translation given sequence and cellular context 