
# Overall
- read each experiment and data before starting on the question
- think about the rationale behind each experiment and what each data is saying
- read figure captions
- Be as specific as possible, e.g. oligodT binds to poly(A) tail of mRNA vs oligoDT binds mRNA, when X was added vs when X was added to Y 
- Use terminology and key words (e.g. IDRs, LLPS) – can steal from intro! 
- Always contextualise to the question 

# Question types 
## Describe the data 

> [!Example] Examples
> - What was the effect of stress on AUG-Fluc levels? 
> - What observations can you make from the data in Figure 9? (2021B)

- Make use of ALL the data. 
- Describe effect of independent variable on each treatment group AND control 
- Compare treatment groups with each other (quantitatively, if possible – can give an estimate) and with the control 
- Describe EACH panel in the figure 
- Be careful that if the question specifies a figure part, e.g. 2A, only look at data presented by 2A. 
- I think u have to be kind of anal lol like look for even the "obvious" discussion points 
	- E.g. "the procedure was successful" (like wtf idk how this is useful scientific info but ok...)
- Compare by **rows** and **columns**; within each row and column, and between each row and column. 
- Graphs – describe the key parameters 
	- Enzymes – max activity, Km, concentration at max / 50% binding, etc. 

## Drawing conclusions 

> [!Example] Examples
> - What do you conclude from the data in Figure 1? 
> - What do the data in Figure 7 suggest about the role(s) of RVB2-1 in stress granule formation (2022A)
> - Using the data shown in Figures 3 and 4 and Table 1, comment on the specific activities of PKM1 and PKM2 and their influences on glycolytic and oxidative metabolism (2019A)

- For each data point in **Describe the data**, provide an implication (marks are given for the **implications**)
- The implication should be as directly linked to the data as possible, try not to speculate! 
- What information does each treatment **on its own** tell you (including the **control**)? 
- Compare the control with the treatment – is there any difference? The presence or absence of any difference provides information. What does it say about the control, and what does it say about the treatment? need to look at BOTH. 
	- If no difference, comment on this too "e.g. expression level is the same for both", "molecule/technique behaves as expected" (be more specific/contextual where possible), etc. don't forget to comment on the "normal" stuff too!!!! 
- Make use of your own background knowledge as well as information provided in the question to explain the observations where possible 
- Then provide an **overall conclusion** that wraps up the figure overall 
	- Conclusion should say something about the **biological process**, so some kind of mechanistic link? 
		- E.g. X increases protein abundance of Y -> X upregulates expression of Y 
- Implications in combination with previous figure – you are meant to combine the conclusions from the previous figure with this current conclusion 
- Be careful not to say **CAUSE**, when only a **CORRELATION** can be drawn 
- Be as precise as possible 
- Some answers also provide justifications for the conclusions while others don't... just include it in case 
- Some answer keys speculate on mechanisms, while others discourage this... I would say just include it in case if possible
- Look at any "noise", background also 
- If the question makes reference to multiple figures, make sure to draw evidence from each of the figures mentioned 
- Make sure to address each point in the question – highlight/underline key words! 
	- E.g. for the example from 2019A, you need to comment on PKM1 AND PKM2, do not forget to leave out either!  
- Maybe follow format of observation -> mechanism -> biological significance 
- Avoid speculating beyond the data – what does it most directly show? I have a tendency to speculate on mechanisms from circumstantial evidence alone... 
- Maybe ok to speculate if it seems rly obvious lol like OH interaction -> H bonding 
- Some of the answer keys ask for supporting evidence and others don't...? Just include it just in case 
- "Level of X increased" is wrong, specify the molecule "Protein level / DNA level / RNA level of X increased" 
- If we see three variables X, Y and Z are associated, how do we know which variable is the causative one? If Z is KO'ed and Y is not affected, suggests that Z does not cause Y. 
- ok to state what u think is the obvious conclusion (even if u think not enough data is shown) but also mention other mechanisms could be possible (and suggest them if u can think of them) and the caveats (i.e. what data has not been shown to prove this conclusion)
	- TBH they don't do negative marking so even if my conclusion turns out to be too speculative they aren't gonna take away marks for it. can only gain marks! 
	- "X is the most likely explanation, but it is also possible that... To resolve ambiguity, we could..."
- Extend conclusion as far as possible – end point should be a **mechanistic** explanation 
- There's a question that asks "describe the conclusion" and it just describes the data in detail instead of giving conclusions, which is so lol so i guess describe AND give conclusion 

Observation -> proxy variable -> dependent variable -> definitive conclusion -> probable conclusion that has biological significance + other possibilities/caveats (i.e. what model / idea is the interpretation of the results supporting)

## Does the data suggest X? 

> [!Example] Examples
> - Based on the data, do you think that zingarine binds to the Thr171->Ala mutant of ZigR? (2022B)
> - A colleague uses isothermal titration calorimetry to investigate whether zingarine binds to wild type ZigR... they concluded that zingarine does not bind ZigR. Are they correct? Briefly justify your answer

- Sometimes it is not possible to answer the question based on the given data alone, in which case it is acceptable to say we don't have enough information, or that the claim is possible/not possible given the data but we can't be sure. 
- If we can't be sure of the claim given in the question, then state what we can definitely conclude from the data 
- Consider other formal possibilities that might explain the conclusion.
- "Data does not prove, but is consistent..."
- Be able to differentiate **circumstantial** and **direct** evidence! 
 
## Is the data consistent? 

> [!Example] Examples
> - Are the data in Figure 2B consistent with Figure 2A? 

- Look for similarities and differences between the datasets 

## Discuss a possible mechanism 

> [!example] Examples
> - Why do you think that ATPases play such an important role in the formation and dissociation of stress granules? 

- Can draw on both the data and also background knowledge of what you learnt in lectures 
- Avoid speculating beyond the data 
- **Consider "reciprocal roles" and "direct and indirect" roles**
	- E.g. If presence of a protein is shown to upregulate the level of another protein, could be that it stabilises it, or inhibits destabilisation; or promotes it expression, or inhibits silencers; and these roles could be mediated directly (e.g. bind to protein or bind to regulatory element) or indirectly (e.g. by interacting with other proteins that do have a direct interaction with the protein)
- As long as your mechanism is logically consistent with the data presented, it should be ok even if it is not factually correct! Such an answer would be a "formal possibility" and is still considered acceptable since we are only provided with the info in the paper. 
- What information is still missing from the current data?
- Where a direct interaction cannot be shown but there is some causative relationship, can say that "X is dependent on Y" or "X is Y-dependent" (or independent) 
	- A lot of times the circumstantial evidence given is not enough to prove the direct interactions or what proteins are involved in the interaction... but we can certainly make conclusions about how different processes are dependent on each other (i.e. causal relationships)

## Suggest an experiment / How would you validate the results
** For proteins, the protein has to be enriched via differential centrifugation first and purified with a suitable technique 

- provide detail on how the experiment will be done
	- E.g. don't just say "immunoprecipitation", specify all the info needed for someone to carry it out, e.g. use antibody against X, then secondary antibody IgG, etc. 
- For validation, suggest a different experiment that could produce the same information as the experiment they did (basically alternative ways to answer the same question) 

## What controls are needed? 

> [!Example] Examples
>- Describe the controls that would have been used in the experiment shown in Figure 3 (2019A) 

Also see [[Data handling#Experimental techniques]] for standard controls 

- Identify each independent variable present in the experiment – each needs a corresponding control, no matter the technique used 
- Positive control
- Negative control 
- Loading control – for blots

## Describe an overall model 
> [!Example] Examples
> - Propose a model of... 

- Ok to be more speculative here if they just ask you to **propose**, but your model still has to be scientifically valid based on solid background knowledge 

## Assumptions 

> [!example] Examples
> What assumptions did you make in your answer? 

- Ask yourself if your reasoning supports claim. Are there scenarios where claim would not be supported by the reasoning? 

## What are the implications for (biological process)?

> [!Example] Examples
> - What are the implications for tumour cell metabolism? (2019A)

- How is the finding significant to the biological process / fitness 

## Why was X done? 

> [!example] Examples
> - In the experiment of Fig. 2d, why was the RNA sheared into small fragments? (2017B)

- You have to explain in the specific context of the experiment mentioned 

# Missing information to look out for 
- Other possible mechanisms
	- If the fold change in one variable does not fully account for the fold change in another variable
		- Could suggest independent pathways for regulating the variable 
		- E.g. in a pathway where X activates Y, if X expression leads to 10-fold increase while Y activation only leads to 7-fold increase, could suggest that the effect of X on the activity is mediated by other pathways apart from Y 
	- Signalling pathways – unlikely that protein has only 1 target or regulator, could it be regulated or target multiple other proteins that could also have an effect? 
	- When attributing cause, add a "at least in part" e.g. X (at least in part) causes Y" – also reminds yourself that there could be another factors 
- Investigating expression levels of a subunit in a complex
	- Consider stoichiometry
		- If increase in subunit leads to increase in activity of the complex, is it that other subunits are sub-stoichiometric and that subunit is limiting factor for complex formation? Or can the excess subunit act independently of the complex? 
- Differences in other cell types 
- Other molecules that were not tested and yet were shortlisted in screens 

# Experimental techniques
- Principle of the technique
- Necessary controls
- Why is the technique done 
- How to interpret data produced by the technique 

## Structural techniques 
- why some protein domains might be challenging to obtain structures for (for all techniques)
	- Lack of crystal contacts for XRC – protein is lipid embedded
	- Difficult to obtain as a stand-alone domain 
	- Too small for cryo yet two big for NMR 
	- Flexibility
- XRC / cryo EM / NMR

## DNA gel electrophoresis
- Describing data – take note of the sizes, band intensities (though this is semi-quantitative without calibration / standard curve)

## Pull down with antibodies 
- Negative controls
	- Pull down with a non-specific IgG 
	- If pulling down on a lysate in an experimental condition, pull down on lysate in a "normal" cell (e.g. if pull down done on stressed cell, also do it on non-stressed cell)
- Positive control
	- If pulling down an enriched fraction, pull down on non-enriched fraction (i.e. lysate) to check that protein is actually present in the cell 

## Western blots 
- Western blots are usually very sus 
- Key parameters – mobility, band intensity 
- Controls
	- Loading control – use a housekeeping protein e.g. actin, GAPDH 
	- Negative control 
		- Antibody control – only secondary antibody or IgG (i.e. a non-specific antibody, to demonstrate that the antibodies used are specific). Must control for both primary and secondary antibodies 
		- Protein control – only water (to demonstrate specificity to protein)
	- Positive control
		- Protein control – Lysate 
	- Calibration – needed for it to be quantitative 
- Describing data 
	- Is antibody specific / not specific to the protein of interest 
	- What does each antibody detect / what is the identity of each band 
	- EACH lane should be explained. 
- Check if blots are clear or not 
- Sometimes there may be a disparity in size between theoretical mass of a protein and size on WB 
	- Artefact of experimental system – full length protein may have been digested by cellular proteases 
	- Truncation of full length may be biologically significant e.g. if cleavage required for its activation or due to AS 

## Northern blot
- Negative control
	- Scrambled/random probe
- Describe size of bands, intensity (semi-quantitative without calibration)

## ChIP  
- for RNA or DNA 
- Why does the RNA / DNA have to be sheared? 
	- To pull down only a specific sequence and avoid immunoprecipitation of other sequences that are joined to the sequence of interest 

## EMSA 
- Negative control
	- Nucleic acid ligand – scrambled/random DNA/RNA, heparin 

## siRNA knockdown / RNAi
- Principle
	- shRNA transcribed by RNA pol II or III, and product mimics pri-miRNA which is processed by Drosha 
	- pre-shRNA exported rom nucleus by exportin-5 
	- pre-shRNA processed by Dicer and loaded into RISC 
	- Passenger strand degraded 
	- Guide strand directs RISC to mRNA with complementary sequence 
	- In case of perfect complementarity, RISC cleaves mRNA; in case of imperfect complementarity, RISC represses translation of the mRNA 
- Negative control 
	- siRNA – scrambled/random siRNA
- Can be used to confirm the identity of the protein causing the activity 

## RNA seq 
- May not be accurate for protein expression levels!
	- A lot of times an mRNA is elevated because it's protein is lowered! 

## Protein analysis 
- IDRs -> protein-protein interactions / multivalent interactions, or could be LLPS (note these two are different! so mention both where it makes sense!)

## Colony selection / cell screening 
- Controls
	- Transfect with empty vector 

## General controls 
- Consider whether some treatment additions may have off-target effects 
	- We need to control for **ALL** components added to cells 
	- E.g. 2022A – testing if ATP availability affects stress granule formation in stressed cells. Important to know whether the component added to stress the cells, sodium arsenite, does NOT interfere with ATP synthesis. 
	- E.g. 2019B – had to test if the ionophore itself interacts with the protein by using other methods of increasing intracellular calcium 
- Having redundant methods of testing the same question can be useful because if multiple different experiments are consistent, it strengthens the evidence for the claim 
	- E.g. using mutation and drug inhibition to test the role of an enzyme 
	- When using drugs / chemicals this is especially important because there can be concerns about off-target effects of the drugs, whereas we can be certain that site-directed mutation or deletion is specific to the gene of interest 

## Fluorescence microscopy
- Look carefully for any heterogeneity in the brightness – are some spots brighter or darker than others? 
	- If yes, **describe the heterogeneity**, e.g. "X is non-uniformly distributed, with clusters at high concentration embedded in a background of X at lower concentration" or "X is concentrated in Y"
- Are the borders of structures defined? 
	- "Limits of foci are relatively well-defined, so structures are differentiated from bulk cytoplasm"
- When imaging cells / structures, ideally should image more than one 

## Lysis 
- If structures/molecule distribution/state remain after lysis, suggests they are stable 
- Sonication is a preferable method; chemical lysis could destroy structures or denature proteins 

## Cycloheximide treatment 
- Inhibits translation; shows if process is linked to active translation 

## FRAP 
- Measures rate of diffusion/exchange of non-bleached fluorophores from an external pool (e.g. structure, lipid domain) into a bleached spot (e.g. cytoplasm, membrane)
- Larger molecules have slower rates of diffusion, so FRAP would be slower (e.g. upon oligomerisation)

## Protein purification
- Membrane proteins tend to form inclusion bodies when over-expressed, resulting in a large amount that can be extracted in e.g. centrifugation 
- Ammonium sulphate -> precipitation, rapid crude purification of proteins
- Sephacryl -> size exclusion chromatography 
- Q-sepharose -> anion exchange chromatography
	- Retains proteins and nucleic acids 
- Heparin -> purify for DNA binding proteins since it mimics DNA struccture
- Superdex -> size exclusion chromatography (finer than sephacryl)
- Affinity chromatography – uses specific ligand for the protein 
- MonoS -> cation exchange -> retains proteins, esp DNA binding proteins since they have positively charged residues 

## Sorbital/glycyl betaine 
- Suppresses inclusion body formation, which increases yield of some membrane proteins  

## Circular dichroism 
- CD spectrum of **208/222 minima** -> **alpha helical** (could be TM helices?)

## Fluorescence emission 
- Aromatic residues contribute intrinsic fluorescence to the protein 
	- Excitation at 295 nm -> tryptophan 
- Changes in fluorescence emission wavelength and intensity are indicative of changes in protein conformational change 
	- If a fluorescent (i.e. aromatic) residue is in a solvent-excluded region, the emission wavelength will be blue-shifted (shorter wavelength) compared to if the protein is denatured 
	- Quenching of the fluorescence in the native protein could result in fluorescence intensity increasing after denaturation; otherwise in most denatured proteins, fluorescence intensity goes down 
- Some reagents (e.g. acrylamide) can cause extrinsic quenching of aromatic residues 

## Migration assays
- Swarm plates – depends on migrating cells metabolising nutrients as they travel, generating their own chemical gradient 
- Plug assay – plug contains attractanat or repellent so chemical gradient is externally created and not dependent on uptake of test compound

## Isothermal titration calorimetry 
- If binding is entropy driven, will not see any change in the calorimetry signal between a bound and non-bound ligand (e.g. if binding takes place in a hydrophobic binding site) bc release of energy predicated on change in binding energy 

## Fluorescence polarisation 
- Can be suggestive of oligomerisation 
	- Oligomerisation reduces rotational correlation coefficient (i.e. it tumbles less rapidly as it has larger volume), which increases polarisation due to slower tumbling in lifetime of excited fluorophore 

## Protein expression
- Promoter
	- Strong inducible promoter for overexpression e.g. T7 
	- Tissue specific promoter could also be considered
	- Constitutive promoter 
	- Regulatable promtoer e.g. Tet 
- Ribosome binding site (SD for bacteria system, Kozak sequence for eukaryotic)
- Antibiotic selection 
- Purification tag 
- Truncate hydrophobic domain to make it soluble 
- Codon optimisation -> order synthetic gene instead of PCR amplifying coding sequence 

## Sequence analysis
- GPI anchored proteins – fully hydrophobic amino acids at C terminus for GPI anchoring, N-terminal signal peptide for targeting to ER 

## Differential centrifugation 
- Fraction contents – depends on solubility and density; low speed pellets gets the most insoluble and more dense molecules, high speed pellet gets less insoluble and less dense molecules 
	- Low speed pellet -> membrane proteins, extensively DNA-associated proteins (e.g. histones) 
	- Low speed supernatant -> soluble proteins
	- High speed pellet -> more soluble membrane proteins
	- High speed supernatant -> soluble proteins 

## Protein correlation profiling 
- Can be used to determine the localisation of the protein via Western Blot if known markers are blotted as well, then match the fractionation pattern to the marker 
	- E.g. Histone -> nucleus 
	- ATP synthase -> mitochondria 
	- PFK -> cytoplasm 
	- PCS1 -> cell surface membrane 
- For the answer, indicate what fractions were obtained, what the fractionation pattern of the protein of interest suggests about its localisation 

## Radioactive Met incorporation
- Useful for low abundance proteins as it allows it to be more sensitively visualised 
- Visualises newly synthesised proteins against background of proteins already in the system 

## Measuring enzymatic activity 
- When interpreting differences in the specific activity of different enzymes in the cell, we also need to check the **relative expression levels** to ensure that concentration comparable.
	- If heterogenous sample used, this could be challenging, but could at least measure total protein abundance 
	- Activity / [enzyme] = specific activity 
- Take note of relative concentrations of substrates
- If Km is given, compare the Km to the concentration of substrate used – if concentration is lower than the Km, then enzymes not at 50% of Vmax yet, i.e. in linear range of graph (assuming MM kinetics)
- If using recombinant proteins, have to assume that tags do not affect activity or binding 
- Enzyme activity can be affected by i) efficiency of binding and ii) catalytic activity 

## Reconstitution 
- Assumption – that the reconstituted system is representative of behaviour *in vivo*

## Mass spectrometry
- Show working in calculation for each residue! 

## Amino acid properties / protein structure 
- Polar -> H bonds
- Charged -> ionic interactions, H bonds
- Non-polar -> van der waals forces, entropy-driven interaction (exclusion of water)
- aromatic -> pi-stacking 
- Pro interacts via H bond on backbone 
- To test interactions, can mutate AA or change pH (for ionic interactions)
	- For Pro, analogue is hard to find but could suggest an unconventional AA 
- Size of the aa is also important apart from chemical nature of R group since length of chain determines positioning of R group 

## PCR to check identity and quantity of nucleotides
- Use a control set of primers to a different region from the region of interest 
- Semi-quantitative – PCR has to be in exponential phase to be quantitative (i.e. not saturating)
	- for mRNA, length of poly(A) tail affects amount of intercalating dye bound 

## Deletion analysis 
- Doing point mutations would validate as deletion could affect relative arrangements of other elements (esp regulatory elements, position is important to their function)

## Plasmid transfection 
- Level of expression has to be similar to physiological level or assumed to be so 

## Synthetic Holliday junctions 
- Can be made by annealing 4 oligonucleotides
	- For detection, one oligo can be labelled, 4 are mixed and denatured, then slowly cooled to allow specific annealing 
- Mobile junctions have a homologous core of base pairs, allowing branch migration across the region so that site of junction is variable
	- Used if junction binding protein has unknown cognate sequence 
- Static junctions have a fixed junction site, contains entirely heterologous nucleotides
	- Design static junction to use sequences specific to junction binding protein 

## Yeast TAP fusion library 
- Library of all yeast ORFs fused to a TAP tag (c-terminal fusion of calmodulin binding protein-TEV-protein A), allowing for tandem purification by pulling down on protein A with IgG, elute by TEV cleavage, then pull down calmodulin binding protein with calmodulin, then elute with EDTA. 
- Multiple affinity steps enhance purification 
- Construction can be done via PCR / gibson etc, then transform PCR into yeast via plasmid for integration into chromosome (if homology arms are included) 
- all proteins of yeast genome can be screened in parallel 

## Ensemble reaction 
- Big caveat – it's an average, doesn't show mechanism for specific molecules (which might differ in sequence, structure, etc.)

# Experiments to suggest 
## Enriching for an organelle/structure 
1. (optional) Treat cells to produce desired structure
2. Lyse the cells via sonication
3. Perform differential centrifugation and collect the supernatant/pellet 
	1. idk what speeds to use, but just need a low speed (1000 g) and high speed (20 000g) – probs this works for very small or not dense particles (e.g. microsomes, stress granules), then collect pellet 

## Testing nature of molecules 
- Guanidinium HCl, urea, SDS -> denatures proteins 
- Salt (e.g. NaCl), EDTA -> electrostatic interactions 
- DTT -> S-S bonds 
- RNase sensitivity -> Accessibility of RNAs 
- 1,6-hexanediol -> liquid-liquid phase separations, hydrophobic interactions

## Determine localisation of proteins 
- GFP tag and see if co-localises with a marker for the compartment 
- GFP could even be fused to a protein fragment to test if it is the signal for localisation 
- If a signal is sequence-specific, jumbling the sequence of residues could be useful as a negative control, but if the signal just relies on the general property of the sequence (e.g. hydrophobicity), then jumbling the sequence may not change the result 

## Check protein expression 
- Western blot analysis 
- ELISA 

## Test role of protein
- Temperature sensitive mutant
- RNAi 
- CRISPR-Cas9 deletion
- Point mutations 
- Tet promoter (vary concentration)
- Genetic screens – construct library in vectors with selectable marker, isolate plasmids that confer selection, sequence inserts. 

## Confirm whether cells are still alive
- Bacteria – plate an aliquot onto LB agar plate and observe for colonies 

## Investigate ligand binding 
- Radioligand binding assay 
	- Quantify bound and unbound ligand 
	- Scintillation counter to measure radio signal intensity 
	- Scatchard plot to determine Kd and stoichiometry of binding ![[Pasted image 20230604234845.png]]

## Remove impurities / purification 
- Salts
	- Desalting column
	- Dialysis 
- Protein
	- Column chromatography
		- Affinity column
		- Ni-NTA resin column 

## Measuring protein abundance
- Western blot
- ELISA 
- RNAseq transcript quantity (*Not always correlative though)
- Mass spec (assuming good coverage, sensitivity)

## Detection of PTMs 
- Mass spectrometry after complete digestion with protease 
- GPI anchoring 
	- Phospholipase C cleavage 
- Confirming identity of modified residue
	- Do site-directed mutagenesis to Ala, then see if recombinant purified protein can be labelled with a radioactive source of PTM e.g. <sup>32</sup>P 
- NO western blot! bc wouldn't answer if that specific residue in the protein is phosphorylated, rather antibodies capture all proteins with that PTM in the lysate... 
- E / negatively charged residues mimic phosphate but may not be a perfect mimic (may be ok to assume this is the intent of the question even if they didn't say it explicitly)

## Check if most molecules identified 
- Repeat screen until no new molecules are discovered (i.e. a saturating screen)

## Investigate origin of a signal within a tissue 
- Single cell methods e.g. single cell sequencing, single cell RNA-seq 

## Studying RNA turnover 
- Need to inhibit translation via cycloheximide so that RNA level only affected by rate of degradation 

## Test more timepoints/concentrations
- Instead of testing 1 or 2, test a range 

## therapeutic strategies 
- Small molecule inhibitors – consider which parts of signalling pathway can be inhibited
- splicing modulators e.g. antisense oligos to cause exon skipping 
- gene editing
- lifestyle changes (lol) – be specific 
- cancer therapies – immuno, antibody 

## enrich for mRNAs 
- Use 5' to 3' exonuclease to digest all RNA other than capped mRNA 
- Use a depletion method for ribosomal RNA, such as biotinylated antisense oligos and streptavidin capture 

(in general enrichment can be done by a) capturing the molecule of interest or b) depleting the other undesired molecules) 

## time course 
- need to specify the time when a treatment (e.g. drug) is added during the time course 

## how many samples
- "biological replicate" is the correct term. 

