---

kanban-plugin: basic

---

## To-read

- [ ] [[KitovEtAl_2019]]<br>-   Method for identifying substrates for CAZymes<br>-   Involves mass spec (which I can’t use), but maybe could adapt some ideas for a lower throughput method?<br>-   Developing a new method would take like a few weeks though…
- [ ] [[AdesioyeEtAl_2016]] See metagenomics/bioprospecting section... may be helpful for screening??
- [ ] [[HelbertEtAl_2019]] Insight into structural features that affect substrate specificity in CAZymes -> predict specificity of ancestral GT34
- [ ] [[ChenEtAl_2020]] <br>-   Methods for identifying substrate specificity in proteases<br>-   Could derive analogous methods for GT34 characterisation
- [ ] [[LiEtAl_2020]] Another method for characterising CAZyme substrate specificity – high throughput but maybe could be adapted?
- [ ] [[SorensenEtAl_2011]] Cell wall biology of charophycean green algae. See if they have any GT34s or related polysaccharides
- [ ] [[MikkelsenEtAl_2021]] Fucosylated xyloglucan exists in charophycean green algae. Is it homologous to GT34, and if so could it represent an evolutionary origin for GT34
- [ ] [[HanEtAl_2019]] Identified putative xylosyltransferases from algae. Could they be similar to GT34s?


## Processing

- [ ] [[HoggEtAl_2003]] Information on mannanase used
- [ ] [[vonFreieslebenEtAl_2016]] Info on mannanase used
- [ ] [[YuEtAl_2022]] Previous lab work related to mannan


## Completed

- [ ] [[GoubetEtAl_2009]] Previous lab work related to mannan
- [ ] [[SchellerUlvskov_2010]]<br>General review of hemicelluloses
- [ ] [[YuEtAl_2018a]] Investigated the fine structure of galactoglucomannan in Arabidopsis seed mucilage, and activity of CSLA and MAGT involved in defining galactoglucomannan structure.<br><br>Does similar work in characterising the activity and acceptor specificity in MAGT, but in Arabidopsis. Present study will probably be modelled after this.




%% kanban:settings
```
{"kanban-plugin":"basic"}
```
%%