**How have structural data of various conformations of elongating ribosomes advanced our understanding of translation?**
= What have we learnt from structural data. 

# **Introduction** 
* Focus on elongation – most conserved between prokaryotes and eukaryotes 
* Structural data has been possible with advances in structural techniques 
* Why is it important to get structural data? 
    * Structure is primarily used to demonstrate function of each component of the elongation machinery. In this case, the focus is on the function that the ribosome plays in each part of elongation. 

# **1. The ribosome directly facilitates peptidyl transfer  **

* X-ray crystallography was used to show how the ribosome contributes to peptidyl transfer. 
    * Polikanov et al (2014) created crystal structures of the pre-attack and postcatalytic complexes of the _Thermus thermophilus_ 70S ribosome. They showed that ribosomes contain a proton wire, which couples aminoacyl-tRNA accommodation to peptide bond formation. 
    * In this model, a series of proton movements occur to activate the nucleophilic –H<sub>2</sub>N of the amino acid on the aminocyl-tRNA. 
    * In the pre-attack complex, a water molecule coordinated by nucleotides in the rRNA deprotonates an OH group on the A2451. This activates A2451 OH, such that it  deprotonates an OH group on the peptidyl-tRNA, which in turn deprotonates the H<sub>2</sub>N group on the attacking amino acid. The amino group is activated and does a nucleophilic attack into the ester C=O between the peptide and the peptidyl-tRNA. 
    * During the reaction, there is a collapse of the tetrahedral intermediate, triggering another series of deprotonation steps that restores the protonation state of all the OH groups involved, except that the peptide bond between the growing peptide and the attacking amino acid is preserved. 
    * Also, showed that rRNA is exclusively involved in peptidyl transfer, and no ribosomal proteins are directly involved.
    * What was the significance of this finding? 
* A76 2’OH also plays a role, as mutation resulted in a fall in rate of peptidyl transfer, but unclear what the precise function is. 
    * Crystal structure – A76 is positioned to participate directly in peptide bond formation 
    * Elaborate?
* Although peptidyl transfer is mostly entropy driven, such structural data shows how the ribosome plays a direct role in peptidyl transfer, activating nucleophiles to lower the activation energy of amide bond formation. The high resolution structure obtained via X-ray crystallography also made it possible to show for the first time that the ribosome deprotonates the nucleophile by activating a water molecule. 

# **2. Conformational changes in the ribosome drive translocation **

* Structural basis for how translocation occurs: It is the ribosome that moves along the mRNA, rather than the mRNA moving through the ribosome. But how does this happen?
* Translocation important to understand as it maintains reading frame and allows translation to proceed
* Following peptidyl transfer, translocation occurs to move the peptidyl-tRNA into the P site and the empty tRNA into the E site. 
    * Advantage of cryo-EM – can determine structures with good enough resolution under conditions representative of translation in vivo 
    * Cryo-EM studies of translocation complexes showed that during translocation, the ribosome exists in a hybrid state where the 30S subunit is rotated with respect to the 50S subunit.  This allows the 3' ends of A-site peptidyl tRNA and P-site deacylated tRNA to move to the P and E sites of 50S subunit, while anticodon ends and mRNA remain anchored in 30S subunit, thus poising the elongation complex for translocation. 
    * Other cryo-EM studies and FRET studies have also showed that EF-G in its GTP-bound form can then bind to the ribosome A site, stabilising the rotated form of the ribosome.
    * Binding stimulates EF-G GTPase activity. GTP hydrolysis is coupled to conformational changes in the ribosome:  It translocates 1 codon along mRNA in 5’ to 3’ direction, resulting in mRNA and anticodon ends of peptidyl-tRNA transferred to P site while deacylated tRNA moves to E site  of 30S subunit, and the ribosome undergoes reverse rotation. This restores the ribosome from the hybrid state to its classical state.   
        * Previous studies on translocation used cryo-EM, which were not high resolution enough to fully understand how the ribosome stimulates EF-G GTP hydrolysis. Gao et al (2009) produced the first crystal structure of a mutant ribosome in complex with EF-G in the post translocational state, using fusidic acid to trap the EF-G in a conformation between GTP and GDP-bound state. In particular, the L10-12 stalk and L11 region have been found to interact with EF-G to facilitate GTPase activation and phosphate release. 
* Time resolved has been useful in demonstrating the conformational states in the sequence they occur 
    * Carbone et al (2021) 
        * 
* The fact that the ribosome has to undergo these structural changes demonstrates a role for EF-G, which stabilises the hybrid state and then catalyses the ribosome’s return to the classical state. 

# **3. The ribosome contributes to translation fidelity directly and through interactions with EF-Tu **

* Insert context for initial selection and proofreading steps to allow reader to understand significance of structural data in understanding these steps 
* The ribosome contributes to translation fidelity mainly by participating in initial selection. This is the stage of elongation where the tRNA anticodon is first brought to the mRNA codon in the A site by EF-Tu. 
    * In this stage, codon recognition occurs, where the decoding center nucleotides of the ribosome senses whether the codon and anticodon match. 
    * Ensembles of cryo-EM structures as determined by Loveland et al (2017) and others have revealed the series of conformational changes within the ribosome that allow for discrimination between cognate and near-cognate codons. 
    * If the codon:anticodon helix is correct, then the decoding center nucleotides G530 and A1492-A1493 of the 16S rRNA undergo a conformational change to interact with the minor groove of the codon:anticodon helix at the first and second position. G530 is responsible for forming the hydrogen bond with the codon:anticodon backbone to stabilise the helix; this does not occur for near-cognate tRNA. At the same time, A1492 and A1493 insert into the minor grooves of the helix and form hydrogen bonds with the nucleotides in the helix, which results in a tertiary structure known as an A-minor motif. 
    * This interaction induces conformational changes in the ribosome: 
        * G530 fastens the codon:anticodon helix into the decoding centre, bringing the 530 loop of the 30S shoulder towards the body, resulting in domain closure 
        * Shoulder domain is brought into contact with EF-Tu
        * EF-Tu domain 2 is shifted towards the shoulder and docked at sarcin-ricin loop (SRL). 
        * 3’ acceptor arm of aa-tRNA is distorted, which disrupts interactions between tRNA and switch I of EF-Tu, thus positioning EF-Tu within the ribosome for GTPase activation. 
        * The phosphate A2662 in the SRL can position the catalytic His84 into the GTPase center, thus activating a water molecule to attack the terminal phosphate of the bound GTP. 
* Structural data from the elongating ribosome thus show how the ribosome effectively acts as a GTPase activating protein (GAP) for EF-Tu, coupling the matching of cognate codon and anticodon to the acceleration of GTP hydrolysis. 
* These findings demonstrated the structural basis for kinetic proofreading by the ribosome. 
    * GTP hydrolysis is accelerated for the cognate but not the near-cognate anticodons, allowing more cognate codons to proceed to the second proofreading step while near-cognate anticodons dissociate at a faster rate. 	
    * Furthermore, domain closure within the ribosome becomes key to rejecting near-cognate tRNAs during accommodation. At this point, EF-Tu has dissociated from the ribosome, so the tRNA is attached to the ribosome via the anticodon:codon interaction. With cognate codons, domain closure ensures that the tRNA remains associated with the ribosome. As near-cognate codons do not trigger this domain closure, they dissociate at a faster rate. 
* It also highlights how the ribosome cooperates with other components of the translation elongation machinery to ensure translational fidelity, e.g. interactions with EF-Tu, aminoacyl-tRNA 

# **Conclusion**

* Structural studies provide important snapshots of the ribosome throughout different stages of elongation 
* Also definitely important are the other components – tRNA, EFs. 
* Caveats
    * Transferability of findings in prokaryotic ribosomes to eukaryotic ribosomes 
    * Need better understanding of how ribosome moves from one state to the next – using molecular dynamics, single molecule biophysical measurements. At the moment they are not high resolution enough time-wise to capture rapid ribosomal movements 
* Potential future research directions or applications of this knowledge in the broader context of biochemistry and molecular biology? 