# Questionse
- ==Discuss the current model of deadenylation in eukaryotic organisms, focusing on the experimental evidence that underpins the model.==
- How do the 5’ cap and 3’ poly(A) tail influence the productive lifetime of eukaryotic mRNAs?
- Discuss the experimental approaches used to study mRNA decay in eukaryotic cells. – [[B6 Cytoplasmic RNA decay#Approaches to studying RNA decay|Approaches to studying RNA decay]]
- Discuss the mechanisms and regulation of mRNA decay in yeast and mammalian cells.

# Why cells degrade RNA 
- **Regulation** **of mRNA levels**: RNA decay is a way of regulating steady state mRNA levels in the cell, in contrast to transcription rates 
	- E.g. In *Xenopus* oocytes, deadenylation occurs after GVBD (nuclear envelope breakdown) as PARN deadenylase is originally in the nucleus. 
- **Quality control**: Remove defective (and potentially toxic) mRNAs 
- **Remove by-products** of gene expression e.g. introns 
- **Recycling** of nucleotide pool 

# Approaches to studying RNA decay 
#comment Provide examples of each technique 
![[Pasted image 20230103183445.png|525]]

## Identifying key factors
### Genetic approaches (KO approach) 
- Generate mutants that inactivate decay pathways 
- Purify protein complexes and RNPs associated with polyA+ RNAs 
- Look at decay intermediates that accumulate in mutants 

### Tethering assays 
- Target specific protein to a given mRNA to assay effect on stability 

1. Tag mRNA with specific sequence (e.g. MS2 coat-protein binding site)
2. Generate fusion between coat protein and interested RNA binding protein 
3. Co-express tagged mRNA and fusion protein
4. As a control, use coat protein without fused RBP 

![[Pasted image 20230103184047.png|500]]

### *In vitro* reconstitution 
- Protein complexes can be purified using affinity tags directly from yeast, or by assembly of individually purified proteins 
- Introduce mutations 
- Incubate protein complexes with synthetic RNA and assay for activity over time using PAGE 
- E.g. #experimental_evidence Stowell et al (2016) reconstituted a fully recombinant Ccr4-Not complex that removes poly(A) tails and characterised its activity 
- E.g. #experimental_evidence [[B6 Cytoplasmic RNA decay#Mechanism of polyadenylation|Webster et al 2018]] 

## Measuring RNA stabilities 
- Stability is quantified in terms of half-life 
- In the absence of transcription, changes in RNA levels reflect decay; hence methods of measuring RNA stability should involve inhibition of transcription

### Transcriptional pulse 
- **Transcriptional pulse** involves a short burst of transcription that is then repressed such that all mRNAs produced are the same age. Then mRNA levels can be followed over time (e.g. using a Northern Blot), and this data can be used to calculate half-life 
- Transcriptional pulses can be generated by placing the gene of interest under the control of a regulatable promoter e.g. GAL 
1. Grow cells on a neutral sugar (e.g. raffinose)
2. Add galactose to induce transcription for a short period of time (e.g. 5 min)
3. Add glucose to repress transcription 

### Measuring poly(A) tail 
#### Bulk poly(A) assay 
1. Label ends of poly(A) tail with <sup>32</sup>P 
2. Treat RNA with RNase A/T1 which only digests G/C/U, leaving poly(A) tail intact 
3. Run sample on PAGE to separate poly(A) tails by lengths 
4. Quantify each band (which each correspond to a poly(A) tail of a different length)

### Deep sequencing methodologies 
- Transcriptional pulse followed by RNA-seq can determine decay rates transcriptome-wide 
- **TAIL-seq** can be used to determine poly(A) tail lengths of specific genes transcriptome-wide (Chang et al 2014)
![[Pasted image 20230103190355.png|400]]
- Short-lived and long-lived RNA can be differentiated by the following graphs from TAIL-Seq![[Pasted image 20230527232045.png|500]]
	- The paradox is that on average, short-lived RNA have longer poly(A) tals than long-lived ones
	- This is because the long-lived RNAs are able to reach a steady state within the cell, but short-lived RNAs have their tails immediately degraded upon export and so are not detected, but if they haven't been exported then their tails are not degraded at all. 
- **Poly(A)-tail Length Profiling by Sequencing (PAL-Seq)** gives similar information to TAIL-seq via different method (Subtelny et al 2014)
	- #experimental_evidence Subtelny et al (2014) used PAL-seq to determine the poly(A) tail lengths of the transcriptomes of several species and found that tail lengths were conserved between orthologous mRNAs. In addition, while tail lengths were linked to translational efficiency in early zebrafish and frog embryos, they became more delinked at gastrulation, suggesting a switch in nature of translational control. 
![[Pasted image 20230111223927.png|600]]
B = biotin 

#### Poly(A) tail length assay (PAT assay)
To measure Poly(A) tail of specific mRNA 
1. Add a G/I tail to the 3' end of a poly(A) tail using poly(A) polymerase 
2. Reverse transcribe from the G/I tail using a primer that targets GI tail 
3. PCR using primer that targets a specific mRNA 
4. Separate PCR products on gel to measure length 

## Isolating decay intermediates 
- Decay intermediates usually have short half-lives that make them difficult to isolate, but their half-life can be increased by including a structure that slows exonuclease activity, e.g. poly(G) tract 

## Areas for future development 
- Temporal approaches to assay changes in RNA stability 

## Limitations 
- Buffering and redundancy within gene expression means that roles of some factors may be masked 
- Cannot KO/deplete essential decay factors (or it kills the cell)

# Pathways of mRNA decay 
## General principles
- In eukaryotes, RNA is usually degraded by exonucleases 
- mRNA ends are protected to limit accessibility to exonucleases
	- Poly(A) tail coated with PABP at 3' end 
	- m7G cap and eIF4E at 5' end 
- Degradation is thus initiated by generation of unprotected ends, via 
	- Removal of one (or both) protecting structures, i.e. decapping and deadenylation 
	- Endonuclease cleavage (less common) by sequence-specific endonucleases or in response to siRNA 

## Canonical decay pathway 
![[Pasted image 20230103211659.png|575]]
1. Deadenylation by deadenylases (PAN2/PAN3, CCR4-NOT)
	1. **Biphasic model of deadenylation**: Initial trimming of poly(A) tail by PAN2/3, which is then continued by CCR4-NOT 
	2. CCR4-NOT is the dominant deadenylase complex 
		1. #experimental_evidence Yi et al (2018) – RNAi depletion of Caf1 (CCR4-NOT) or Pan2 in HeLa cells followed by TAIL-seq found that   Caf1 depletion leads to most mRNAs gaining elongated poly(A) tails, particularly cytosolic mRNAs. 
2. Decapping by a complex containing DCP2 with DCP1 as a cofactor
	1. Leads to irreversible commitment to decay 
3. Xrn1 5'->3' exonuclease 
	1. Main pathway is decapping followed by Xrn1 rather than exosome route; cells deleted for xrn1 gene grow very slowly, while cells deleted for exosome component genes continue to grow well 

#experimental_evidence Horvathova et al (2017) used single molecule imaging of fluorescent reporter mRNA 
![[Pasted image 20230103212623.png]]
Degradation from 5' end -> GFP is removed first. 

## Components of canonical decay pathway 
### Deadenylase enzymes 
#### PAN2-PAN3 
- PAN2-PAN3 interacts directly with poly(A) binding protein, which stimulates its activity 
- Complex of 2 copies of PAN3 with 1 PAN2 catalytic subunit 
- In yeast, PAN2-PAN3 does a slight trimming of the poly(A) tails in the nucleus 
- In mammals, it does this trimming prior to CCR4-NOT 

#### CCR4-NOT 
##### Structure 
- Large complex that contains 2 separate deadenylase subunits 
- 7 core subunits arranged around Not1 scaffold protein 
- CCR4 and CAF1 are the nuclease subunits and form a heterodimer 
- CAF1 trims naked poly(A) segments and is blocked by PABPC, while CCR4 is activated by PABPC to shorten PABPC-protected sequences (Yi et al 2018)
	- #experimental_evidence Yi et al 2018 – RNAi of CCR4 shows footprinting consistent with PABP 
- CCR4 is needed and has the ability to remove PABP from RNA, unlike CAF1 
	- #experimental_evidence Webster et al (2018) performed deadenylation assays with point mutants in nuclease active sites; Caf1(D53A) had no effect on deadenylation in the presence of PABP1, while Ccr4(E387A) reduces deadenylation in the presence of PABP1 
- CAF1 binds NOT1 scaffold directly and tethers CCR4 to the rest of the complex 
- Binds many trans-acting RBPs and adaptor factors (e.g. [[B7 Small RNAs#GW182|GW182]] – important for miRNA silencing)

##### Mechanism of polyadenylation 
- PABP inhibits CAF1 while promoting CCR4 deadenylation. When a molecule of PABP is released, CCR4 becomes inactive since PABP is absent. CAF1 inhibition is removed, and it becomes active to trim the PABP-free poly(A). On encountering the next PABP, CAF1 is inhibited and CCR4 is stimulated, starting the cycle again. 
![[Pasted image 20230103215449.png|350]]
-  #experimental_evidence Webster et al (2018) incubated recombinant CCR4-Not with RNA substrate with a 30(A) tail, +/- PABP. Tail length was resolved over time using denaturing PAGE; presence/absence of PABP did not affect deadenylation. 
	- This is probably because CAF1 could act in the absence of PABP, while CCR4 could act in the presence of PABP 

### Decapping enzymes 
- DCP2 is the catalytic unit for decapping, and uses DCP1 as a co-factor. Other co-factors with DCP1/2 in the decapping complex regulate decapping 
- Produces RNAs with 5' monophosphate, which is a good substrate for XRN1

### Deceleration of deadenylation of shorter poly(A) tails
- [[ParkEtAl_2023]] observed the steady state and pulse-chased distribution of poly(A) tail lengths using various seq methods (e.g. TAIL-Seq). They found that deadenylation is slower then the tails are 30-60 nt long. LARP1 knockdown globally reduced mRNA abundance.

> [!image] Model from [[ParkEtAl_2023]]
> ![[Pasted image 20230522215942.png]]

# Targeted mRNA decay 
- mRNAs show a wide range of half-lives that are transcript-specific 
- A large determinant of half-life is the presence of *cis*-acting RNA elements that are recognised by *trans*-acting RNA-binding protein factors 

## AU-rich elements (AREs)
- **Negative** regulators of mRNA stability
	- #experimental_evidence Shaw and Kamen (1986) engineered a beta globin gene to include an ARE. Half-life of beta globin transcript with ARE is shorter than one without ARE, as shown by time course PAGE 
- Present in 3'UTRs in clusters 
- There are several ARE-binding proteins that directly bind and destabilise target mRNA 
	- e.g. Tristetraprolin (TTP) recruits decay factors directly to ARE-containing mRNAs 
		- #experimental_evidence TTP RNAi stabilises ARE-mRNAs while overexpression enhances ARE-mediated decay 
		- #experimental_evidence Lykke-Anderson & Wagner (2005) – TTP tethered to 3'UTR of reporter mRNA via MS2 binding sites results in increased rate of decay 
- HuR is an ARE-binding protein that *stabilises* mRNA 
- ARE-containing RNAs are degraded by canonical route, suggesting that they recruit deadenylases more efficiently 

## CCR4-NOT and IDRs
- Many decay factors  have modular architecture where RNA-binding domain is flanked by IDRs for protein:protein interactions 
- Short linear motifs (SLiMs) in intrinsically disordered regions (IDRs) of proteins can recruit CCR4-NOT via Not1 scaffold and non-enzymatic subunits (e.g. Rcd1)
- Many trans-acting decay factors (including TTP) interact with hydrophobic grooves on CCR4-NOT

# Effect of codon usage on mRNA stability 
- Codon usage leads to changes in speed of translation during elongation due to variation in 
	- Amino acid composition 
	- Secondary structure 
	- Charged tRNA abundance 
- Codon optimality refers to "a scale that reflects the balance between the supply of charged tRNA molecules in the cytoplasmic pool and the demand of tRNA usage by translating ribosomes" ([[PresnyakEtAl_2015]]). Rare codons tend to be suboptimal
- Non-optimal codons are postulated to be decoded more slowly and less accurately than optimal ones, which may slow translation 
- The usage of non-optimal codons destabilises mRNA 
	- #experimental_evidence [[PresnyakEtAl_2015]] – substitution of optimal for non-optimal codons destabilises synthetic mRNA reporter. 
- CCR4-NOT is the sensor for slow translation as a result of nonoptimal codon usage 
	- Ribosomes take longer than usual to decode nonoptimal codons, which increases the chance of dissociation of the E-site tRNA before a cognate tRNA is accommodated in the A-site. This results in the E-site adopting a specific conformation recognised by the Ccr4-Not complex via the Not5 N-terminus to initiate the canonical mRNA degradation pathway. 
	  ![[Pasted image 20230111221337.png|575]]
	- #structural_evidence  Cryo-EM study by [[BuschauerEtAl_2020]] showed that the N-terminal domain of NOT3 inserts into the empty ribosomal E site only when the ribosome adopts a distinct conformation that lacks an accommodated tRNA in the A-site, which indicates impaired decoding kinetics. Furthermore, [[B5 Translation and translation regulation in eukaryotes#Ribosome profiling|ribosomal profiling]] indicates that low optimality codons are enriched in the A site in Ccr4-Not-bound ribosomes. 
- Synonymous gene mutations may be a way for cells to evolve advantageous mRNA stability rates without affecting the structure of the protein 
	- #experimental_evidence [[PresnyakEtAl_2015]] found that mRNAs that encode proteins that acct in similar pathways or are part of the same stoichiometric complex encode nearly identical proportions of optimal codons; such mRNAs have also been observed by others to have similar decay rates 