# Questions 
- What does oncogene cooperation mean and how does it work?
- Functions of cancer-associated genes 
	- How does the p53 protein suppress tumourigenesis?
	- How do members of the myc gene family become oncogenically activated?
	- ==**Given that proto-oncogenes carry liabilities for cancer, why have they not been eliminated from the genome of multicellular organisms?**==
	- How is the p53 tumour suppressor function engaged specifically in tumour cells but not normal cells?
- Genes vs Pathways 
	- ==Discuss whether attenuation of intracellular signalling is essential for tumour suppression.==
		- Tumour suppression = mechanisms to inhibit cell proliferation and cell cycle progression 
		- Yes, attenuation of some intracellular signals is needed
			- e.g. BRCA, p53, APC attenuates signals from oncogenes 
			- e.g. Oncogenes also attenuate their own signals via intrinsic tumour suppression mechanisms 
		- Extracellular signalling is also needed for tumour suppression e.g. external growth factors, contact inhibition
	- ==**Discuss the proposition that pathways rather than genes are the target of mutation in cancer.**==
- *==How many mutations are necessary for cancer?==*
- Discuss the different substrates that tumours can use to drive their growth.
- Discuss the evidence for an inherited basis to cancer.
- **==Discuss the notion that cancer is simply aberrant regulation of normal processes.==**

# Definition of oncogene and tumour suppressor 
* Oncogene = mutated or overexpressed protein or altered pattern of expression leads to tumour. “Gain of function” 
* Tumour suppressor = loss of function of a normal cellular protein leads to a tumour. “Loss of function” 

# Hallmarks of cancer 
See [[HanahanWeinberg_2011]]
## Cell intrinsic 
* Growth signal autonomy
    * Reduced requirement for growth factors 
* Resistance to inhibitory growth signals 
    * Loss of contact inhibition 
    * Anchorage independence 
    * High saturation density 
* Unlimited replicative capacity
    * Immortalization 
    * Re-expression of telomerase 
* Reprogramming of cell metabolism 
    * E.g. aerobic glycolysis 
    * Increased glucose transport 
* Resistance to apoptosis 
    * Most incipient tumour cells die by apoptosis, so development of apoptosis resistance can lead to cancer 
* Genetic instability 
    * Can lead to rapid accumulation of further mutations 
    * Both a symptom (e.g. resistance to apoptosis allowing mutations to accumulate) and driver of cancer (e.g. BRCA) 
    * Positive feedback 

## Cell extrinsic 
i.e. affects tumour microenvironment 
* Induction of angiogenesis / neovascularization 
    * To supply O<sub>2</sub>, nutrients 
* Metastatic potential 
    * Epithelial to mesenchymal transition (EMT) – cancer cells develop extended filopodia that confer motility
    * Tumour cells invade the surrounding tissue and enter the blood or lymphatic system to lodge in distant sites, developing into secondary (metastatic) tumours 
* Evasion of immune destruction 
    * Via production of cytokines that suppress immune response in tumours. The tumour cells may also recruit normal cells to produce the cytokines 

# Cancer is a genetic disease 
* Cancer is caused by genetic abnormalities
* Chemical can be tested for their mutagenicity in several ways 
    * Ames test: Measures ability of a chemical to mutate the genome of salmonella  
      ![[Pasted image 20230202235309.png|275]]
	    * Test compound must be metabolically activated by the liver as liver metabolism may convert the chemical to a different product with a different level of mutagenicity 
	* Mutagens can also be injected into mouse fibroblasts to test their ability to transform cells in culture, and then the transformed fibroblasts can be injected into mice to test their ability to form tumours 
* There is a correlation between agents that are mutagenic and those that are tumorigenic → cancer has a genetic basis 
* Tumours result from the loss of normal control mechanisms due to mutations in oncogenes and tumour suppressor genes 
![[Pasted image 20230202235407.png|325]]
# Oncogenes 
## Activation 
Proto-oncogenes can be activated to become oncogenic via two mechanisms 
* **Deregulated activity due to mutation in coding sequence** – affects regulated effectors e.g. most kinases (e.g. Src), phosphatases, protases, G proteins (e.g. Ras) 
* **Deregulated/elevated expression of wt protein** – affects innately active effectors e.g. transcription factors (e.g. Myc), some kinases 

### Types of mutations that can cause activation 
* Point mutations, e.g. missense, single base indel 
* Structural changes e.g. indels of regions, inversions, amplifications, translocations 

## Discovery  
* Activated endogenous oncogenes can be found directly by transfection of tumour DNA 
* #experimental_evidence  Tumour cell DNA is injected into normal mouse fibroblasts. Fibroblasts are observed for formation of a focus of morphologically transformed cells, then transformed fibroblasts are injected into an immunocompromised mouse host to observe for tumour formation  
    * Responsible oncogene can be identified due to the presence of human-specific Alu sequences adjacent to the oncogene on the same fragment 
    * Oncogenes identified in this way are those that produce mutated proteins with constitutive activity (e.g. Ras) 
* Kinetic expression array analysis 

## Mechanisms of causing cancer 
### Aberrant expression of growth factors 
* Tumour cells express a growth factor at elevated levels or one not normally expressed that can act in an autocrine fashion to stimulate growth and/or  inappropriate survival 
* E.g. overexpression of platelet-derived growth factor (PDGF) 

![[Pasted image 20230202235440.png|225]]

### Ligand-independent firing of growth factor receptors 
* Receptors (often RTKs) may express inappropriately such that they are more sensitive to a growth factor or transmit signals even in the absence of growth factors 
![[Pasted image 20230202235500.png|250]]
(loss of extracellular binding domain)
- #experimental_evidence Flt3 receptor – catalytic cleft is normally obstructed by an activation loop and a juxtamembrane domain. Phosphorylation of the tyrosine in the activation loop causes it to swing out of the way, allowing the catalytic domain access to substrates. In many gastrointestinal stromal tumours, mutations in the juxtamembrane domain gives access to the catalytic domain independent of activation loop phosphorylation state, causing the receptor to be constitutively active. 

### Aberrant expression of growth factor receptors 
* #experimental_evidence EGFR receptor is overexpressed in breast cancer, head and neck cancer, NSCLC (non-small cell lung cancer) 
    * Increased expression in one component of EGFR (R2/HER2) results in increased receptor at the cell surface → increased intracellular signalling
    * Herceptin is a monoclonal Ab to HER2 used to treat HER2-dependent breast cancer 

![[Pasted image 20230202235514.png|325]]

### Activation of intracellular signalling cascades 
* Activated receptors can recruit SH2-containing proteins that act as adaptors that can recruit activators of Ras signalling (GEFs) or that possess a catalytic function (e.g. Src, PLC) 
* SH2-containing phosphatases can also be recruited to switch off receptor activity 
* Mutations in intracellular signalling proteins can lead to inappropriate signalling 

#### Src 
* Non-receptor tyrosine kinase involved in intracellular signal transduction
* Phosphorylation at Y527 causes a conformational change that blocks the catalytic kinase site, inactivating Src
* Recruitment of Src via SH2 site exposes the Y527 site, allowing for dephosphorylation of Y527 and phosphorylation of Y416 to activate Src
* Mutation at Y527 generates a constitutively active Src protein. 
* Constitutive Src activity results in a 10-fold increase in pTyr in cells 

![[Pasted image 20230404155608.png]]

#### Ras
* Ras is a GTPase that transduces signals from membrane receptors to intracellular signalling pathways 
* Ras is activated by recruitment of guanine exchange factors (GEFs) to the membrane by the receptor, which add GTP 
  ![[Pasted image 20230404155642.png|550]]
* GTPase-activating proteins (GAPs) turn off the Ras signal 

![[Pasted image 20230202235533.png|400]]

* Oncogenic point mutations in Ras increase its affinity for GTP such that GAPs cannot activate the intrinsic Ras GTPase activity → Ras constitutively active
    * Mutations include G12E, G12V, G12D, G13D, Q61K which are involved in contacting GTP 
* Ras has many downstream targets and oncogenic Ras leads to persistent activity of these downstream pathways 
	* PI3K converts PIP2 to PIP3 which interacts with PH domain of Akt/PKB.  
	  ![[Pasted image 20230404155917.png|425]]
	* Ral-GEFs affect cell migration, acting on filopodia and lamellipodia
	* Akt/Pkb can also affect cell migration by activating Cdc42 
* Multiple inhibitors can regulate Ras downstream activity 

![[Pasted image 20230404160155.png]]

### Deregulation and/or elevated expression
* Gene expression can be deregulated in several ways 
    * Retroviral insertion 
    * Persistent stimulation by upstream oncogenic activity 
    * Mutation of control regions (e.g. promoter, enhancer) 
    * Genomic amplification (increase in copy number) 
    * Chromosomal translocation 

#### Myc
* Myc is a **transcription factor** involved in coordinating cell cycle progression, building/repairing tissues 
* In normal cells, Myc is only **expressed transiently** in response to developmental or mitogenic signals, and Myc is tightly regulated at the transcriptional, translational and post-translational levels 
    * Half-life of myc RNA and protein is only 15-20 minutes ⇒ Myc signal rapidly removed after growth factor stimulation 
    * Elevated and persistent expression of Myc causes apoptosis in normal cells 
* **Normal function in tissues**. Myc is required for proliferation of all normal cells and acts as a gatekeeper of the stem cell compartment. 
* **Activation in cancer**. Myc is upregulated in cancer through mutations in Myc gene, upregulation of upstream signals (e.g. RTK-Ras, Wnt-β catenin, NOTCH), etc. 
	* In cancers, Myc is often activated at a **low level** to drive proliferation, but not so high as to engage tumour suppressor mechanisms (i.e. apoptosis) – [[Threshold crossing in cellular decision-making#Implications of thresholds on oncogenic signalling|implications of thresholds]] 
* **Downstream effects**
	* Net consequence of Myc activity depends on context 
		* Many genes that Myc regulates are cell-type specific 
    * **Cell cycle progression.** Myc transcriptionally activates D and E cyclins and E2F transcription factors → cyclin D/cdk4 complex recruits and targets p27 for degradation by phosphorylating it → cyclin E /cdk2 complex becomes active 
      ![[Pasted image 20230202235605.png|375]]
    * **Effect on tumour microenvironment**
	    * Myc (with KRas<sup>G12D</sup>) increases expression of PDL-1, a ligand that switches off T-cell function → immune evasion. 
	    * Myc also causes [[D13 Tumour microenvironment#Endothelial cells mediate angiogenesis|angiogenesis]] 
	    * #experimental_evidence Sodir et al (2020) showed that activation of Myc in indolent pancreatic intraepithelial neoplasm (PanIN) epithelial cells *in vivo* induces changes in multiple types of stromal and immune cells, driving the transition to a pancreatic adenocarcinoma that resembles cases in humans. Deactivation of Myc disassembles the tumour and causes death of tumour cells. 
	      ![[Pasted image 20230404162837.png]]
* **Examples of myc dysregulation**
    * Persistent upstream oncogenic activity (e.g. Ras, Wnt) can deregulate Myc 
	    * Myc may thus act as a conduit/node for various upstream oncogenic signals
      ![[Pasted image 20230202235633.png|350]]
        * Ras stabilises Myc, beta-catenin transcriptionally activates Myc 
    * In neuroblastoma, degree of N-myc amplification associated with poor prognosis
    * In non-AIDS Burkitt’s lymphoma, c-myc expression is deregulated by chromosomal translocation which places myc gene adjacent to Ig heavy chains which are extremely active in B cells 
        * Chromosomal translocation likely caused by aberrant regulation of isotype switching or somatic hypermutation in B cells 
    * In rat embryo fibroblasts, constitutive expression of Myc protein prevents growth arrest and makes cells mitogen independent 
    * In a transgenic mouse model where Myc is linked to the insulin promoter, expression is restricted to beta cells of the pancreatic islets → activation of Myc by tamoxifen induces cell proliferation within pancreatic islets. Ki67 was used as a proliferative marker (turns brown) 

## Tissue specificity 
- The same two oncogenes (KRAS<sup>G12D</sup>, Myc) can elicit different tumour architectures in different tissues, e.g. lung vs pancreas
- Oncogenes may activate a tissue-specific regenerative programme? 
![[Pasted image 20230404162751.png]]

#### bcr-abl
* c-ABL - encodes a non-receptor tyrosine kinase
* In chronic myeloid leukaemia (CML), a reciprocal translocation between chromosomes 9 and 22 creates a fusion gene BCR-ABL which encodes a deregulated tyrosine kinase

## Cellular counters to oncogene activation 
### Oncogene-induced senescence 
![[Pasted image 20230404160633.png]]

### Intrinsic tumour suppression 
* For many oncogenes, elevated activity induces senescence or apoptosis via activation of tumour suppressors
    * There is a threshold level of oncogenic signalling above which tumour suppressor pathways are engaged 
    * Whether senescence or apoptosis is engaged depends on the cell type, level of oncogenic signalling, and extent of other pro-apoptotic / pro-growth signals in the cell 
    * Both programmes could be backups for each other 
* E.g. elevated Myc expression can induce the intrinsic apoptosis pathway via p53 signalling 
  ![[Pasted image 20230202235657.png|350]]
    * High levels of Myc expression increases E2F activity
    * Alternate reading frame (ARF) protein from the *INK4A* locus (that also encodes p16) senses the increased oncogenic activity (above a threshold)
    * ARF then inhibits Mdm2 from binding to p53 → p53 stabilisation 
    * Myc can also activate ARF via E2F1-independent mechanism 
    * #experimental_evidence Ulz, Ellen & Speicher (2016) found that there is a highly significant co-occurrence of MYC amplification with mutations in the TP53 gene in the TCGA breast cancer data set 
    * #experimental_evidence Downregulation of apoptotic genes e.g. *bax*, *puma* promotes tumourigenesis even in the presence of wt p53, showing that apoptosis is an important mechanism for intrinsic tumour suppression 
    * p63, p73 may also play a role in tumour suppression. p73 is a direct transcriptional target of Myc and E2F, and p73 has been implicated in apoptosis 
* E.g. oncogenic Ras can induce senescence (permanent cell cycle arrest) via activation of p16<sup>INK4A</sup> which activates [[D3 Oncogenes and Tumour Suppressors#Cell cycle checkpoint proteins|Rb]]. It also activates p14<sup>ARF</sup> which activates p53. 
	* p53 and Rb promote senescence via various effectors e.g. p21CIP1
	* Contribution of Rb vs p53 depends on cell type
		* E.g. MEFs depend on ARF-p53
		* E.g. Human fibroblasts depend more on p16-Rb 
	* #experimental_evidence In mouse embryonic fibroblasts, disruption of ARF or p53 prevents Ras-induced senescence 
* Alternatively, intrinsic tumour suppression may be activated indirectly via DNA damage detected by repair machinery, which then signals to pro-apoptotic machinery 
* Preferable for cells with oncogenic mutations to be killed off as a single uncontrolled proliferating cell is lethal for the organism, while the loss or permanent growth arrest of a few cells has negligible effect on a large multicellular organism 

![[Pasted image 20230409123957.png]]

## Tumours develop multiple mutations 
### Genomic instability 

### Cooperation between oncogenes
* Single oncogenes are insufficient to drive and maintain cancer – tumour cells require at least 2 complimentary oncogenic activities to overcome regulatory pathways that inhibit inappropriate signalling (e.g. intrinsic tumour suppression, senescence)
	* #experimental_evidence HRAS<sup>V12G</sup> is necessary but not sufficient for genesis and maintenance of melanoma 
    * Common for Myc to cooperate with anti apoptotic protein since Myc overexpression is inhibited by apoptosis 
* E.g. Myc and Ras – both must be oncogenic to transform cells in culture. 
    * #experimental_evidence  Land et al 1983 found that neither Ras nor Myc were able to transform rat embryo fibroblasts on their own, but when both oncogenes were introduced into the fibroblasts at the same time, the cells were able to yield fibrosarcomas after injection into mice  	
        * The authors explain this observation by stating that, while the Ras oncogene is a strong inducer of refractile morphology, anchorage independence, and growth factor secretion, and weak in its ability to immortalise cells, the Myc oncogene is good at immortalising cells while weak at inducing refractility, anchorage independence and growth factor secretion (likely as they act via different signalling pathways) 
        * Concurrent mutation in both genes thus allows for maintenance of a fuller spectrum of cancer hallmarks that allows for successful tumour formation and maintenance 
    * in hematopoietic tumorigenesis, Ras is thought to suppress immune surveillance, allowing for cells overexpressing Myc to form tumours (Mahauad-Fernandez & Felsher 2020) 
    * Ras may also stabilise Myc, via phosphorylation of Myc through the Raf-1 and PI3K pathways (Bachireddy et al 2005)
      ![[Pasted image 20230404161420.png|475]]
* #experimental_evidence  in murine melanoma, oncogenic H-Ras<sup>V12G</sup> only causes tumours in mice that have an additional lesion, in this case deletion of p16 (which mediates growth arrest in response to high levels of oncogenic Ras signalling) 
* #experimental_evidence  Myc and Bcl-XL – elevated levels of Myc promote apoptosis in beta cells of pancreatic islets, but co-expression of the anti-apoptotic protein Bcl-XL inhibits Myc-driven apoptosis → allows pancreatic islets to expand and invade surrounding tissue 
* #experimental_evidence  Myc and Bcl-2 in B cell lymphomagenesis – Bcl-2 expression results in retention of B cells that are normally deleted  

![[Pasted image 20230202235714.png|375]]

### Cancer cell evolution 
* Tumours are heterogenous – each tumour cell contains different mutations 
* Nascent tumour cells with higher fitness allows the cell to outcompete other cells (by having mutations that confer e.g. faster proliferation, resistance to apoptosis), which are then selected for 
* Some oncogenic mutations (e.g. Myc) are lethal (by activating apoptosis), so simultaneous mutation in other pathways (e.g. for apoptosis resistance) may be necessary 

### Different mutations are required at different stages of cancer development
* Cancers have multiple hallmarks, and development of each hallmark requires different mutations
    * E.g. Myc can cause most of the intrinsic hallmarks, but cannot cause apoptosis resistance since it activates apoptotic pathways → need to cooperate with Myc 
    * Initiation vs metastasis require different mutations 
* Following metastasis, different mutations are required at primary and secondary sites which are likely to have different environments
    * E.g. if pancreatic cancer metastasises to the lung – hypoxic environment → high O<sub>2</sub> environment 
    * New mutations needed to adapt to the new tumour site 

## Which mutations are necessary for tumorigenesis?
* Driver and passenger mutations
* Genome wide association studies (GWAS) can be used to identify mutations involved in tumorigenesis 
* Can also compare transgenic mice to a control group (e.g. Ras<sup>G12V</sup> to a control) 

# Tumour suppressors 
## Definition 
* Genes that become inactivated in naturally occuring cancers
* Gene whose inactivation promotes incidence of cancer 

## Evolutionary origin  
* Many tumour suppressor genes are evolutionarily ancient and highly conserved ⇒ suppression of cancer likely a re-tasking of preexisting functions

## Loss of function in tumour suppressors is a recessive phenotype 
* **Two-hit hypothesis** – familial cancers require only “1 hit” as patient already has inherited one inactivated tumour suppressor allele (i.e. patient inherited first hit), while sporadic cancers require 2 independent hits to inactivate both tumour suppressor alleles
    * Exception: p53 – dominant negative. Tetrameric, if one allele is non-functional, then nonfunctional subunits can combine with functional subunits to form a tetramer that is overall non-functional. Most missense mutations in p53 are in the DNA binding domain as it enables dominant negative. 
* #experimental_evidence  – Cell fusion studies between normal cell and cancer cell resulted in a hybrid cell that was **non-tumorigenic** ⇒ cancer cells are recessive 
* #experimental_evidence  – Knudson found that diagnosis for bilateral hereditary cases of retinoblastoma occurred at a younger age and showed a curve consistent with a single-mutation process. He also noted that the rate of diagnosis for unilateral nonhereditary retinoblastoma was delayed relative to that of the bilateral cases and showed a curve consistent with a two-mutation process
    * If already had 1 mutant allele → less time needed for other allele to mutate as well 

### Mechanisms of loss of function 
* Chromosomal lesion in 1 Rb allele was found to be replicated in the other allele
* Mitotic recombination 
  ![[Pasted image 20230404170129.png]]
* Chromosomal nondisjunction – problems with separating chromosomes
  ![[Pasted image 20230404170137.png|475]]
* Gene conversion – DNA polymerase jumping between chromosomes to use different chromosomes as template in replication
  ![[Pasted image 20230404170050.png|550]]
* Hypermethylation of promoter of non-damaged second allele silences its expression e.g. Timp3, VHL, PTEN
* Haploinsufficiency e.g. Nf1 in neurofibroma, p53 (dominant negative)
    * "The situation that occurs when one copy of a gene is inactivated or deleted and the remaining functional copy of the gene is not adequate to produce the needed gene product to preserve normal function."

## Discovery 
### Comparative genome hybridisation (CGH) 
* Method for analysing copy number variations (CNVs) in a test sample compared to a reference sample
* CGH uses two genomes, a test and a control, which labelled with different coloured fluorophores, mixed and competitively hybridised to ssDNA probes on a microarray 
* Fluorescent signal intensity of the labelled test DNA relative to that of the reference DNA can be determined → determine relative copy number of sequences in the test genome as compared to the normal genome

![[Pasted image 20230202235742.png]]

* Genomes of many solid human cancers have multiple regions of gene loss and gain 

## Mechanisms of tumour suppression 
### Caretakers e.g. BRCA1/2
* Caretaker genes maintain genetic/genomic integrity 
    * Maintenance critical to cancer suppression – loss of integrity arises as a consequence of DNA damage along with erosion of DNA damage sensing/repair machineries and/or failed checkpoints 
* BRCA1/2 involved in DS break repair, particularly DNA damage sensing and maintenance of genomic integrity 
    * BRCA1 – mediates repair of DNA double strand breaks 
    * BRCA2 – homologous recombination 
* BRCA1 forms a dimer with partner BARD1 via an N-terminal RING domain, and this dimer binds damaged DNA and interacts with DNA damage sensing machinery 
* BRCA2 also interacts with multiple proteins involved in DNA damage sensing and repair 
* The loss of caretaker genes does not directly cause cancer as it is a repair mechanism. In the theoretical situation where no DNA damage occurs, then no cancer develops even if caretakers are mutated
    * Baseline mutation rate is the same, but repair rate in BRCA mutants are lower → net mutation rate increases 
![[Pasted image 20230202235752.png|475]]

### Gatekeepers e.g. APC 
* Involved in cell cycle checkpoints and attenuation of growth/survival/  migration signals 
    * E.g. APC – attenuates signalling by Wnt growth factors
    * Attenuators of mitogenic or survival signalling pathways (e.g. PTEN, NF1, GSK3) 
    * Cell cycle checkpoints e.g. p16<sup><em>INK4A</em></sup>, p21<sup>cip1<sub>,</sub></sup> Rb
    * Intrinsic tumour suppression by oncogenes (e.g. Ras, myc) 

#### APC
* Familial adenomatous polyposis (FAP) is due to inherited inactivation of 1 *APC *allele, with cancer being caused by inactivation of remaining APC allele 
* APC is involved in Wnt signalling: 
    * Wnt is a growth/developmental ligand that binds to Frizzled receptors
    * Activated Frizzled induces release of intracellular Dishevelled to inhibit GSK-3β kinase/APC/Axin complex from degrading β-catenin 
    * Stabilised β-catenin relocates to nucleus to promote transcription of Myc, cyclin D1, etc. via Tcf/Lef
* Inactivation of APC (or axin, or other stabilising mutations of β-catenin) drives persistent Tcf/LEF transcription 
![[Pasted image 20230202235804.png|200]]

#### MAPK pathway 
![[Pasted image 20230202235817.png|425]]

* Negative feedback from tumour suppressors downstream in the pathway regulate growth signals 
* Ras, Raf, MEK oncoproteins act as their own tumour suppressors via negative feedback 

![[Pasted image 20230202235827.png|350]]

* This system of negative feedback can lead to **adaptive rewiring **in tumours e.g. Melanomas are often caused by oncogenic BRaf<sup>V600E</sup>, which acts in an analogous manner to Raf to upregulate ERK and hence cell growth and proliferation 
    * Although inhibition of BRaf<sup>V600E</sup> initially leads to remission, the reduction in activity of BRaf<sup>V600E</sup> releases inhibition of Raf, which then takes over in upregulating ERK activity → new steady state  
    * Adaptive rewiring allows for drug resistance without _de novo _evolution (no new mutations) 
![[Pasted image 20230202235924.png|500]]

- The energy status of the cell also influences MAPK signalling via the tumour suppressor LKB1/STK11 (a serine/threonine kinase) 
	- Under low glucose, low ATP conditions, LKB1 activates AMPK which suppresses proliferation
		- Competition between growth factor signalling (via Ras) and suppression via LKB1 
    * However, such activity to suppress proliferation can be oncogenic because in hypoxic environments (low ATP), moderation of proliferation may help the tumour survive 
    * LKB1/STK11 is functionally inactivated in Peutz-Jeghers Syndrome (PJS) 
![[Pasted image 20230203000004.png|325]]
![[Pasted image 20230203000013.png|375]]

#### Krebs cycle enzymes
* Krebs cycle enzymes may be tumour suppressors (e.g. fumarate hydratase, succinate dehydrogenase) or oncogene (isocitrate dehydrogenase) 

#### Hh Hedhehog signalling in cancer 
![[Pasted image 20230203000042.png|275]]

* When Sonic Hedgehog (Shh) ligand bind to the Patched receptor (a tumour suppressor), its inhibition of Smoothened (proto-oncogene) is lifted 
* If Patched is deleted or Smoothened is mutated such that it cannot be inhibited by Patched, the Hh pathway is constitutively activated 
* Hh pathway mainly involved in embryogenesis → mainly affects young children
    * Most precocious Hh activation involved in basal cell carcinomas, 25% of which are medulloblastomas 
* Cyclopamine, vismodegib acts downstream of Smo to inhibit its signalling
    * Not effective in medulloblastoma as the medulla is robust to growth inhibitors (will just compensate with other redundant pathways) 

#### Cell cycle checkpoint proteins e.g. pRb 
See [[BurkhartSage_2008]]
![[Pasted image 20230203000055.png|275]]

* E2F1-3a are transcriptional activators of genes involved in S phase progression 
* E2F3b,4-7 are transcriptional repressors that regulate genes involved in quiescence and/or differentiation 
* E2Fs are controlled by pocket proteins e.g. p105<sup>RB</sup>
    * p105<sup>RB</sup> sequesters E2F in a pocket to keep them inactivated 
    * pRB also activates HDACs to promote chromatin condensation → inactivation of transcription 
    * G1 cyclins inactivate pRB to liberate E2F for G1/S phase transition  
      ![[Pasted image 20230203000105.png|375]]
    * Upon release, E2F activates transcription by recruiting HATs that decondense chromatin 
* Mutations in pRB → inhibition of E2F1-3 lifted → retinoblastoma 
* pRB is bound and inactivated by DNA virus oncoproteins (e.g. adenovirus E1A, SV40 T Ag, HPV E7) 

### Stress sensors e.g. p53
* Monitor dysfunction or damage and coordinate repair where possible, and senescence or cell death where not 
    * Includes DNA damage, unfolding proteins, toxins, etc.

#### Activation of p53
- p53 can be activated by DNA damage, oncogenic signalling, and other stresses (e.g. hypoxia, translation stress, replication stress)
- **Activation by DNA damage**. DNA damage → p53 phosphorylation → prevents inhibition of p53 by Mdm2 + p53 stops upregulating Mdm2 → [p53] increases
- **Activation by oncogenic signalling.** Oncogenic signals -> p19<sup>ARF</sup> -| Mdm2 
- #experimental_evidence Christophorou et al (2006) – showed that p53 can be activated by oncogenic stress (i.e. signalling from oncogenes) rather than by DNA damage. 
	- Engineered mice to have switchable p53 that is activated by treating the mice with tamoxifen. p53 was then activated before irradiation or after irradiation.
	- When p53 was activated before irradiation, apoptosis was observed but tumour formed early  
		- The incipient tumour cells likely arose from a distinct cell population that survived the DNA-damage induced, p53-mediated apoptosis. But p53 could not suppress them because p53 activity would have faded after withdrawal of tamoxifen. Early tumour formation may be due to genomic instability from irradiation. 
	- When p53 was activated after irradiation, p53-mediated apoptosis was not observed but tumour still formed late => p53 suppresses tumours via a non-apoptotic pathway, which is activated by oncogenic signalling
		- But in p19<sup>Arf</sup> KO mice, there is still no apoptosis BUT the tumour forms early => p53 suppresses tumours via oncogenic-stress pathway (oncogenic signals activated, p19<sup>Arf</sup> which then signals to p53) 
- **Activation by epigenetic changes.** p53 may be able to detect aberrant epigenetic changes, leading to its activation and thus cell death 
	- #experimental_evidence DNMT1 is a methyltransferase for newly synthesised CpG in a hemimethylated region of DNA, resulting in methylation of both strands. When DNMT1 was deleted in mice via Cre-Lox, CpG methylation does not occur, and apoptosis occurs after two cell divisions. However, when *TP53* was also deleted in the mice, the cells do not undergo apoptosis. 
	- #experimental_evidence The drug Azacytidine prevents methylation of CpG. This kills cells with p53 KO at higher frequency than cells with wt p53 
- **Activation by cellular stresses**. 

> [!image] Signalling inputs to p53
> **Activation**
> ![[Pasted image 20230523194431.png]]
> ![[Pasted image 20230404174351.png]]
> 
> **Inhibition**
> ![[Pasted image 20230523194525.png]]

#### Regulation of p53 output 
- Interacting partners
	-  MDM2 and MDMX interact to regulate p53 function by acting as E3 ubiquitin ligases for p53
		- MDM2 and MDMX interact via RING-finger motifs 
		- #experimental_evidence Transgenic mice overexpressing Mdm2 have spontaneous tumourigenesis 
		- MDM2 levels can also be regulated by miRNAs, e.g. miR-143/145 
			- Some miRNAs are themselves regulated by p53
		- Role of AS – many human cancers have high levels of MDM2 splice variants, usually involving varying extens of p53-binding domain -> p53-independent activity? 
	- ASPPs are apoptotic-specific regulators of p53 
		- ASPP1 and ASPP2 binds to DNA-binding domain of p53 and may recruit it to promoters of pro-apoptotic genes to stimulate their transcription 
		- iASPP → decrease in apoptosis
![[Pasted image 20230404175340.png]]
- Post-translational modifications of p53 
![[Pasted image 20230404175134.png]]
- Strength of p53 response element
- Epigenetics – level of methylation at target gene promoter can affect how much it can be upregulated by p53 

![[Pasted image 20230404174644.png|500]]

#### Activity of MDM2 
- MDM2 is an E3 ubiquitin ligase that responds to intrinsic and extrinsic cellular stresses, which results in PTMs on MDM2 that change its affinity for p53 or alter its ligase activity 

#### Downstream effects of p53
- p53 is a transcription factor and triggers transcription of target genes encoding cytostatic and/or apoptotic effectors 
- p53 target genes are regulated in a stimulus-, locus-, and context-specific manner 
- Pathways in the MDM2-p53 hub are often redundant, with different pathways leading to the same cellular outcome 
	- E.g. p53 can arrest cells at G2 phase by inhibiting CDC25C transcription, or lower levels of cyclin B1 required for G2-to-M transition
	- E.g. multiple ways to reduce Warburg effect (see belo)
- **Apoptosis**. p53 activation results in macrophage recruitment, recruitment of NK cells, and CD8+ T cells to cell 
	- p53 directly expression of macrophage inhibitory cytokine 1, which is secreted to attract macrophages 
	- p53 also activates Bim, Noxa, PUMA, etc which activate Bax/Bak 
- **Metabolism.** p53 induces genes that reduce Warburg effect 
	- p53 induces transcription of the gene TP53-induced glycolysis and apoptosis regulator (TIGAR), which reduces glycolytic activity and ROS levels during tumourigenesis
	- p53 induces transcription of SCO2, which regulates mitochondrial respiration
	- p53 signalling activates PTEN, AMPK and tuberin, which regulate mTORC1 and mTORC2 
- **ROS and mitochondria.**
	- Glutaminase 2. Converts glutamine to glutamate, which is a precursor for glutathione that protects against ROS 
- **Senescence**. p53 induces induction of senescence-associated secretory phenotype (SASP). SASP-associated genes are secondary transcripts i.e. activated by primary p53 transcripts 

> [!image] Transcriptional output of p53 (Levine et al 2020)
> ![[Pasted image 20230523194622.png]]

#### Evolutionary origin of p53
- p53 is highly conserved across a wide diversity of taxa, including in species with much fewer cells and short life spans that are unlikely to develop cancer ⇒ p53 likely exapted as a tumour suppressor 
- Modern choanoflagellates (outgroup for metazoans) contain p53 orthologue 
- Sea anemones, flies, round worms, *C. elegans* found to have p53 orthologs 
- DNA sequences that p53 family proteins bind to for regulation of gene expression in invertebrates and vertebrates are identical 
	- E.g. p63-like genes in *Drosophila* (*Dmp53*) and *C. elegans* (*cep-1*) regulate genes in the apoptotic pathway similar to humans 
	- However p63-like genes only expressed in germline of these invertebrates, whereas in vertebrates, p53 is expressed in soma and p63 is expressed in both soma and germline 

#### p53 mutations
- Most p53 missense mutations are in the DNA binding domain as it results in a dominant negative effect 

### Landscapers 
* Restrain precocious clonal expansion through restrictive impact on somatic microenvironment, e.g. stromal remodelling, angiogenesis, immune surveillance 

# Integrating signals at signalling nodes
- Tumour suppressor or oncogenes that are commonly implicated in cancer are often nodes in signalling networks, with no redundancies
	- Easier to eliminate a tumour suppressor node (e.g. p53) instead of knock out all of the redundant downstream pathways 
- Mutation in such nodes can cause loss of the entire pathway 
- However, having a single node is an efficient way of integrating signals from different stresses at the same time, thus allowing for the response to the stresses to be adjusted 
- With multiple nodes, there would be a higher error rate in signalling as there would need to be more steps in signalling pathways 
- Such errors could lead to inappropriate stress responses that are deleterious – e.g. if the cell has sustained too much DNA damage, the cell may attempt to endlessly repair the DNA instead of commit to apoptosis 

“Quorum sensing” function? 

# Order of mutations matters 
- The next mutation is selected for to increase the fitness of the subclonal population, but which next mutation increases fitness the most depends on what previous mutations are present in the lineage 
	- Many mutations in genes involved in DNA damage repair are often found together with p53, e.g. BRCA1/2 mutations are usually followed by p53 mutations. BRCA1/2  reduces the ability of the cell to repair DNA damage, but this DNA damage is recognised by ATM which signals to p53, increasing p53 signalling, finally resulting in apoptosis or DNA repair. So any phenotypic effects of BRCA1/2 mutations that could cause cancer development are suppressed by p53. 
	- E.g. Ras and p53 mutation also co-occur frequently due to intrinsic tumour suppression by Ras signalling. KRAS -> ARF -| MDM2 -| p53. p53 activation leads to SASP which places cell in a senescent state, thus preventing RAS-expressing cells from proliferating. Alternatively p53->PML->nuclear body formation->p54 acetylation by histone acetylase->cellular senescence   
	- E.g. Myc induces apoptosis but overrides senescence, so immortalising mutations would be redundant, instead mutations that result in anti-apoptotic signals would be selected for. In contrast, Ras promotes senescence but attenuates apoptosis, so mutations that target senescence would be selected for 
- #experimental_evidence In colon cancer, order of different gene mutations (APC->RAS->SMAD4) led to increasingly larger benign tumours, then a fourth mutation in p53 caused the formation of a malignant tumour. If the order was randomised, tumour formation was not efficient 
	- Matano, M. et al. Modeling colorectal cancer using CRISPR–Cas9-mediated engineering of human intestinal organoids. Nat. Med. 21, 256–262 (2015)
	- Takeda et a (2015)
- It has been observed that patients with Li-Fraumeni syndrome, i.e. have a heterozygous *TP53* germline mutation, develop different tumours at different ages. 
	- Hypothesised that in some tissue-specific stem cells, initial p53 mutation can initiate a benign tumour, while in other tissues, p53 mutation does not lead to phenotype until additional mutations occur (e.g. in the colon cancer example given above), so tumours only develop in older age when mutations accumulate 

# Tissue-dependence of mutations 


# Therapeutic strategies
## Broad spectrum vs narrow spectrum inhibitors 
* Broad spectrum proliferation inhibitors target cancer cells more than normal cells as they replicate more → selectivity window / therapeutic index 
    * But have issue with selectivity 
* Narrow spectrum inhibitors target specific receptors 
    * E.g. Erlotinib (EGF receptor), herceptin (Her2 receptor)
    * But these are vulnerable to drug resistance development 

## Issues
### Selectivity 
* Need to kill proliferating tumour cells without affecting normal proliferating tissues e.g. stem cells (selectivity) 
    * Cancer cells are similar to stem cells (e.g. Warburg metabolism, proliferation markers)

### Evolution of drug resistance 
* Cells with additional mutations or altered signalling confer a selective advantage in the presence of the drug  
![[Pasted image 20230203000146.png|150]]
* Evolves via 
    * Secondary mutations that reduce drug binding affinity without affecting target activity (“gatekeeper” mutations) 
    * Compensation in a related/redundant pathway (see adaptive rewiring)  
    * Increased genetic instability 
    * Formation of desmoplasia – in some types of cancer, tumour cells can stimulate stellate cells to grow which form a mesh outside the tumour → reduce blood supply to cancer → prevent drug delivery 
* E.g. Erlotinib is a drug targeted at EGF receptor, but has a high rate of acquired resistance via 
    * Secondary mutations (e.g. T790M that reduce the affinity of drug binding without a significant drop in kinase activity) 
    * Elevated expression of MET receptor which can activate the same effectors as EGFR 
    * Increased MET receptor activation via elevated production of its ligand, HGF 

#### Overcoming drug resistance 
- **Secondary inhibitors**: If we can predict the major mechanism of resistance, we can develop secondary inhibitors that circumvent the secondary mutation 
    * E.g. Erlotinib can be used as a first line treatment, and Osimertinib as a second line treatment for EGFR inhibition (which can still inhibit T790M EGFR) 
* **Targeting multiple pathways at once** via combined therapies or sequential therapies 
* **Target common, non-redundant signalling nodes** (which cannot be compensated for with other pathways) 
	* E.g. Myc
    * Non-redundant signalling nodes are less robust than having parallel pathways, but allow for binary decision-making in cells 

## Myc inhibition 
* Targeting inhibition of the driving oncogene usually causes tumour cell death rather than growth arrest e.g. Myc inhibition 
    * #experimental_evidence  – Feeding mice with doxycycline activates expression of a Myc inhibitor that binds to both Myc and Max → prevents then from forming active DNA-binding Myc/Max heterodimer 
    * #experimental_evidence  Soucek et al (2008) – Myc inhibition triggered significant regression of KRas<sup>G12D</sup> lung tumours with remarkably mild side effects 
        * Oncogenic Ras increases Myc protein stability 
        * Regression is also independent of p53, i.e. also occurred in p53 -/- mice 
        * They used Omomyc, which is a dominant negative Myc mutant that prevents Myc transactivation of other genes, by blocking dimerisation of the Myc proteins to its obligate partner, Max. Omomyc prevents binding of Myc to consensus E-box DNA elements. On the other hand, Omomyc does not inhibit Myc-dependent gene repression. 
    * #experimental_evidence – Mammary tumours driven by Her2 oncoprotein in mice are dependent on continuous Myc activity; myc inhibition leads to decrease in tumour size 
    * Side effects – stops growth in normally proliferating tissues (e.g. villus attrition in intestine, aspermia in testis, epidermal thinning and arrested hair growth in skin), but side-effects are well-tolerated (in mice) and stopping treatment reverses the effects rapidly 
    * Even a short period of Myc inhibition confers significant survival advantage, and repeated rounds of Myc inhibition can completely contain lung tumours in mice without side-effects or tumour evolution 
    * The high therapeutic index of systemic Myc inhibition could make it a good target for cancer treatment 
* Myc inhibition in tumour cells causes tumour cell death, but inhibition in non-tumour cells allows for survival – due to **oncogene addiction**
    * Myc is required for maintenance of the tumour microenvironment, e.g. tumour angiogenic response (via vascular endothelial growth factor [VEGF]), immune evasion (via PDL-1, IL23), stromal remodelling (via matrix metalloproteinase [MMP])  
      ![[Pasted image 20230203000239.png|350]]
    * Collapse of tumour microenvironment → tumour cell death 
    * Oncogene may also be needed to maintain Warburg metabolism (aerobic glycolysis)
* Myc is often considered undruggable as it has no obvious binding clefts for binding by small molecules (similar to [[A7 G proteins#Direct targeting of Ras|Ras]]) 

### Exploiting oncogene addiction 
* Oncogene addiction = cancer becomes dependent on one or several oncogenes for survival such that inhibition results in cell death (rather than just growth arrest) 
* Because of oncogene addiction, **persistent** expression of the oncogene is needed to maintain the tumour 
* Extrinsic hypothesis – oncogene is required for maintenance of the tumour microenvironment and immune evasion, and inhibition results in collapse of the microenvironment. Also to maintain aerobic glycolysis (Warburg metabolism)?
* Intrinsic hypothesis (wrong) – cells rewired when cancerous, so if Myc is inhibited, they become maladapted to their environment 
* Tumour cells may also be addicted to certain anti-apoptotic or anti-senescence genes due to an intrinsic need to prevent apoptosis and senescence -> inhibition of these genes (e.g. Bcl2) or increasing apoptotic/senescent signalling (e.g. p53) can help to target tumour cells 

## Restoring tumour suppressor function 
* Restoration of p53 even after DNA damage can still suppress tumours (Christophorou et al 2008)
* Restoration of p53 function in established Eμ-Myc lymphomas triggers apoptosis, but only transiently
    * #experimental_evidence ... 
    * Tumours recur due to secondary loss of either p53ER or p19ARF
* p53 restoration does not trigger significant NSCLC tumour regression, but has an effect in some tumours
    * However, proliferation does go down, and apoptosis does go up

## Inhibiting tumour suppressors 
- Synthetic lethality 
- Temporary inactivation of p53 during radiotherapy or chemotherapies that induce DNA damage may help to protect normal cells from p53-mediated apoptosis (see [[D3 Oncogenes and Tumour Suppressors#^90c0e2]])

## Other types of therapy 
* Immune checkpoint therapy
* Personalised cancer therapy – tailor therapy to mutations in the tumour
    * Problem – tumour is heterogenous; each cell may have different mutations 
    * Useful for stratifying tumours 