# Structural techniques 
"It is very easy to answer many of these fundamental biological questions; you just look at the thing!" -- Richard Feynman 

> [!summary]- [[A1 Mass Spectrometry and Proteomics]]
> 1. Principle
> 2. Process
> 3. Single protein applications
> 	1. Structural determination
> 	2. Protein identification
> 4. Proteomics applications 
> 	1. Protein complexes
> 	2. Quantification
> 		1. Label-free methods
> 		2. Stable isotope labelling 
> 		3. Isobaric labelling
> 	3. Protein interactions
> 		1. Cross-link MS
> 		2. H-D exchange 
> 		3. AP-MS
> 	4. Spatial localisation of proteins 
> 		1. LOPIT 
> 		2. Protein correlation profiling 
> 		3. Proximity tagging 
> 		4. Organelle enrichment/subtractive enrichment 
> 	5. Clinical proteomics 
> 5. Limitations

**Essay**
- [[Describe how mass spectrometry can be used to gain insight into multi-protein complex components.]]

**Detailed outline**
- [[To what extent is it possible to use mass spectrometry to systematically identify and study the protein complement within a cell? How might one best go about this?]]
- [[Using mass spectrometry to investigate PTMs and PPIs|How can Mass Spectrometry-based approaches be used to investigate changes in a cellular response that involve stable protein modifications and protein-protein interactions? Briefly outline complementary methods which could be used to verify the initial findings.]] 

**Outline**
- [[Protein identification-what are the strengths and weaknesses of using mass spectrometry over an ELISA?]]

**Read** (lmao)
- In what situations would you consider using SILAC labelling for a quantitative proteomics experiment? Give examples of applications. (refer to [[ChenEtAl_2015]])

> [!summary]- [[A2 X-ray crystallography]]
> 1. Principle
> 2. Process
> 	1. Expression and purification
> 	2. Crystallisation 
> 	3. Acquire diffraction pattern 
> 	4. Solve phase problem 
> 	5. Fitting of electron density map to atomic model 
> 	6. Analysis of model 
> 3. Applications 
> 	1. Protein dynamics 
> 		1. Time-resolved XRC
> 		2. X-ray free electron laser 
> 4. Membrane XRC 
> 5. Advantages and disadvantages

> [!summary]- [[A3 Cryo-EM]]
> 1. Principle
> 2. Process
> 3. Applications
> 	1. Studying difficult-to-crystallise proteins
> 	2. Studying large proteins 
> 	3. Protein dynamics (time-resolved cryo-EM)
> 	4. Studying rare proteins
> 	5. Studying structures in-situ (cryo-ET)
> 4. Membrane cryo-EM 
> 5. Advantages and disadvantages

**Essay**
- [[Cryo-electron microscopy is the method of choice for determining protein structure and function. Discuss and give pros and cons of the method.]]

> [!summary]- [[A4 NMR]]
> 1. Principle 
> 2. Process 
> 3. Applications
> 	1. Structural determination 
> 	2. Protein folding
> 	3. Protein interactions
> 		1. Ligand-observed
> 			1. Saturation transfer difference spectroscopy 
> 			2. Cross-transfer spectroscopy
> 			3. WaterLOGSY
> 		2. Protein-observed
> 			1. 2D NMR 
> 			2. H-D exchange 
> 			3. NOE
> 	4. Protein dynamics 
> 		1. Fast (ns-ps)
> 			1. 15N spin relaxation analysis
> 			2. Paramagnetic relaxation enhancement
> 			3. Molecular dynamics simulations / chemical shift predictions
> 		2. Slow (µs-ms)
> 			1. Real time 2D NMR 
> 			2. Relaxation dispersion 
> 			3. T2 measurement 
> 		3. Very slow (>ms)
> 			1. ZZ 
> 			2. HDX 
> 			3. 2D EXSY 
> 4. Membrane protein NMR 
> 5. Limitations

**Detailed outline**
- [[How has NMR spectroscopy advanced the understanding of G-protein coupled receptor signalling?]]

> [!summary]- [[A5 Preparing membrane proteins for structural study]]
> 1. Interactions of membrane proteins with membrane lipids
> 2. Challenges in structural determination of membrane proteins 
> 3. Expression of membrane proteins
> 4. Membrane mimetic systems 
> 5. Comparing methods for structural determination of membrane protein

**Outline**
- [[Explain why the outcomes of X-ray crystallography and NMR structural studies of the same membrane protein might differ and describe the possible implications of such a scenario.]]

> [!summary]- [[A6 Comparing XRC, CryoEM, NMR]]
> 1. NMR vs XRC
> 2. XRC vs Cryo-EM 
> 3. Combining methods
> 4. Other structural methods
> 	1. Atomic force microscopy 
> 	2. Electron paramagnetic resonance
> 	3. Machine learning

# Structure, dynamics and function 

> [!summary]- [[A7 G proteins]]
> Ras, Rho, Arf, Ran, Rab 
> 1. Small G protein signalling pathway
> 	1. G proteins 
> 	2. GDIs
> 	3. GEFs
> 	4. GAPs
> 	5. Receptors 
> 2. Small G protein structure and function 
> 	1. Nucleotide binding site
> 	2. Switch mechanism 
> 	3. Allosteric lobe 
> 	4. Post-translational modifications
> 		1. Prenylation 
> 		2. Ubiquitination 
> 		3. Phosphorylation 
> 3. Heterotrimeric G protein signalling pathway 
> 	1. Signalling cycle
> 	2. Activators: GPCRs and GBA motif proteins 
> 	3. Effectors of Gα
> 		1. Gt and cGMP PDE
> 		2. Gs, Gi and adenylyl cyclase 
> 		3. Gq and PLC 
> 	4. Effectors of Gβ/γ
> 		1. Ion channels 
> 	5. Terminators: Gα, Effectors, RGS proteins
> 4. Heterotrimeric G protein structure and function 
> 	1. Gα
> 	2. Gβ/γ
> 	3. Lipid modifications 
> 5. Structural techniques for studying G proteins
> 	1. NMR 
> 6. Structure-based drug design for Ras 
> 	1. Direct targeting 
> 	2. Fragment-based drug discovery 
> 	3. Covalent inhibitors
> 	4. Protacs 
> 	5. Prenylation inhibition 
> 	6. GDI inhibitors
> 	7. Disruption of membrane interactions 
> 	8. Effector inhibition (RAF, PI3K, Ral)

**Essay**
- [[Discuss how G proteins are post-translationally modified. How do these modifications contribute to their functional interactions with other proteins?]]

**Detailed outline**
- [[Based on your knowledge of how G proteins function, design strategies to interfere with G protein signalling.]]
- [[Describe the unique features of the small G protein ___ compared to the canonical small G protein, Ras. How do these features of ___ allow it to function?]]
- [[Compare and contrast the small G proteins with heterotrimeric G proteins. Briefly speculate why these different mechanisms might have evolved.]]

> [!summary]- [[A8 Machinery of Translation]]
> 1. Structural components of translation 
> 	1. Ribosome 
> 	2. EF-Tu 
> 2. Conformational changes during translation 
> 	1. Initiation 
> 	2. Elongation
> 		1. Recruitment of tRNA to ribosome
> 		2. Decoding 
> 		3. Post-GTP hydrolysis 
> 		4. Accommodation 
> 		5. Peptidyl transfer 
> 		6. Translocation 
> 		7. Recycling of EF-Tu 
> 	3. Structural techniques to study elongation 
> 	4. Termination
> 	5. Ribosome recycling 
> 3. Fidelity of translation
> 	1. Errors in protein biosynthesis 
> 	2. tRNA charging 
> 	3. Decoding 
> 		1. Watson-crick pairing 
> 		2. Interaction between codon:anticodon helix and decoding center nucleotides 
> 		3. Distortions in tRNA body beyond anticodon 
> 		4. Accommodation 
> 		5. Kinetic proofreading 
> 4. Chemistry of peptide bond formation 
> 5. Protein biogenesis factors 
> 	1. Chaperones
> 	2. Localisation  

**Essay**
- [[Explain why energy is needed in translation.]]

**Detailed outline**
- [[Discuss how the ribosome might be considered an allosteric assembly.]]
- [[What are the outstanding mechanistic questions in understanding the translation of mRNA into protein, and how would you go about solving them?]]
- [[How have structural data of various conformations of elongating ribosomes advanced our understanding of translation?]]

**Outline**

# Designing new proteins based on structural knowledge 

> [!summary]- [[A9 Protein Engineering (NEW)]]
> 1. Enzyme catalysis 
> 2. Rational design 
> 3. Directed evolution of RNA 
> 	1. Ribozymes 
> 	2. SELEX
> 	3. Tagged substrates 
> 4. Fitness landscapes 
> 	1. Directed evolution is navigation of a fitness landscape 
> 	2. Features of fitness landscapes
> 	3. Effect of epistasis on landscape 
> 5. Strategies for directed evolution
> 	1. Selecting a starting sequence 
> 	2. Generate a DNA library 
> 		1. Random methods 
> 		2. Directed methods 
> 		3. Recombination methods 
> 	3. Stepwise vs multiple mutations
> 	4. Screening methods 
> 		1. Functional assays
> 		2. Phage display 
> 		3. Colony screening 
> 		4. Ribosome display
> 		5. mRNA-puromycin fusion/mRNA display 
> 		6. Droplet screening 
> 			1. In-vitro compartmentalisation 
> 			2. Compartmentalised self-replication (CSR)
> 			3. Microfluidics 
> 	5. Solving local optima problem 
> 6. Strategies for specific goals
> 	1. Novel enzymatic activity 
> 	2. Increasing specificity 
> 	3. Improving stability 
> 7. Natural selection vs directed evolution 

**Outline**
- [[Outline different approaches to generate functional proteins with novel properties and discuss any difficulties likely to be encountered]]

# Broad questions
- Comparing different methods – mostly, is NMR and X-ray crystallography redundant? 
- Understanding both structure and dynamics, not just either 
- Combining techniques to get a holistic understanding of protein **structure** AND **function** (and possibly other single molecules)

**Detailed outline**
- [[Discuss how the structures of dynamic proteins and assemblies can be studied. What functional insights can be gained from such studies?]]
- [[What methods can be used to study single molecules in biology? Using examples, outline some of the advantages of such approaches.]]
- [[How would you identify and characterise the components of an extracellular-ligand-dependent signalling pathway that activates transcription factors?]]

**Outline**
- [[What is the experimental evidence for internal motions in proteins? How do these motions contribute to an enzyme’s function?]]